<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>

<style type='text/css'>html {overflow-x: initial !important;}:root { --bg-color: #ffffff; --text-color: #333333; --select-text-bg-color: #B5D6FC; --select-text-font-color: auto; --monospace: "Lucida Console",Consolas,"Courier",monospace; --title-bar-height: 20px; }
.mac-os-11 { --title-bar-height: 28px; }
html { font-size: 14px; background-color: var(--bg-color); color: var(--text-color); font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; -webkit-font-smoothing: antialiased; }
body { margin: 0px; padding: 0px; height: auto; inset: 0px; font-size: 1rem; line-height: 1.42857143; overflow-x: hidden; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; tab-size: 4; background-position: inherit; background-repeat: inherit; }
iframe { margin: auto; }
a.url { word-break: break-all; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { text-shadow: none; background: var(--select-text-bg-color); color: var(--select-text-font-color); }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; white-space: normal; overflow-x: visible; padding-top: 36px; }
#write.first-line-indent p { text-indent: 2em; }
#write.first-line-indent li p, #write.first-line-indent p * { text-indent: 0px; }
#write.first-line-indent li { margin-left: 2em; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
.typora-export .footnote-line, .typora-export li, .typora-export p { white-space: pre-wrap; }
.typora-export .task-list-item input { pointer-events: none; }
@media screen and (max-width: 500px) { 
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  #write { padding-left: 20px; padding-right: 20px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
#write li > figure:last-child { margin-bottom: 0.5rem; }
#write ol, #write ul { position: relative; }
img { max-width: 100%; vertical-align: middle; image-orientation: from-image; }
button, input, select, textarea { color: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-caps: inherit; font-weight: inherit; font-stretch: inherit; line-height: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
*, ::after, ::before { box-sizing: border-box; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p, #write pre { width: inherit; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p { position: relative; }
p { line-height: inherit; }
h1, h2, h3, h4, h5, h6 { break-after: avoid-page; break-inside: avoid; orphans: 4; }
p { orphans: 4; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
.md-math-block, .md-rawblock, h1, h2, h3, h4, h5, h6, p { margin-top: 1rem; margin-bottom: 1rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: 700; font-style: italic; }
a { cursor: pointer; }
sup.md-footnote { padding: 2px 4px; background-color: rgba(238, 238, 238, 0.7); color: rgb(85, 85, 85); border-top-left-radius: 4px; border-top-right-radius: 4px; border-bottom-right-radius: 4px; border-bottom-left-radius: 4px; cursor: pointer; }
sup.md-footnote a, sup.md-footnote a:hover { color: inherit; text-transform: inherit; text-decoration: inherit; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; }
figure { overflow-x: auto; margin: 1.2em 0px; max-width: calc(100% + 16px); padding: 0px; }
figure > table { margin: 0px; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 32px; }
.CodeMirror-gutters { border-right-width: 0px; background-color: inherit; }
.CodeMirror-linenumber { }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
#write pre { white-space: pre-wrap; }
#write.fences-no-line-wrapping pre { white-space: pre; }
#write pre.ty-contain-cm { white-space: normal; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; position: relative !important; background-position: inherit; background-repeat: inherit; }
.md-fences-adv-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
#write .md-fences.mock-cm { white-space: pre-wrap; }
.md-fences.md-fences-with-lineno { padding-left: 0px; }
#write.fences-no-line-wrapping .md-fences.mock-cm { white-space: pre; overflow-x: auto; }
.md-fences.mock-cm.md-fences-with-lineno { padding-left: 8px; }
.CodeMirror-line, twitterwidget { break-inside: avoid; }
.footnotes { opacity: 0.8; font-size: 0.9rem; margin-top: 1em; margin-bottom: 1em; }
.footnotes + .footnotes { margin-top: 0px; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; line-height: normal; font-weight: 400; text-align: left; box-sizing: content-box; direction: ltr; background-position: 0px 0px; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li .mathjax-block, li p { margin: 0.5rem 0px; }
li blockquote { margin: 1rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child, li > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; white-space: normal; }
#write .footnote-line { white-space: pre-wrap; }
@media print { 
  body, html { border: 1px solid transparent; height: 99%; break-after: avoid; break-before: avoid; font-variant-ligatures: no-common-ligatures; }
  #write { margin-top: 0px; padding-top: 0px; border-color: transparent !important; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  .typora-export #write { break-after: avoid; }
  .typora-export #write::after { height: 0px; }
  .is-mac table { break-inside: avoid; }
  .typora-export-show-outline .typora-export-sidebar { display: none; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 0.8rem; white-space: pre-wrap; background-color: rgb(204, 204, 204); display: block; overflow-x: hidden; }
p > .md-image:only-child:not(.md-img-error) img, p > img:only-child { display: block; margin: auto; }
#write.first-line-indent p > .md-image:only-child:not(.md-img-error) img { left: -2em; position: relative; }
p > .md-image:only-child { display: inline-block; width: 100%; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.md-math-block { width: 100%; }
.md-math-block:not(:empty)::after { display: none; }
.MathJax_ref { fill: currentcolor; }
[contenteditable="true"]:active, [contenteditable="true"]:focus, [contenteditable="false"]:active, [contenteditable="false"]:focus { outline: 0px; box-shadow: none; }
.md-task-list-item { position: relative; list-style-type: none; }
.task-list-item.md-task-list-item { padding-left: 0px; }
.md-task-list-item > input { position: absolute; top: 0px; left: 0px; margin-left: -1.2em; margin-top: calc(1em - 10px); border: none; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-top-left-radius: 10px; border-top-right-radius: 10px; border-bottom-right-radius: 10px; border-bottom-left-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc-content::after, .md-toc::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); }
.md-toc-item a { text-decoration: none; }
.md-toc-inner:hover { text-decoration: underline; }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: 700; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) { 
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
code, pre, samp, tt { font-family: var(--monospace); }
kbd { margin: 0px 0.1em; padding: 0.1em 0.6em; font-size: 0.8em; color: rgb(36, 39, 41); background-color: rgb(255, 255, 255); border: 1px solid rgb(173, 179, 185); border-top-left-radius: 3px; border-top-right-radius: 3px; border-bottom-right-radius: 3px; border-bottom-left-radius: 3px; box-shadow: rgba(12, 13, 14, 0.2) 0px 1px 0px, rgb(255, 255, 255) 0px 0px 0px 2px inset; white-space: nowrap; vertical-align: middle; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: var(--monospace); }
code { text-align: left; }
a.md-print-anchor { white-space: pre !important; border: none !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: 0px !important; text-shadow: initial !important; background-position: 0px 0px !important; }
.os-windows.monocolor-emoji .md-emoji { font-family: "Segoe UI Symbol", sans-serif; }
.md-diagram-panel > svg { max-width: 100%; }
[lang="flow"] svg, [lang="mermaid"] svg { max-width: 100%; height: auto; }
[lang="mermaid"] .node text { font-size: 1rem; }
table tr th { border-bottom-width: 0px; }
video { max-width: 100%; display: block; margin: 0px auto; }
iframe { max-width: 100%; width: 100%; border: none; }
.highlight td, .highlight tr { border: 0px; }
mark { background-color: rgb(255, 255, 0); color: rgb(0, 0, 0); }
.md-html-inline .md-plain, .md-html-inline strong, mark .md-inline-math, mark strong { color: inherit; }
.md-expand mark .md-meta { opacity: 0.3 !important; }
mark .md-meta { color: rgb(0, 0, 0); }
@media print { 
  .typora-export h1, .typora-export h2, .typora-export h3, .typora-export h4, .typora-export h5, .typora-export h6 { break-inside: avoid; }
}
.md-diagram-panel .messageText { stroke: none !important; }
.md-diagram-panel .start-state { fill: var(--node-fill); }
.md-diagram-panel .edgeLabel rect { opacity: 1 !important; }
.md-require-zoom-fix { height: auto; margin-top: 16px; margin-bottom: 16px; }
.md-require-zoom-fix foreignObject { font-size: var(--mermaid-font-zoom); }
.md-fences.md-fences-math { font-size: 1em; }
.md-fences-advanced:not(.md-focus) { padding: 0px; white-space: nowrap; border: 0px; }
.md-fences-advanced:not(.md-focus) { background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; background-position: inherit; background-repeat: inherit; }
.typora-export-show-outline .typora-export-content { max-width: 1440px; margin: auto; display: flex; flex-direction: row; }
.typora-export-sidebar { width: 300px; font-size: 0.8rem; margin-top: 80px; margin-right: 18px; }
.typora-export-show-outline #write { --webkit-flex: 2; flex: 2 1 0%; }
.typora-export-sidebar .outline-content { position: fixed; top: 0px; max-height: 100%; overflow: hidden auto; padding-bottom: 30px; padding-top: 60px; width: 300px; }
@media screen and (max-width: 1024px) { 
  .typora-export-sidebar, .typora-export-sidebar .outline-content { width: 240px; }
}
@media screen and (max-width: 800px) { 
  .typora-export-sidebar { display: none; }
}
.outline-content li, .outline-content ul { margin-left: 0px; margin-right: 0px; padding-left: 0px; padding-right: 0px; list-style: none; }
.outline-content ul { margin-top: 0px; margin-bottom: 0px; }
.outline-content strong { font-weight: 400; }
.outline-expander { width: 1rem; height: 1.428571429rem; position: relative; display: table-cell; vertical-align: middle; cursor: pointer; padding-left: 4px; }
.outline-expander::before { content: ''; position: relative; font-family: Ionicons; display: inline-block; font-size: 8px; vertical-align: middle; }
.outline-item { padding-top: 3px; padding-bottom: 3px; cursor: pointer; }
.outline-expander:hover::before { content: ''; }
.outline-h1 > .outline-item { padding-left: 0px; }
.outline-h2 > .outline-item { padding-left: 1em; }
.outline-h3 > .outline-item { padding-left: 2em; }
.outline-h4 > .outline-item { padding-left: 3em; }
.outline-h5 > .outline-item { padding-left: 4em; }
.outline-h6 > .outline-item { padding-left: 5em; }
.outline-label { cursor: pointer; display: table-cell; vertical-align: middle; text-decoration: none; color: inherit; }
.outline-label:hover { text-decoration: underline; }
.outline-item:hover { border-color: rgb(245, 245, 245); background-color: var(--item-hover-bg-color); }
.outline-item:hover { margin-left: -28px; margin-right: -28px; border-left-width: 28px; border-left-style: solid; border-left-color: transparent; border-right-width: 28px; border-right-style: solid; border-right-color: transparent; }
.outline-item-single .outline-expander::before, .outline-item-single .outline-expander:hover::before { display: none; }
.outline-item-open > .outline-item > .outline-expander::before { content: ''; }
.outline-children { display: none; }
.info-panel-tab-wrapper { display: none; }
.outline-item-open > .outline-children { display: block; }
.typora-export .outline-item { padding-top: 1px; padding-bottom: 1px; }
.typora-export .outline-item:hover { margin-right: -8px; border-right-width: 8px; border-right-style: solid; border-right-color: transparent; }
.typora-export .outline-expander::before { content: "+"; font-family: inherit; top: -1px; }
.typora-export .outline-expander:hover::before, .typora-export .outline-item-open > .outline-item > .outline-expander::before { content: '−'; }
.typora-export-collapse-outline .outline-children { display: none; }
.typora-export-collapse-outline .outline-item-open > .outline-children, .typora-export-no-collapse-outline .outline-children { display: block; }
.typora-export-no-collapse-outline .outline-expander::before { content: "" !important; }
.typora-export-show-outline .outline-item-active > .outline-item .outline-label { font-weight: 700; }
.md-inline-math-container mjx-container { zoom: 0.95; }


:root {
  --mermaid-theme: night;
}

[lang='mermaid'] .label {
  color: #333;
}

/* CSS Document */

/** code highlight */

.cm-s-inner .cm-variable,
.cm-s-inner .cm-operator,
.cm-s-inner .cm-property {
    color: #b8bfc6;
}

.cm-s-inner .cm-keyword {
    color: #C88FD0;
}

.cm-s-inner .cm-tag {
    color: #7DF46A;
}

.cm-s-inner .cm-attribute {
    color: #7575E4;
}

.CodeMirror div.CodeMirror-cursor {
    border-left: 1px solid #b8bfc6;
    z-index: 3;
}

.cm-s-inner .cm-string {
    color: #D26B6B;
}

.cm-s-inner .cm-comment,
.cm-s-inner.cm-comment {
    color: #DA924A;
}

.cm-s-inner .cm-header,
.cm-s-inner .cm-def,
.cm-s-inner.cm-header,
.cm-s-inner.cm-def {
    color: #8d8df0;
}

.cm-s-inner .cm-quote,
.cm-s-inner.cm-quote {
    color: #57ac57;
}

.cm-s-inner .cm-hr {
    color: #d8d5d5;
}

.cm-s-inner .cm-link {
    color: #d3d3ef;
}

.cm-s-inner .cm-negative {
    color: #d95050;
}

.cm-s-inner .cm-positive {
    color: #50e650;
}

.cm-s-inner .cm-string-2 {
    color: #f50;
}

.cm-s-inner .cm-meta,
.cm-s-inner .cm-qualifier {
    color: #b7b3b3;
}

.cm-s-inner .cm-builtin {
    color: #f3b3f8;
}

.cm-s-inner .cm-bracket {
    color: #997;
}

.cm-s-inner .cm-atom,
.cm-s-inner.cm-atom {
    color: #84B6CB;
}

.cm-s-inner .cm-number {
    color: #64AB8F;
}

.cm-s-inner .cm-variable {
    color: #b8bfc6;
}

.cm-s-inner .cm-variable-2 {
    color: #9FBAD5;
}

.cm-s-inner .cm-variable-3 {
    color: #1cc685;
}

.CodeMirror-selectedtext,
.CodeMirror-selected {
    background: #4a89dc;
    color: #fff !important;
    text-shadow: none;
}

.CodeMirror-gutters {
    border-right: none;
}

/* CSS Document */

/** markdown source **/
.cm-s-typora-default .cm-header, 
.cm-s-typora-default .cm-property
{
    color: #cebcca;
}

.CodeMirror.cm-s-typora-default div.CodeMirror-cursor{
    border-left: 3px solid #b8bfc6;
}

.cm-s-typora-default .cm-comment {
    color: #9FB1FF;
}

.cm-s-typora-default .cm-string {
    color: #A7A7D9
}

.cm-s-typora-default .cm-atom, .cm-s-typora-default .cm-number {
    color: #848695;
    font-style: italic;
}

.cm-s-typora-default .cm-link {
    color: #95B94B;
}

.cm-s-typora-default .CodeMirror-activeline-background {
    background: rgba(51, 51, 51, 0.72);
}

.cm-s-typora-default .cm-comment, .cm-s-typora-default .cm-code {
	color: #8aa1e1;
}@import "";
@import "";
@import "";

:root {
    --bg-color:  #363B40;
    --side-bar-bg-color: #2E3033;
    --text-color: #b8bfc6;

    --select-text-bg-color:#4a89dc;

    --item-hover-bg-color: #0a0d16;
    --control-text-color: #b7b7b7;
    --control-text-hover-color: #eee;
    --window-border: 1px solid #555;

    --active-file-bg-color: rgb(34, 34, 34);
    --active-file-border-color: #8d8df0;

    --primary-color: #a3d5fe;

    --active-file-text-color: white;
    --item-hover-bg-color: #70717d;
    --item-hover-text-color: white;
    --primary-color: #6dc1e7;

    --rawblock-edit-panel-bd: #333;

    --search-select-bg-color: #428bca;
}

html {
    font-size: 16px;
    -webkit-font-smoothing: antialiased;
}

html,
body {
    -webkit-text-size-adjust: 100%;
    -ms-text-size-adjust: 100%;
    background: #363B40;
    background: var(--bg-color);
    fill: currentColor;
    line-height: 1.625rem;
}

#write {
    max-width: 914px;
}


@media only screen and (min-width: 1400px) {
    #write {
        max-width: 1024px;
    }
}

@media only screen and (min-width: 1800px) {
    #write {
        max-width: 1200px;
    }
}

html,
body,
button,
input,
select,
textarea,
div.code-tooltip-content {
    color: #b8bfc6;
    border-color: transparent;
}

div.code-tooltip,
.md-hover-tip .md-arrow:after {
    background: #333;
}

.native-window #md-notification {
    border: 1px solid #70717d;
}

.popover.bottom > .arrow:after {
    border-bottom-color: #333;
}

html,
body,
button,
input,
select,
textarea {
    font-family: "Helvetica Neue", Helvetica, Arial, 'Segoe UI Emoji', sans-serif;
}

hr {
    height: 2px;
    border: 0;
    margin: 24px 0 !important;
}

h1,
h2,
h3,
h4,
h5,
h6 {
    font-family: "Lucida Grande", "Corbel", sans-serif;
    font-weight: normal;
    clear: both;
    -ms-word-wrap: break-word;
    word-wrap: break-word;
    margin: 0;
    padding: 0;
    color: #DEDEDE
}

h1 {
    font-size: 2.5rem;
    /* 36px */
    line-height: 2.75rem;
    /* 40px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1.5px;
}

h2 {
    font-size: 1.63rem;
    /* 24px */
    line-height: 1.875rem;
    /* 30px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h3 {
    font-size: 1.17rem;
    /* 18px */
    line-height: 1.5rem;
    /* 24px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h4 {
    font-size: 1.12rem;
    /* 16px */
    line-height: 1.375rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    color: white;
}

h5 {
    font-size: 0.97rem;
    /* 16px */
    line-height: 1.25rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    font-weight: bold;
}

h6 {
    font-size: 0.93rem;
    /* 16px */
    line-height: 1rem;
    /* 16px */
    margin-bottom: 0.75rem;
    color: white;
}

@media (min-width: 980px) {
    h3.md-focus:before,
    h4.md-focus:before,
    h5.md-focus:before,
    h6.md-focus:before {
        color: #ddd;
        border: 1px solid #ddd;
        border-radius: 3px;
        position: absolute;
        left: -1.642857143rem;
        top: .357142857rem;
        float: left;
        font-size: 9px;
        padding-left: 2px;
        padding-right: 2px;
        vertical-align: bottom;
        font-weight: normal;
        line-height: normal;
    }

    h3.md-focus:before {
        content: 'h3';
    }

    h4.md-focus:before {
        content: 'h4';
    }

    h5.md-focus:before {
        content: 'h5';
        top: 0px;
    }

    h6.md-focus:before {
        content: 'h6';
        top: 0px;
    }
}

a {
    text-decoration: none;
    outline: 0;
}

a:hover {
    outline: 0;
}

a:focus {
    outline: thin dotted;
}

sup.md-footnote {
    background-color: #555;
    color: #ddd;
}

p {
    -ms-word-wrap: break-word;
    word-wrap: break-word;
}

p,
ul,
dd,
ol,
hr,
address,
pre,
table,
iframe,
.wp-caption,
.wp-audio-shortcode,
.wp-video-shortcode {
    margin-top: 0;
    margin-bottom: 1.5rem;
    /* 24px */
}

audio:not([controls]) {
    display: none;
}

[hidden] {
    display: none;
}

::-moz-selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

*.in-text-selection,
::selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

ul,
ol {
    padding: 0 0 0 1.875rem;
    /* 30px */
}

ul {
    list-style: square;
}

ol {
    list-style: decimal;
}

ul ul,
ol ol,
ul ol,
ol ul {
    margin: 0;
}

b,
th,
dt,
strong {
    font-weight: bold;
}

i,
em,
dfn,
cite {
    font-style: italic;
}

/*blockquote {
    padding-left: 1.875rem;
    margin: 0 0 1.875rem 1.875rem;
    border-left: solid 2px #474d54;
    padding-left: 30px;
    margin-top: 35px;
}*/

blockquote {
    padding-left: 1.875rem;
    margin: 0 0 1.875rem 1.875rem;
    padding-left: 30px;
    margin-top: 35px;
    border-left: 2px solid #D5C5C8;
    background: #474d54;
    background-color: #474d54;
    color: #B197FC;
}

/*#772D8B , #2660A4 , #D90368 , #048BA8*/
/*https://coolors.co/d6ffb7-61cc9e-048ba8-ffbfa0-d5c5c8*/
/*blockquote blockquote { border-left: 3px solid #61CC9E }*/
blockquote blockquote { border-left: 2px solid #D6FFB7; border-bottom:none; }
blockquote blockquote blockquote { border-left: 2px solid #FFBFA0; border-bottom:none; }
blockquote blockquote blockquote blockquote { border-left: 2px solid #048BA8; border-bottom:none; }
blockquote blockquote blockquote blockquote blockquote { border-left: 2px solid #D90368; border-bottom:none; }

blockquote .md-math-block {
    color: #FFEBFF;
}

blockquote h2 {
    color: #EBD6A6;
}

blockquote h6 {
    border-radius:2em;
    opacity: 0.9;
    background-color:#B1C6FC;
    color:black;
    hite-space:normal;
    display:inline-block;
    padding:0.2em;
}


pre,
code,
kbd,
tt,
var {
    font-size: 0.875em;
    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
}

code,
tt,
var {
    background: rgba(0, 0, 0, 0.05);
}

kbd {
    padding: 2px 4px;
    font-size: 90%;
    color: #fff;
    background-color: #333;
    border-radius: 3px;
    box-shadow: inset 0 -1px 0 rgba(0,0,0,.25);
}

pre.md-fences {
    padding: 10px 10px 10px 30px;
    margin-bottom: 20px;
    background: #333;
}

.CodeMirror-gutters {
    background: #333;
    border-right: 1px solid transparent;
}

.enable-diagrams pre.md-fences[lang="sequence"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="flow"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="mermaid"] .code-tooltip {
    bottom: -2.2em;
    right: 4px;
}

code,
kbd,
tt,
var {
    padding: 2px 5px;
}

table {
    max-width: 100%;
    width: 100%;
    border-collapse: collapse;
    border-spacing: 0;
}

th,
td {
    padding: 5px 10px;
    vertical-align: top;
}

a {
    -webkit-transition: all .2s ease-in-out;
    transition: all .2s ease-in-out;
}

hr {
    background: #474d54;
    /* variable */
}

h1 {
    margin-top: 2em;
}

a {
    color: #e0e0e0;
    text-decoration: underline;
}

a:hover {
    color: #fff;
}

.md-inline-math script {
    color: #81b1db;
}

b,
th,
dt,
strong {
    color: #DEDEDE;
    /* variable */
}

mark {
    background: #D3D40E;
}

/*blockquote {
    color: #9DA2A6;
}
*/
table a {
    color: #DEDEDE;
    /* variable */
}

th,
td {
    border: solid 1px #474d54;
    /* variable */
}

.task-list {
    padding-left: 0;
}

.md-task-list-item {
    padding-left: 1.25rem;
}

.md-task-list-item > input {
    top: auto;
}

.md-task-list-item > input:before {
    content: "";
    display: inline-block;
    width: 0.875rem;
    height: 0.875rem;
    vertical-align: middle;
    text-align: center;
    border: 1px solid #b8bfc6;
    background-color: #363B40;
    margin-top: -0.4rem;
}

.md-task-list-item > input:checked:before,
.md-task-list-item > input[checked]:before {
    content: '\221A';
    /*◘*/
    font-size: 0.625rem;
    line-height: 0.625rem;
    color: #DEDEDE;
}

/** quick open **/
.auto-suggest-container {
    border: 0px;
    background-color: #525C65;
}

#typora-quick-open {
    background-color: #525C65;
}

#typora-quick-open input{
    background-color: #525C65;
    border: 0;
    border-bottom: 1px solid grey;
}

.typora-quick-open-item {
    background-color: inherit;
    color: inherit;
}

.typora-quick-open-item.active,
.typora-quick-open-item:hover {
    background-color: #4D8BDB;
    color: white;
}

.typora-quick-open-item:hover {
    background-color: rgba(77, 139, 219, 0.8);
}

.typora-search-spinner > div {
  background-color: #fff;
}

#write pre.md-meta-block {
    border-bottom: 1px dashed #ccc;
    background: transparent;
    padding-bottom: 0.6em;
    line-height: 1.6em;
}

.btn,
.btn .btn-default {
    background: transparent;
    color: #b8bfc6;
}

.ty-table-edit {
    border-top: 1px solid gray;
    background-color: #363B40;
}

.popover-title {
    background: transparent;
}

.md-image>.md-meta {
    color: #BBBBBB;
    background: transparent;
}

.md-expand.md-image>.md-meta {
    color: #DDD;
}

#write>h3:before,
#write>h4:before,
#write>h5:before,
#write>h6:before {
    border: none;
    border-radius: 0px;
    color: #888;
    text-decoration: underline;
    left: -1.4rem;
    top: 0.2rem;
}

#write>h3.md-focus:before {
    top: 2px;
}

#write>h4.md-focus:before {
    top: 2px;
}

.md-toc-item {
    color: #A8C2DC;
}

#write div.md-toc-tooltip {
    background-color: #363B40;
}

.dropdown-menu .btn:hover,
.dropdown-menu .btn:focus,
.md-toc .btn:hover,
.md-toc .btn:focus {
    color: white;
    background: black;
}

#toc-dropmenu {
    background: rgba(50, 54, 59, 0.93);
    border: 1px solid rgba(253, 253, 253, 0.15);
}

#toc-dropmenu .divider {
    background-color: #9b9b9b;
}

.outline-expander:before {
    top: 2px;
}

#typora-sidebar {
    box-shadow: none;
    border-right: 1px dashed;
    border-right: none;
}

.sidebar-tabs {
    border-bottom:0;
}

#typora-sidebar:hover .outline-title-wrapper {
    border-left: 1px dashed;
}

.outline-title-wrapper .btn {
    color: inherit;
}

.outline-item:hover {
    border-color: #363B40;
    background-color: #363B40;
    color: white;
}

h1.md-focus .md-attr,
h2.md-focus .md-attr,
h3.md-focus .md-attr,
h4.md-focus .md-attr,
h5.md-focus .md-attr,
h6.md-focus .md-attr,
.md-header-span .md-attr {
    color: #8C8E92;
    display: inline;
}

.md-comment {
    color: #5a95e3;
    opacity: 1;
}

.md-inline-math svg {
    color: #b8bfc6;
}

#math-inline-preview .md-arrow:after {
    background: black;
}

.modal-content {
    background: var(--bg-color);
    border: 0;
}

.modal-title {
    font-size: 1.5em;
}

.modal-content input {
    background-color: rgba(26, 21, 21, 0.51);
    color: white;
}

.modal-content .input-group-addon {
    color: white;
}

.modal-backdrop {
    background-color: rgba(174, 174, 174, 0.7);
}

.modal-content .btn-primary {
    border-color: var(--primary-color);
}

.md-table-resize-popover {
    background-color: #333;
}

.form-inline .input-group .input-group-addon {
    color: white;
}

#md-searchpanel {
    border-bottom: 1px dashed grey;
}

/** UI for electron */

.context-menu,
#spell-check-panel,
#footer-word-count-info {
    background-color: #42464A;
}

.context-menu.dropdown-menu .divider,
.dropdown-menu .divider {
    background-color: #777777;
    opacity: 1;
}

footer {
    color: inherit;
}

@media (max-width: 1000px) {
    footer {
        border-top: none;
    }
    footer:hover {
        color: inherit;
    }
}

#file-info-file-path .file-info-field-value:hover {
    background-color: #555;
    color: #dedede;
}

.megamenu-content,
.megamenu-opened header {
    background: var(--bg-color);
}

.megamenu-menu-panel h2,
.megamenu-menu-panel h1,
.long-btn {
    color: inherit;
}

.megamenu-menu-panel input[type='text'] {
    background: inherit;
    border: 0;
    border-bottom: 1px solid;
}

#recent-file-panel-action-btn {
    background: inherit;
    border: 1px grey solid;
}

.megamenu-menu-panel .dropdown-menu > li > a {
    color: inherit;
    background-color: #2F353A;
    text-decoration: none;
}

.megamenu-menu-panel table td:nth-child(1) {
    color: inherit;
    font-weight: bold;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(1) {
    color: white;
}

.modal-footer .btn-default,
.modal-footer .btn-primary,
.modal-footer .btn-default:not(:hover) {
    border: 1px solid;
    border-color: transparent;
}

.btn-primary {
    color: white;
}

.btn-default:hover, .btn-default:focus, .btn-default.focus, .btn-default:active, .btn-default.active, .open > .dropdown-toggle.btn-default {
    color: white;
    border: 1px solid #ddd;
    background-color: inherit;
}

.modal-header {
    border-bottom: 0;
}

.modal-footer {
    border-top: 0;
}

#recent-file-panel tbody tr:nth-child(2n-1) {
    background-color: transparent !important;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(2) {
    color: inherit;
}

.megamenu-menu-panel .btn {
    border: 1px solid #eee;
    background: transparent;
}

.mouse-hover .toolbar-icon.btn:hover,
#w-full.mouse-hover,
#w-pin.mouse-hover {
    background-color: inherit;
}

.typora-node::-webkit-scrollbar {
    width: 5px;
}

.typora-node::-webkit-scrollbar-thumb:vertical {
    background: rgba(250, 250, 250, 0.3);
}

.typora-node::-webkit-scrollbar-thumb:vertical:active {
    background: rgba(250, 250, 250, 0.5);
}

#w-unpin {
    background-color: #4182c4;
}

#top-titlebar, #top-titlebar * {
    color: var(--item-hover-text-color);
}

.typora-sourceview-on #toggle-sourceview-btn,
#footer-word-count:hover,
.ty-show-word-count #footer-word-count {
    background: #333333;
}

#toggle-sourceview-btn:hover {
    color: #eee;
    background: #333333;
}

/** focus mode */
.on-focus-mode .md-end-block:not(.md-focus):not(.md-focus-container) * {
    color: #686868 !important;
}

.on-focus-mode .md-end-block:not(.md-focus) img,
.on-focus-mode .md-task-list-item:not(.md-focus-container)>input {
    opacity: #686868 !important;
}

.on-focus-mode li[cid]:not(.md-focus-container){
    color: #686868;
}

.on-focus-mode .md-fences.md-focus .CodeMirror-code>*:not(.CodeMirror-activeline) *,
.on-focus-mode .CodeMirror.cm-s-inner:not(.CodeMirror-focused) * {
    color: #686868 !important;
}

.on-focus-mode .md-focus,
.on-focus-mode .md-focus-container {
    color: #fff;
}

.on-focus-mode #typora-source .CodeMirror-code>*:not(.CodeMirror-activeline) * {
    color: #686868 !important;
}


/*diagrams*/
#write .md-focus .md-diagram-panel {
    border: 1px solid #ddd;
    margin-left: -1px;
    width: calc(100% + 2px);
}

/*diagrams*/
#write .md-focus.md-fences-with-lineno .md-diagram-panel {
    margin-left: auto;
}

.md-diagram-panel-error {
    color: #f1908e;
}

.active-tab-files #info-panel-tab-file,
.active-tab-files #info-panel-tab-file:hover,
.active-tab-outline #info-panel-tab-outline,
.active-tab-outline #info-panel-tab-outline:hover {
    color: #eee;
}

.sidebar-footer-item:hover,
.footer-item:hover {
    background: inherit;
    color: white;
}

.ty-side-sort-btn.active,
.ty-side-sort-btn:hover,
.selected-folder-menu-item a:after {
    color: white;
}

#sidebar-files-menu {
    border:solid 1px;
    box-shadow: 4px 4px 20px rgba(0, 0, 0, 0.79);
    background-color: var(--bg-color);
}

.file-list-item {
    border-bottom:none;
}

.file-list-item-summary {
    opacity: 1;
}

.file-list-item.active:first-child {
    border-top: none;
}

.file-node-background {
    height: 32px;
}

.file-library-node.active>.file-node-content,
.file-list-item.active {
    color: white;
    color: var(--active-file-text-color);
}

.file-library-node.active>.file-node-background{
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}
.file-list-item.active {
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}

#ty-tooltip {
    background-color: black;
    color: #eee;
}

.md-task-list-item>input {
    margin-left: -1.3em;
    margin-top: 0.3rem;
    -webkit-appearance: none;
}

.md-mathjax-midline {
    background-color: #57616b;
    border-bottom: none;
}

footer.ty-footer {
    border-color: #656565;
}

.ty-preferences .btn-default {
    background: transparent;
}
.ty-preferences .btn-default:hover {
    background: #57616b;
}

.ty-preferences select {
    border: 1px solid #989698;
    height: 21px;
}

.ty-preferences .nav-group-item.active,
.export-item.active,
.export-items-list-control,
.export-detail {
    background: var(--item-hover-bg-color);
}

.ty-preferences input[type="search"] {
    border-color: #333;
    background: #333;
    line-height: 22px;
    border-radius: 6px;
    color: white;
}

.ty-preferences input[type="search"]:focus {
    box-shadow: none;
}

[data-is-directory="true"] .file-node-content {
    margin-bottom: 0;
}

.file-node-title {
    line-height: 22px;
}

.html-for-mac .file-node-open-state, .html-for-mac .file-node-icon {
    line-height: 26px;
}

::-webkit-scrollbar-thumb {
    background: rgba(230, 230, 230, 0.30);
}

::-webkit-scrollbar-thumb:active {
    background: rgba(230, 230, 230, 0.50);
}

#typora-sidebar:hover div.sidebar-content-content::-webkit-scrollbar-thumb:horizontal {
    background: rgba(230, 230, 230, 0.30);
}

.nav-group-item:active {
    background-color: #474d54 !important;
}

.md-search-hit {
    background: rgba(199, 140, 60, 0.81);
    color: #eee;
}

.md-search-hit * {
    color: #eee;
}

#md-searchpanel input {
    color: white;
}

.modal-backdrop.in {
    opacity: 1;
    backdrop-filter: blur(1px);
}

.clear-btn-icon {
    top: 8px;
}


mjx-container[jax="SVG"] {
  direction: ltr;
}

mjx-container[jax="SVG"] > svg {
  overflow: visible;
  min-height: 1px;
  min-width: 1px;
}

mjx-container[jax="SVG"] > svg a {
  fill: blue;
  stroke: blue;
}

mjx-assistive-mml {
  position: absolute !important;
  top: 0px;
  left: 0px;
  clip: rect(1px, 1px, 1px, 1px);
  padding: 1px 0px 0px 0px !important;
  border: 0px !important;
  display: block !important;
  width: auto !important;
  overflow: hidden !important;
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -khtml-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

mjx-assistive-mml[display="block"] {
  width: 100% !important;
}

mjx-container[jax="SVG"][display="true"] {
  display: block;
  text-align: center;
  margin: 1em 0;
}

mjx-container[jax="SVG"][display="true"][width="full"] {
  display: flex;
}

mjx-container[jax="SVG"][justify="left"] {
  text-align: left;
}

mjx-container[jax="SVG"][justify="right"] {
  text-align: right;
}

g[data-mml-node="merror"] > g {
  fill: red;
  stroke: red;
}

g[data-mml-node="merror"] > rect[data-background] {
  fill: yellow;
  stroke: none;
}

g[data-mml-node="mtable"] > line[data-line], svg[data-table] > g > line[data-line] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > rect[data-frame], svg[data-table] > g > rect[data-frame] {
  stroke-width: 70px;
  fill: none;
}

g[data-mml-node="mtable"] > .mjx-dashed, svg[data-table] > g > .mjx-dashed {
  stroke-dasharray: 140;
}

g[data-mml-node="mtable"] > .mjx-dotted, svg[data-table] > g > .mjx-dotted {
  stroke-linecap: round;
  stroke-dasharray: 0,140;
}

g[data-mml-node="mtable"] > g > svg {
  overflow: visible;
}

[jax="SVG"] mjx-tool {
  display: inline-block;
  position: relative;
  width: 0;
  height: 0;
}

[jax="SVG"] mjx-tool > mjx-tip {
  position: absolute;
  top: 0;
  left: 0;
}

mjx-tool > mjx-tip {
  display: inline-block;
  padding: .2em;
  border: 1px solid #888;
  font-size: 70%;
  background-color: #F8F8F8;
  color: black;
  box-shadow: 2px 2px 5px #AAAAAA;
}

g[data-mml-node="maction"][data-toggle] {
  cursor: pointer;
}

mjx-status {
  display: block;
  position: fixed;
  left: 1em;
  bottom: 1em;
  min-width: 25%;
  padding: .2em .4em;
  border: 1px solid #888;
  font-size: 90%;
  background-color: #F8F8F8;
  color: black;
}

foreignObject[data-mjx-xml] {
  font-family: initial;
  line-height: normal;
  overflow: visible;
}

mjx-container[jax="SVG"] path[data-c], mjx-container[jax="SVG"] use[data-c] {
  stroke-width: 3;
}

g[data-mml-node="xypic"] path {
  stroke-width: inherit;
}

.MathJax g[data-mml-node="xypic"] path {
  stroke-width: inherit;
}
mjx-container[jax="SVG"] path[data-c], mjx-container[jax="SVG"] use[data-c] {
							stroke-width: 0;
						} :root {--mermaid-font-zoom:1em ;}  @media print { @page {margin: 0 0 0 0;} body.typora-export {padding-left: 0; padding-right: 0;} #write {padding:0;}}
</style><title>Janicki et al - From Silencing to Gene Expression  Real-Time Analysis in Single Cells</title>
</head>
<body class='typora-export'><div class='typora-export-content'>
<div id='write'  class=''><center><h1>Janicki et al - From Silencing to Gene Expression - Real-Time Analysis in Single Cells</h1></center><p>&nbsp;</p><h2 id='abstract'><span>Abstract</span></h2><ul><li><span>We have developed an inducible system to visualize gene expression at the levels of DNA, RNA and protein in living cells.</span></li><li><span>The system is composed of a 200 copy transgene array integrated into a euchromatic region of chromosome 1 in human U2OS cells.</span></li><li><span>The condensed array is heterochromatic as it is associated with HP1, histone H3 methylated at lysine 9, and several histone methyltransferases.</span></li><li><span>Upon transcriptional induction, HP1 is depleted from the locus and the histone variant H3.3 is deposited suggesting that histone exchange is a mechanism through which heterochromatin is transformed into a transcriptionally active state.</span></li><li><span>RNA levels at the transcription site increase immediately after the induction of transcription and the rate of synthesis slows over time.</span></li><li><span>Using this system, we are able to correlate changes in chromatin structure with the progression of transcriptional activation allowing us to obtain a real-time integrative view of gene expression.</span></li></ul><h2 id='key-points'><span>Key points</span></h2><ul><li><span>Genes are expressed as a result of the concerted processes of transcription, pre-mRNA processing, messen- Efforts to label specific regions of chromatin in vivo have utilized interactions between DNA binding proteins and their target sequences.</span></li><li><span>The affinity of a green fluorescent protein (GFP) glucocorticoid receptor fusion protein for a tandem array of the mouse mammary tumor virus (MMTV) driving a ras reporter was used to identify a region of transcriptionally active chromatin in living cells and to show that transcriptional activators assemble into dynamic complexes at transcription sites.</span></li><li><span>Development of a Cell Line to Study Gene Expression in Living Cells In order to study the dynamics of gene expression in vivo, we constructed a plasmid, containing a transcription unit, that allows the DNA, RNA, and translated protein product to be directly visualized in living cells (Figure 1A).</span></li><li><span>This plasmid, based upon one previously generated in our laboratory, was stably integrated into human U2OS cells. 256 copies of lac operator sequence are present at the 5Ј end of the construct such that when the lac repressor binding protein is expressed as a cyan fluorescent protein (CFP) or a yellow fluorescent protein (YFP) fusion protein (CFPlac repressor/YFP-lac repressor), the integration site/ transgene array can be visualized.</span></li><li><span>Expression of the MS2 coat-YFP fusion protein (MS2-YFP) allows the transcribed RNA to be visualized and the intron/ exon module and cleavage/polyadenylation signal allow the recruitment of the pre-mRNA processing machinery to the transcription site to be studied.</span></li><li><span>When the mRNA is exported from the nucleus and translated in the cytoplasm, the CFP-SKL protein product is targeted to peroxisomes and serves as a reporter confirming that all processes required for gene expression have been successfully completed.</span></li></ul><h2 id='summary'><span>Summary</span></h2><h3 id='introduction-1'><span>Introduction</span></h3><ul><li><span>Genes are expressed as a result of the concerted processes of transcription, pre-mRNA processing, messen-</span></li><li><span>Efforts to label specific regions of chromatin in vivo have utilized interactions between DNA binding proteins and their target sequences.</span></li><li><span>The introduction of bacterial lac operator repeats into the genomes of eukaryotic cells and expression of a green fluorescent protein (GFP) lac repressor fusion protein is a noninvasive means of identifying and studying specific regions of chromatin.</span></li><li><span>Using this approach, the large-scale unfolding of a chromatin structure induced by the VP16 acidic activation domain and the induction of a tetracycline regulatable array of transcription units have been visualized in living cells.</span></li><li><span>Lysine residues in histone tails can be either mono, di or trimethylated and a number of histone H3 K9 methyltransferases (HMTases) have been identified.</span></li></ul><h3 id='results-1'><span>Results</span></h3><ul><li><span>Development of a Cell Line to Study Gene Expression in Living Cells In order to study the dynamics of gene expression in vivo, the authors constructed a plasmid, containing a transcription unit, that allows the DNA, RNA, and translated protein product to be directly visualized in living cells (Figure 1A).</span></li><li><span>This plasmid, based upon one previously generated in the laboratory (Tsukamoto et al, 2000), was stably integrated into human U2OS cells.</span></li><li><span>When the mRNA is exported from the nucleus and translated in the cytoplasm, the CFP-SKL protein product is targeted to peroxisomes and serves as a reporter confirming that all processes required for gene expression have been successfully completed</span></li></ul><h3 id='discussion-1'><span>Discussion</span></h3><ul><li><span>Due to the complexity of the gene expression pathway, the individual processes have long been studied as discrete events.</span></li><li><span>The authors have developed a cell system that allows them to visualize an inducible array of transcription units and their RNA and protein products in living cells and to evaluate dynamic changes in chromatin structure, RNA synthesis, and factor dissociation/association during the induction of transcription.</span></li><li><span>Using this system, the authors have provided significant insight into the dynamic spatial and temporal changes that occur as chromatin transitions from a heterochromatic to a euchromatic state.</span></li><li><span>High-resolution analysis of the locus during transcriptional activation will be required to determine the effects of subchromosomal features on the kinetics of RNA synthesis</span></li></ul><h3 id='conclusion-1'><span>Conclusion</span></h3><ul><li><span>The authors have developed an inducible system to visualize gene expression at the levels of DNA, RNA and protein in living cells.</span></li><li><span>The system is composed of a 200 copy transgene array integrated into a euchromatic region of chromosome 1 in human U2OS cells.</span></li><li><span>The condensed array is heterochromatic as it is associated with HP1, histone H3 methylated at lysine 9, and several histone methyltransferases.</span></li><li><span>RNA levels at the transcription site increase immediately after the induction of transcription and the rate of synthesis slows over time.</span></li><li><span>Using this system, the authors are able to correlate changes in chromatin structure with the progression of transcriptional activation allowing them to obtain a real-time integrative view of gene expression</span></li></ul><p>&nbsp;</p><h2 id='introduction-2'><span>Introduction</span></h2><ul><li><span>Genes are expressed as a result of the concerted processes of transcription, pre-mRNA processing, messenger ribonucleoprotein particle (mRNP) export and translation.</span></li><li><span>Genetic and biochemical analyses have identified a large number of factors required for the execution of these processes, but how their functions are spatially and temporally coordinated is not well understood.</span></li><li><span>Additionally, the transcriptional status of a gene is tightly linked to the structure of its chromatin, but how chromatin proteins are organized and how their dynamics change during the induction of transcription is not well characterized within the context of the living cell.</span></li><li><span>Efforts to label specific regions of chromatin in vivo have utilized interactions between DNA binding proteins and their target sequences</span></li><li><span>The introduction of bacterial lac operator repeats into the genomes of eukaryotic cells and expression of a green fluorescent protein (GFP) lac repressor fusion protein is a noninvasive means of identifying and studying specific regions of chromatin.</span></li><li><span>Using this approach, the large-scale unfolding of a chromatin structure induced by the VP16 acidic activation domain and the induction of a tetracycline regulatable array of transcription units have been visualized in living cells.</span></li><li><span>Additionally, the affinity of a GFP glucocorticoid receptor fusion protein for a tandem array of the mouse mammary tumor virus (MMTV) driving a ras reporter was used not only to identify a region of transcriptionally active chromatin in living cells but also to show that transcriptional activators assemble into dynamic complexes at transcription sites.</span></li><li><span>Gene expression is initiated by transcriptional activators that recruit both ATP-dependent nucleosome remodeling complexes and enzymes that posttranslationally modify histone tail domains.</span></li><li><span>The covalent modification of histone tails (i.e., acetylation, phosphorylation, ubiquitylation, and methylation) is believed to create a “code” that regulates the transcriptional status of chromatin by both modulating nucleosomal structure and promoting and/or preventing the binding of regulatory factors.</span></li><li><span>For example, methylation of histone H3 at lysine 9 (H3 K9), a modification associated with silent chromatin, creates a binding site for the chromodomain of heterochromatin protein 1 (HP1).</span></li><li><span>Lysine residues in histone tails can be either mono, di, or trimethylated and a number of histone H3 K9 methyltransferases (HMTases) have been identified.</span></li><li><span>As biochemical analyses of histone lysine methylation suggest that the modification is permanent in nature and a histone demethylase has not yet been identified, it is not clear how the H3 K9 methyl mark is removed during gene activation.</span></li><li><span>Of particular interest in this regard is the histone H3 variant, H3.3, which has been shown to deposit in active ribosomal DNA genes in a replication-independent manner.</span></li><li><span>Therefore, histone exchange may be a general mechanism through which histone methylation is removed during transitions in the transcriptional state of chromatin.</span></li><li><span>The direct readout of a gene template is produced by the synthesis of the encoded mRNA.</span></li><li><span>Although specific mRNAs can be localized to their transcription sites by RNA fluorescence in situ hybridization (RNA FISH), the kinetics of RNA synthesis at a specific transcription site in a single living cell have not been evaluated.</span></li><li><span>In order to track the movement of specific RNAs in living cells, several studies have utilized the MS2 bacteriophage viral replicase translational operator (a 19 nucleotide RNA stem loop) and a GFP MS2 coat fusion protein.</span></li><li><span>The GFP-MS2 coat protein binds to the stem loop of the translational operator as a dimer and allows the RNA to be visualized.</span></li><li><span>This labeling approach was used to track the asymmetrical movement of ASH1 mRNA in dividing yeast cells, the movement of RNA particles in the cytoplasm of hippocampal neurons and COS cells, and the dynamic localization of endogenous nanos RNA in Drosophila oocytes.</span></li><li><span>Thus far, it has not been used to examine the dynamics of mRNA synthesis at a single specific transcription site.</span></li><li><span>We have developed a cell line that allows us to investigate how gene expression events are coordinated spatially and temporally in vivo.</span></li><li><span>By combining the lac operator/lac repressor, tetracycline inducible, and MS2 translational operator/coat protein systems, we are able to visualize DNA, RNA, and protein in living cells.</span></li><li><span>Using this system, we show that prior to transcriptional activation a stably integrated 200 copy array of inducible transcription units is highly condensed and heterochromatic and we characterize the dynamic loss of HP1 and the recruitment of the histone variant H3.3 during the induction of transcription. </span></li><li><span>Furthermore, we correlate changes in chromatin structure to RNA synthesis and the recruitment of components of the gene expression machinery.</span></li><li><span>This system allows us to obtain a real-time integrative view of gene expression by simultaneously following the dynamics of a specific region of chromatin and its RNA and protein products in living cells.</span></li></ul><h2 id='results-2'><span>Results</span></h2><h3 id='development-of-a-cell-line-to-study-gene-expression-in-living-cells'><span>Development of a Cell Line to Study Gene Expression in Living Cells</span></h3><ul><li><span>In order to study the dynamics of gene expression in vivo, we constructed a plasmid, containing a transcription unit, that allows the DNA, RNA, and translated protein product to be directly visualized in living cells (Figure 1A).</span></li><li><span>This plasmid, based upon one previously generated in our laboratory, was stably integrated into human U2OS cells.</span></li><li><span>256 copies of lac operator sequence are present at the 5 end of the construct such that when the lac repressor binding protein is expressed as a cyan fluorescent protein (CFP) or a yellow fluorescent protein (YFP) fusion protein (CFPlac repressor/YFP-lac repressor), the integration site/ transgene array can be visualized.</span></li><li><span>Immediately downstream, 96 copies of the tetracycline response element (TRE) allow the transcription units to be regulated.</span></li><li><span>When the tetracycline-controlled transactivator (rtTA) (rTetR fused to the VP16 activation domain; pTet-On), is expressed in the presence of doxycycline (dox), it binds to the TREs and activates transcription from the minimal CMV promoter.</span></li><li><span>The transcribed RNA encodes CFP with peroxisomal targeting signal-1 [Ser-Lys-Leu (SKL)] fused to its carboxyl terminus, 24 tandem MS2 translational operators (MS2 repeats), a  </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>-globin intron/exon module, and a cleavage/polyadenylation signal (Figure 1A).E</span></li><li><span>xpression of the MS2 coat-YFP fusion protein (MS2-YFP) allows the transcribed RNA to be visualized and the intron/ exon module and cleavage/polyadenylation signal allow the recruitment of the pre-mRNA processing machinery to the transcription site to be studied.</span></li><li><span>When the mRNA is exported from the nucleus and translated in the cytoplasm, the CFP-SKL protein product is targeted to peroxisomes and serves as a reporter confirming that all processes required for gene expression have been successfully completed.</span></li><li><span>The 20 kb gene expression plasmid ( p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>  ) was cotransfected with the hygromycin resistance plasmid, pTK-Hyg, into U2OS cells and drug resistant colonies were isolated and screened for the presence of a single integration site (visualized by the transient expression of YFP-lac repressor) and the ability to be transcriptionally activated by rtTA expression (+) dox (determined by the presence of CFP-labeled peroxisomes in the cytoplasm).</span></li><li><span>Cell line U2OS 2-6-3 was selected for further characterization because of its easily detectable integration site and the high proportion of transcriptionally activated cells.</span></li><li><span>In order to determine the relative copy number of the integrated plasmid, genomic DNA from U2OS 2-6-3 cells was isolated and used for Southern blotting. This analysis showed that cell line U2OS 2-6-3 contains 200 copies of  p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>  (Figure 1B).</span></li><li><span>As the chromosomal location of transgene loci has been shown to influence their expression as a result of position effects, we were interested in determining the chromosomal integration site of the  p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>  transgene.</span></li><li><span>DNA fluorescence in situ hybridization (DNA FISH) showed that there is a single integration site in an euchromatic region of chromosome 1p36 (Figure 1C).</span></li><li><span>In order to characterize the induction of RNA synthesis upon transcriptional activation, Northern blot analysis was performed after a time-course induction. </span></li><li><span>U2OS 2-6-3 cells were transiently transfected with pTet-On, dox was added 2.5 hr posttransfection and RNA was isolated 0, 5, 10, 15, and 30 min later (Figure 1D).</span></li><li><span>RNA was first detected 10 min after the addition of dox with levels increasing dramatically between the 15 and 30 min time points.</span></li><li><span>The fact that RNA transcripts were not detected at 0 min demonstrates that transcription is not “leaky” when pTet-On is expressed in the absence of dox.</span></li><li><span>At the 10 min time point, there was slightly more unspliced RNA compared to spliced mRNA (1.0: 0.9375) (Figure 1D).</span></li><li><span>Over time, the spliced product predominated (15 min, 1.3-fold increase; 30 min  3-fold increase).</span></li><li><span>As the level of the spliced RNA increased dramatically between 15 and 30 min postinduction, this suggests that the changes that occur at the locus between these time points increase the efficiency of both transcription and pre-mRNA processing.</span></li><li><span>This lag time may reflect the time required for the chromatin remodeling machinery to decondense the chromatin and hence make more of the transcription units accessible in addition to the time required for the pre-mRNA processing machinery to be recruited to the transcription site and the splicing reaction to occur.</span></li><li><span>At later time points, both higher and lower molecular weight species of RNA were detected (Supplemental Data, Supplemental Figure S1 683/DC1; 120 min) suggesting that aberrant products may be produced when transcription is elevated to high levels from a strong promoter.</span></li><li><span>Immunoblotting of protein lysates collected between 0 and 6 hr after the induction of transcription showed that the CFP-SKL protein can be detected at the 1 hr time point with expression levels increasing steadily over time (Figure 1E).</span></li></ul><h3 id='visualization-of-gene-expression-and-the-recruitment-of-the-gene-expression-machinery-to-a-transcription-site'><span>Visualization of Gene Expression and the Recruitment of the Gene Expression Machinery to a Transcription Site</span></h3><ul><li><span>To simultaneously analyze chromatin and RNA dynamics at a transcription site in individual cells, U2OS 2-6-3 cells were transiently cotransfected with CFP-lac repressor, rtTA and MS2-YFP and visualized 2.5 hr posttransfection.</span></li><li><span>In the absence of dox, the chromatin of the transgene locus, marked by CFP-lac repressor, appears as a highly condensed small dot (Figure 2A) (0.89  0.12  m in diameter; n 13) that is DAPI dense (data not shown).</span></li><li><span>MS2-YFP is diffusely distributed in a smooth pattern throughout the nucleoplasm without any particular enrichment at the locus (Figure 2B).</span></li><li><span>Interestingly, mRNA transcripts from the constitutively active hygromycin resistance plasmid (pTK-Hyg) were detected at the condensed locus by RNA FISH (Supplemental Figure S2) confirming that pTK-Hyg integrated into the array with the gene expression plasmid (p3216PECMS2  ).</span></li><li><span>As pTK-Hyg transcripts were generally on the periphery of the locus, it is possible that it integrated throughout the array but that only those copies accessible to the transcription machinery are transcribed.</span></li><li><span>Although pTKHyg is constitutively active, the locus is condensed when p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span> is inactive suggesting that the higher order structure of the array is dictated by the transcriptional state of the construct in the higher proportion (9:1).</span></li><li><span>When transcription is induced by the addition of dox to the medium and cells examined 2.5 hr later, the locus appears decondensed and CFP-SKL is localized to the cytoplasmic peroxisomes (Figure 2D).</span></li><li><span>Decondensed loci range in size from 2.0–4.5m in diameter with the variability likely resulting from the transient expression of rtTA.</span></li><li><span>Additionally, MS2-YFP concentrates at the decondensed locus and is seen in a particulate pattern throughout the nucleoplasm that is consistent with the transcribed RNAs being packaged into mRNPs (Figure 2E).</span></li><li><span>RNA in situ hybridization with a probe that recognizes sequence in the  </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>-globin intron confirmed that nascent transcripts are present at the site of the decondensed locus (Supplemental Figure S3).</span></li><li><span>In order to visualize the association of the transcriptional activator with the lac repressor, we expressed CFP-lac repressor (pseudocolored red) and rtTA as a YFP fusion protein (pEYFP-rtTA-N1) (pseudocolored green) (Figures 2G–2L) and collected and deconvolved image stacks 5 and 60 min after the addition of dox.</span></li><li><span>Though the locus is still compact 5 min postinduction, (Figures 2G–2I, single sections from deconvolved image stacks), the regions bound by CFP-lac repressor and rtTA-YFP are easily distinguishable and minimally overlapping.</span></li><li><span>In fact, rtTA-YFP only binds to a region of the condensed locus suggesting that a small subset of the repeated templates are transcribed at early time points—a result consistent with the low levels of RNA detected in the northern analysis (Figure 1D).</span></li><li><span>In a cell fixed 60 min postinduction (Figures 2J–2L), the locus is highly decondensed and CFP-lac repressor and rtTAYFP also appear in distinct foci that partially overlap.</span></li><li><span>As we did not observe 200 foci at the decondensed locus, this suggests that the transcription units assemble into repetitive substructures within the array or that the array does not fully decondense.</span></li><li><span>We next examined the recruitment of factors required for transcription and pre-mRNA processing (Supplemental Figure S4 immunoblot of constructs).</span></li><li><span>YFP-RNA polymerase II (Figures 2M-2O), the premRNA splicing factor YFP-SF2/ASF (Figures 2P–2R) and the 3 end processing factor Cstf64 (Figures 2S–2U) all colocalized with the decondensed locus confirming that gene expression factors from different nuclear domains are recruited to the transcription site.</span></li></ul><h3 id='the-kinetics-of-chromatin-decondensation-and-rna-synthesis'><span>The Kinetics of Chromatin Decondensation and RNA Synthesis</span></h3><ul><li><span>We were next interested in evaluating the kinetics of RNA synthesis in individual cells during transcriptional activation.</span></li><li><span>Cells were transiently cotransfected with CFP-lac repressor, MS2-YFP, and rtTA, and 2 hr posttransfection placed in a FCS2 live-cell chamber (37</span>
<span>C) and visualized.</span></li><li><span>Medium containing dox was perfused into the chamber 0.5 hr later, and cells were imaged every 2.5 min for 4–6 hr in both the YFP and CFP channels (Supplemental Movie S1).</span></li><li><span>An increase in RNA levels was detected immediately after the addition of dox (i.e., Figure 3E, 5 min; 3H, 7.5 min; 3P).</span></li><li><span>The highly condensed state of the chromatin at the early time points suggests that only a subset of the repeats are being transcribed which is supported by the limited amount of rtTA-YFP detected at the locus 5 min post dox (Figure 2H).</span></li><li><span>By 17.5 min when noticeable changes in higher order chromatin structure begin to be detected (Figures 3J–3L), the RNA coats the entire region of the locus.</span></li><li><span>By 130 min when RNA levels at the transcription site are at their peak (Figure 3P), the locus is maximally decondensed and CFP-SKL is concentrated in the peroxisomes (Figures 3M–3O).</span></li><li><span>The CFP-SKL protein can first be detected in the cytoplasmic peroxisomes 1 hr after the addition of dox (data not shown).</span></li><li><span>By measuring RNA levels at the transcription site over time, we found that they increase, peak, and subsequently decrease (Figure 3P).</span></li><li><span>Modeling of this data showed that the rate of increase slows with time suggesting that increases in RNA production from this transgene array inhibit further synthesis (Figure 3P).</span></li><li><span>A delay in the increase was also often observed before the peak.</span></li><li><span>It is possible that this delay reflects the slower decondensation of a region of chromatin within the array that forms a specific substructure.</span></li><li><span>Though we found that there is variability in the timing of these features between cells, likely resulting from the transient expression of rtTA, the general features are consistent (data not shown).</span></li></ul><h3 id='the-condensed-locus-is-heterochromatic'><span>The Condensed Locus is Heterochromatic</span></h3><ul><li><span>As the ~200 copies of the gene expression plasmid formed a highly condensed structure in interphase cells, we also wanted to characterize the locus in this state.</span></li><li><span>When U2OS 2-6-3 cells expressing YFP-lac repressor were independently labeled with antibodies to HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span>,  </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span> , and </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span> , all three isoforms were detected at the condensed locus (Supplemental Figure S5).</span></li><li><span>Similarly, YFP-HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span>, </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>  and </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span> fusion proteins colocalized with CFP-lac repressor (Figures 4A–4I; Supplemental Figure S4; Western blot of constructs).</span></li><li><span>The association of HP1 with the condensed locus suggested that the histone H3 might be methylated at lysine 9 as this modification serves as a binding site for HP1.</span></li><li><span>When cells were labeled with an antibody made to a branched histone H3 peptide methylated at lysine 9 which had been affinity purified for antibodies with a preference for trimethylated H3 lysine 9 (H3 tri-meK9), this modification was detected at the condensed locus (Figures 4J–4L) but not at decondensed loci in transcriptionally activated cells (Figures 4M–4O, 2.5 hr postinduction).</span></li><li><span>As we found all three HP1 isoforms and histone H3 tri-meK9 at the inactive locus, we also wanted to see if we could visualize an association with H3 K9 HMTases.</span></li><li><span>When we expressed Suv39h1, a H3 K9-specific HMTase associated with constitutive heterochromatin as a YFP fusion protein, it colocalized with CFPlac repressor (Figures 4P–4R; Supplemental Figure S4, Western blot).</span></li><li><span>Additionally, YFP-G9a-L, the longer G9a splice variant shown to methylate both H3 K9 and K27 in vitro and to localize to euchromatin, colocalized with CFP-lac repressor (Figures 4S4U; Supplemental Figure S4, Western blot).</span></li></ul><h3 id='hp1-isoforms--h3-k9-methylation-and-euhmtase-1-differentially-associate-with-the-gene-expression-plasmid-by-chip-analysis'><span>HP1 Isoforms , H3 K9 Methylation, and EuHMTase 1 Differentially Associate with the Gene Expression Plasmid by ChIP Analysis</span></h3><ul><li><span>As we were able to visually detect HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> and </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span> H3 tri-meK9 at the condensed locus, we wanted to analyze the specific associations of HP1 and  , H3 di- and tri-meK9, and H3 K9 HMTases by chromatin immunoprecipitation (ChIP) (Figure 5).</span></li><li><span>Strikingly, HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span>  displayed a distinct preference for the promoter and the transcribed regions of the gene in contrast to HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span>, which was only found downstream on the bacterial plasmid sequence (Figures 5B and 5C).</span></li><li><span>H3 tri-meK9 was present throughout the assayed regions while di-meK9 was enriched on the bacterial plasmid sequence and at the beginning of the intron/exon module in a region immediately after the MS2 repeats (Figures 5B and 5C).</span></li><li><span>Interestingly, euchromatic histone methyltransferase1 (Eu-HMTase 1) colocalized with the H3 K9 dimethylated regions of the gene expression plasmid. As G9a has been reported to be a H3 K9 di-HMTase and Eu-HMTase1 is 63% homologous to G9a and possesses the same basic structure, this suggests that Eu-HMTase1 is the H3 K9 di-HMTase responsible for the pattern of H3 K9 di-methylation at the locus.</span></li></ul><h3 id='hp1\nα\n-is-depleted-from-the-locus-during-transcriptional-activation'><span>HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> Is Depleted from the Locus during Transcriptional Activation</span></h3><ul><li><span>As HP1 is a protein critical to the stability of transcriptional silencing, we wanted to examine the kinetics of the association of YFP-HP1 with the locus during transcriptional activation and to relate these dynamics to changes in chromatin structure (Supplemental Movie S2).</span></li><li><span>YFPHP1 was detected at the locus before the addition of dox (Figures 6A–6C; 0 min) and during the early time points postinduction (Figures 6D–6F; 17.5 min).</span></li><li><span>At the 30 min time point when significant chromatin decondensation had occurred (Figure 6G), YFP-HP1 was seen in punctate structures which were possibly regions of chromatin not yet remodeled by transcriptional activity (Figure 6H).</span></li><li><span>By 50 min, accumulations of YFP-HP1 could no longer be detected on the locus (Figures 6K–6L) and between 100–140 min, YFP-HP1 became progressively depleted from this region (data not shown) with levels subsequently decreasing further over time.</span></li><li><span>The presence of this dark region, depleted of YFP-HP1 below background levels (Figures 6M–6O; 180 min), suggests that HP1 binding sites are no longer present at the transcriptionally active locus.</span></li></ul><h3 id='the-histone-h3-variant-h33-is-deposited-at-the-locus-during-transcriptional-activation'><span>The Histone H3 Variant, H3.3, Is Deposited at the Locus during Transcriptional Activation</span></h3><ul><li><span>As YFP-HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> was dynamically depleted during the induction of transcription and we could not detect the histone H3 tri-MeK9 modification at transcriptionally active loci (2.5 hr postinduction), we were interested in understanding how this modification is removed from the chromatin during transcriptional activation.</span></li><li><span>A histone demethylase has not yet been identified but the histone H3 variant, H3.3, has been shown to be deposited into active ribosomal DNA in a replication-independent manner.</span></li><li><span>Therefore, we were interested in determining whether histone exchange is a general mechanism through which a heterochromatic region is transformed into the active state.</span></li><li><span>Before the induction of transcription, H3.3-YFP did not show any specific deposition at the locus (Figures 7A–7C; Supplemental Movie S3).</span></li><li><span>A small spot of H3.3-YFP was seen, however, ~7.5 min after the addition of dox adjacent to the region marked by CFP-lac repressor (pseudocolored red) (Figures 7D–7F).</span></li><li><span>Significant incorporation of H3.3-YFP appeared around the periphery of the locus (Figures 7G–7I; 40 min) and subsequently progressed into the interior (Figures 7J–7L; 75 min).</span></li><li><span>Over time, H3.3-YFP deposition became highly concentrated at the locus and did not completely overlap with the regions marked by CFP-lac repressor (Figures 7M–7O) suggesting that deposition may specifically occur in the regions associated with the transcription machinery.</span></li><li><span>Nucleoli were also labeled by H3.3-YFP, as confirmed by double labeling with an antibody to fibrillarin (data not shown), which is consistent with the deposition of H3.3 into rDNA.</span></li><li><span>Therefore, histone exchange appears to be part of the mechanism by which heterochromatin can be converted to active chromatin.</span></li></ul><h2 id='discussion-2'><span>Discussion</span></h2><ul><li><span>Due to the complexity of the gene expression pathway, the individual processes have long been studied as discrete events</span></li><li><span>However, as the interconnectedness of chromatin remodeling, transcription, mRNA processing, and mRNP export becomes increasingly apparent, it is necessary to understand how they are coordinated in vivo.</span></li><li><span>We have developed a cell system that allows us to visualize an inducible array of transcription units and their RNA and protein products in living cells and thus, to evaluate dynamic changes in chromatin structure, RNA synthesis, and factor dissociation/association during the induction of transcription.</span></li><li><span>Using this system, we have provided significant insight into the dynamic spatial and temporal changes that occur as chromatin transitions from a heterochromatic to a euchromatic state.</span></li><li><span>This system has extensive potential to address a broad range of questions relating to gene expression, DNA replication, and chromatin stability.</span></li></ul><h3 id='mrna-dynamics-at-a-transcription-site'><span>mRNA Dynamics at a Transcription Site</span></h3><ul><li><span>We have been able to evaluate the dynamics of RNA synthesis in real-time by measuring MS2-YFP levels at the site of transcription.</span></li><li><span>Thus far, mRNA levels at a single specific transcription site have not been evaluated over time in living cells although the diffusive properties of the total endogenous population of nuclear poly(</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="3.129ex" height="1.754ex" role="img" focusable="false" viewBox="0 -775.2 1383.1 775.2" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: 0px;"><defs><path id="MJX-54-TEX-I-1D434" d="M208 74Q208 50 254 46Q272 46 272 35Q272 34 270 22Q267 8 264 4T251 0Q249 0 239 0T205 1T141 2Q70 2 50 0H42Q35 7 35 11Q37 38 48 46H62Q132 49 164 96Q170 102 345 401T523 704Q530 716 547 716H555H572Q578 707 578 706L606 383Q634 60 636 57Q641 46 701 46Q726 46 726 36Q726 34 723 22Q720 7 718 4T704 0Q701 0 690 0T651 1T578 2Q484 2 455 0H443Q437 6 437 9T439 27Q443 40 445 43L449 46H469Q523 49 533 63L521 213H283L249 155Q208 86 208 74ZM516 260Q516 271 504 416T490 562L463 519Q447 492 400 412L310 260L413 259Q516 259 516 260Z"></path><path id="MJX-54-TEX-N-2B" d="M56 237T56 250T70 270H369V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T707 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 250Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="msup"><g data-mml-node="mi"><use data-c="1D434" xlink:href="#MJX-54-TEX-I-1D434"></use></g><g data-mml-node="mo" transform="translate(783,363) scale(0.707)"><use data-c="2B" xlink:href="#MJX-54-TEX-N-2B"></use></g></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><msup><mi>A</mi><mo>+</mo></msup></math></mjx-assistive-mml></mjx-container><script type="math/tex">A^+</script><span>) RNA have been studied using fluorescently tagged oligodeoxynucleotides labeled with caged flurochromes and fluorescence correlation spectroscopy and by examining the dynamics of mRNP proteins.</span></li><li><span>We have observed that RNA levels begin increasing immediately after the induction of transcription, peak, and subsequently decrease.</span></li><li><span>A delay in the increase was often seen at a time point prior to the peak.</span></li><li><span>It is possible that this delay reflects the decondensation of a substructure of the locus exposed after the initial large-scale unfolding.</span></li><li><span>Interestingly, independently isolated clones containing an amplified region of DHFR cDNA labeled with lac operator repeats showed distinct but reproducible arrangements of chromosome segments within the metaphase chromatid axis suggesting that different chromatin regions acquire distinct folding characteristics.</span></li><li><span>In the case of our locus, the juxtaposition of the plasmids within the array could create different substructures that exhibit distinct activation profiles.</span></li><li><span>High-resolution analysis of the locus during transcriptional activation will be required to determine the effects of subchromosomal features on the kinetics of RNA synthesis.</span></li><li><span>Modeling the rate of increase of RNA at this transgene array locus upon the induction of transcription showed that it slows with time, suggesting that as more RNA is synthesized it may have a negative feedback effect on further transcription.</span></li><li><span>A similar trend was seen in both a cell line containing a tandem array of ras reporters driven by MMTV promoters activated by the glucocorticoid receptor and at the endogenous  </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>-actin gene upon serum induction.</span></li><li><span>By run-on assay, transcription peaked in the MMTV reporter cell line within the first 2 hr after hormone treatment and declined by 6 hr.</span></li><li><span>The arrays recondensed 3–8 hr after induction with array size correlated to decreasing amounts of transcript as determined by RNA FISH analysis.</span></li><li><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>-actin RNA levels peaked 15 min after serum induction after which they diminished until the 1.5 hr time point when they were indistinguishable from those seen before induction.</span></li><li><span>Although we do not know the mechanism(s) through which the rate of transcription is slowed in our cell line, it is possible that the accumulation of RNA at the transcription site sterically hinders the transcription machinery.</span></li><li><span>A number of pathways that monitor mRNA quality in the nucleus have been identified and mRNA that is too slowly or improperly spliced, polyadenylated, or transported is retained at the transcription site and degraded in the nucleus.</span></li><li><span>Northern blotting of RNA from the 120 min time point (Supplemental Figure S1) identified species with molecular weights both higher and lower than the unspliced and spliced products.</span></li><li><span>It is possible that they are truncated, degraded, or readthrough products produced when high levels of RNA are transcribed from the strong CMV promoter. </span></li><li><span>Although we do not know the cellular location of these aberrant transcripts, it is possible that they are retained at the transcription site and affect the rate of transcription. In the case of the  </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>-actin gene, transcription was shown to be inhibited at initiation.</span></li><li><span>The mechanism(s) through which the cell regulates the transcription cycle of a gene is not well understood, but the emerging importance of RNA as a signaling molecule suggests that understanding the kinetics of its production may provide insight not only into transcriptional regulation but possibly also into chromatin structure.</span></li></ul><h3 id='characterization-and-dynamics-of-the-condensed-locus-and-its-transition-to-the-active-state'><span>Characterization and Dynamics of the Condensed Locus and Its Transition to the Active State</span></h3><ul><li><span>We have shown that the 200 copies of the gene expression plasmid that integrated into the genome of the U2OS cell line formed a heterochromatic structure.</span></li><li><span>As transcription can be activated at this array in an experimentally regulated manner, we were able to study in living cells the dynamic changes that occur as heterochromatin transits to the active state.</span></li><li><span>By examining YFPHP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> dynamics during the induction of transcription, we showed that it is depleted from the locus as it decondenses suggesting that H3 K9 methylation is removed during gene activation.</span></li><li><span>We also found that the histone H3 variant, H3.3, is deposited during activation.</span></li><li><span>Therefore, histone exchange is likely the mechanism through which H3 K9 methylation is removed.</span></li><li><span>By 50 min postinduction, there are no longer accumulations of YFPHP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> at the locus but because YFPHP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span>  does not become depleted until 100–140 min, this suggests that at least a population of histone H3 is still methylated at K9 prior to these time points.</span></li><li><span>As we see RNA coating the decondensing chromatin 20–25 min postinduction, it is possible that transcription can occur in the presence of HP1 and H3 K9 trimethylation.</span></li><li><span>It is also possible that these silencing marks are removed immediately from specific sequence elements (i.e., promoters) upon induction and that we do not have the resolution to detect these changes or to determine which of the multiple repeats are activated first.</span></li><li><span>Interestingly, HP1 has been shown to be required for expression of genes located in heterochromatin in Drosophila and to be recruited to heat shock and ecdysone-activated puffs on polytene chromosomes.</span></li><li><span>Future studies are required to address how the activational and repressive roles of HP1 may be modulated and coordinated in our system.</span></li><li><span>As significant deposition of H3.3 at the locus is not seen until later time points (75–150 min postinduction), it is possible that histone exchange is a late event in the transition of heterochromatin to the active stateperhaps playing more of a role in changing and maintaining the epigenetic state of a gene than in aiding in the progression of transcription.</span></li><li><span>It is also possible that in order for transcription to begin, histone exchange must first occur in specific sequence elements, such as promoters, and that we are not able to detect deposition in such small regions.</span></li><li><span>Subsequently, histones may be exchanged throughout the entire coding region resulting in the significant accumulation visualized at later time points.</span></li><li><span>As H3.3-YFP did not completely colocalize with CFP-lac repressor (Figures 7M–7O), it is likely that exchange occurs specifically in actively transcribed areas.</span></li><li><span>Heterochromatin has been reported to have both dynamic and stable qualities.</span></li><li><span>Its high degree of condensation is believed to contribute to the inhibition of transcription by preventing transcription factors from accessing their binding elements yet it is not intractable to transcriptional activation.</span></li><li><span>Photobleaching studies have shown HP1 to be a dynamic component of heterochromatin in contrast to the core histones, which show little turnover.</span></li><li><span>Likewise, biochemical analyses of histone methylation suggest that this modification is highly stable.</span></li><li><span>Therefore, the combination of both dynamic and durable elements may be what makes heterochromatin simultaneously stable and flexible.</span></li><li><span>As histone H3 has been shown to be tightly associated with chromatin, it will be of particular interest to characterize the histone chaperones and chromatin remodeling proteins that function to remove H3 and exchange it for H3.3 as they may be important regulators of epigenetic states.</span></li><li><span>In fact, the histone chaperone HIRA was recently shown to be required for DNA synthesisindependent nucleosome assembly and Swr1, a Swi2/Snf2 related ATPase was identified in a complex that catalyzes the exchange of H2A with H2AZ in nucleosome arrays.</span></li><li><span>Interestingly, we found, by ChIP analysis, that HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> and </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span> , di-and trimethyl H3 K9 and Eu-HMTase1 are differentially localized along the inactive gene expression plasmid. It is unlikely that the HP1 isoforms associate in a sequence specific manner, as they have not been reported to show any DNA motif preferences.</span></li><li><span>The distinct associations of HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> and </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span> with the chromatin are, therefore, more likely determined by complex interactions with other factors involved in establishing and maintaining heterochromatin.</span></li><li><span>It has been suggested that HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span>  associates with euchromatin while HP1 associates with constitutive heterochromatin.</span></li><li><span>As the bacterial plasmid sequence is composed of elements that are noneuchromatic as well as noneukaryotic, it may therefore be recognized as constitutive heterochromatin by HP1.</span></li><li><span>Likewise, the promoter, the coding region and the intron/ exon module may recruit HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span>  because they are recognized as euchromatic elements.</span></li><li><span>Interestingly, there was a strong coincidence between H3 di-meK9 and Eu-HMTase1 suggesting that EuHMTase1 is the histone H3 HMTase that dimethylates the locus.</span></li><li><span>As Suv39h1 and G9a-L were also detected, more than one H3 K9 HMTase seems to be responsible for the H3 K9 methylation patterns suggesting that the regulation of the chromatin at the condensed locus is complex.</span></li><li><span>We also observed that the association of HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span>  is inversely related to the H3 di-meK9 and Eu-HMTase1 patterns as it is not detected in the downstream region of the bacterial plasmid sequence and is reduced in the region immediately after the MS2 repeats which is another noneukaryotic sequence element.</span></li><li><span>As Eu-HMTase1 was originally isolated in a complex with E2F-6 that included HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span>  but not HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> and </span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span> , it seems that that Eu-HMTase1 does not preferentially associate with a specific HP1 isoform or alone specify associations of the HP1 isoforms with DNA sequences.</span></li><li><span>Therefore, other as yet unidentified factors may be antagonistic to HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.229ex" height="1.486ex" role="img" focusable="false" viewBox="0 -441 543 657" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.489ex;"><defs><path id="MJX-84-TEX-I-1D6FE" d="M31 249Q11 249 11 258Q11 275 26 304T66 365T129 418T206 441Q233 441 239 440Q287 429 318 386T371 255Q385 195 385 170Q385 166 386 166L398 193Q418 244 443 300T486 391T508 430Q510 431 524 431H537Q543 425 543 422Q543 418 522 378T463 251T391 71Q385 55 378 6T357 -100Q341 -165 330 -190T303 -216Q286 -216 286 -188Q286 -138 340 32L346 51L347 69Q348 79 348 100Q348 257 291 317Q251 355 196 355Q148 355 108 329T51 260Q49 251 47 251Q45 249 31 249Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FE" xlink:href="#MJX-84-TEX-I-1D6FE"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>γ</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\gamma</script><span>  binding.</span></li><li><span>Additionally, the relationship between the strong peaks of H3 K9 dimethylation and HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span> on the bacterial plasmid sequence is not clear.</span></li><li><span>H3 di-meK9 has been reported to localize in a pattern consistent with euchromatin and constitutive heterochromatin is characteristic of HP1</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.448ex" height="1.025ex" role="img" focusable="false" viewBox="0 -442 640 453" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.025ex;"><defs><path id="MJX-21-TEX-I-1D6FC" d="M34 156Q34 270 120 356T309 442Q379 442 421 402T478 304Q484 275 485 237V208Q534 282 560 374Q564 388 566 390T582 393Q603 393 603 385Q603 376 594 346T558 261T497 161L486 147L487 123Q489 67 495 47T514 26Q528 28 540 37T557 60Q559 67 562 68T577 70Q597 70 597 62Q597 56 591 43Q579 19 556 5T512 -10H505Q438 -10 414 62L411 69L400 61Q390 53 370 41T325 18T267 -2T203 -11Q124 -11 79 39T34 156ZM208 26Q257 26 306 47T379 90L403 112Q401 255 396 290Q382 405 304 405Q235 405 183 332Q156 292 139 224T121 120Q121 71 146 49T208 26Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FC" xlink:href="#MJX-21-TEX-I-1D6FC"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>α</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\alpha</script><span>.</span></li><li><span>These results, therefore, suggest that there are no simple rules to explain how heterochromatin proteins and modifications are targeted to DNA sequences.</span></li><li><span>Perhaps the high degree of complexity revealed in the studies of heterochromatin structure and regulation suggest that cells employ multiple complex interactions in order to package the diverse range of sequence elements in the eukaryotic genome such that heritable gene expression patterns are ensured.</span></li></ul><h3 id='heterochromatin-and-transgene-arrays'><span>Heterochromatin and Transgene Arrays</span></h3><ul><li><span>When the 200 copies of the gene expression plasmid integrated into a euchromatic region of the U2OS cell genome, they formed a structure that is highly condensed in interphase cells and heterochromatic.</span></li><li><span>Many regions of constitutive heterochromatin, such as centromeres, telomeres, and transposons, are composed of repetitive sequence elements.</span></li><li><span>In addition, multicopy transgene arrays as small as two copies inserted into euchromatic regions of the genome are often silenced, DNA methylated and, in Drosophila polytene chromosomes, seen as DAPI dense structures enriched with HP1</span></li><li><span>Transgene silencing is increased in arrays with inverted repeats which are likely a potent source of double-stranded RNA (dsRNA) and small interfering RNAs (siRNAs) generated by the RNA interference pathway (RNAi) have been detected in Drosophila cells with transgene arrays.</span></li><li><span>RNAi is also involved in the establishment and maintenance of heterochromatin.</span></li><li><span>Therefore, it will be interesting to look for the presence of siRNAs and inverted repeats in 2-6-3 cells. As we have a cell line in which we can visualize the conversion of a region of heterochromatin to a transcriptionally active state, it may be a useful system in which to study the mechanisms of RNAi-based gene silencing in vivo.</span></li><li><span>To date, it is not well understood how primary DNA sequence content regulates higher order chromatin structure, how distinct functional domains are established and how these structures regulate gene expression.</span></li><li><span>Repetitive arrays occur naturally in the human genome and include transposons, ribosomal DNA, centromeres, and telomeres.</span></li><li><span>Recently, a reduction in the number of copies of a repetitive element at 4q35 below a certain threshold was shown to cause Facioscapulohumeral muscular dystrophy (FSHD) with copy number a critical determinant of the age of onset as well as the clinical severity of the disease.</span></li><li><span>Therefore, an understanding of the packaging and regulation of repetitive elements in the genome will provide important insight into the mechanisms of both gene regulation and disease.</span></li></ul><h2 id='conclusion-2'><span>Conclusion</span></h2><ul><li><span>We have developed an inducible system to visualize gene expression at the levels of DNA, RNA and protein in living cells</span></li><li><span>The system is composed of a 200 copy transgene array integrated into a euchromatic region of chromosome 1 in human U2OS cells</span></li><li><span>The condensed array is heterochromatic as it is associated with HP1, histone H3 methylated at lysine 9, and several histone methyltransferases</span></li><li><span>Upon transcriptional induction, HP1␣ is depleted from the locus and the histone variant H3.3 is deposited suggesting that histone exchange is a mechanism through which heterochromatin is transformed into a transcriptionally active state</span></li><li><span>RNA levels at the transcription site increase immediately after the induction of transcription and the rate of synthesis slows over time</span></li><li><span>Using this system, we are able to correlate changes in chromatin structure with the progression of transcriptional activation allowing us to obtain a real-time integrative view of gene expression.</span></li></ul><p>&nbsp;</p><center><h1>Figures</h1></center><p><img src="https://39363.org/IMAGE_BUCKET/1632357893864-467639335.jpeg" style="zoom:67%;" /></p><center><b></b>Figure 1. Characterization of Human U2OS 2-6-3 Cells</center><ul><li><p><span>(A) Schematic representation of the gene expression plasmid, p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span> .</span></p><ul><li><span>The plasmid is composed of 256 copies of the lac operator, 96 tetracycline response elements, a minimal CMV promoter, CFP fused to the peroxisomal targeting signal SKL, 24 MS2 translational operators (MS2 repeats), a rabbit globin intron/exon module, and a cleavage/polyadenylation signal. </span></li><li><span>Expression of CFP-lac repressor allows the DNA to be visualized and expression of pTet-On (rtTA) in the presence of doxycycline (dox) drives expression from the CMV minimal promoter.</span></li><li><span>When MS2-YFP (YFP fused to the MS2 coat protein) dimerizes and interacts with the stem loop structure of the translational operator, it allows the transcribed RNA to be visualized.</span></li></ul></li><li><p><span>(B) Quantitative Southern blot of clone 2-6-3 genomic DNA.</span></p><ul><li><span>A 2.4 kb fragment is produced when clone 2-6-3 genomic DNA and p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>  are digested with NcoI which cuts at the 5 end of CFP and within the globin intron.</span></li><li><span>Comparison of known quantities of plasmid DNA equal to 100, 200, 300, and 400 copies per cell showed that 2-6-3 cells contain 200 stably integrated copies of p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script><span>.</span></li></ul></li><li><p><span>(C) DNA fluorescence in situ hybridization (DNA FISH) of 2-6-3 cells shows that there is a single integration site in the euchromatic region of chromosome 1p36.</span></p></li><li><p><span>(D) Northern blot time course analysis of RNA isolated 0, 5, 10, 15, and 30 min after the induction of transcription.</span></p><ul><li><span>The last lane shows a lighter exposure of the 30 min time point.</span></li><li><span>Pre-mRNA transcripts run at 3.4 kb and spliced mRNA at 2.8 kb.</span></li><li><span>The probe recognizes the MS2 repeats.</span></li><li><span>Actin was probed as a loading control.</span></li></ul></li><li><p><span>(E) Immunoblot time course analysis of CFPSKL expression 0, 1, 2, 3, 4, 5, and 6 hr after the addition of doxycycline.</span></p></li></ul><p>&nbsp;</p><p><img src="https://39363.org/IMAGE_BUCKET/1632358051282-97344529.jpeg" style="zoom:67%;" /></p><center>Visualization of DNA, RNA, and Protein in Living Cells</center><ul><li><p><span>(A–F) U2OS 2-6-3 cells were transiently transfected with pSV2-CFP-lac repressor, pTet-ON (rtTA) and MS2-YFP, and imaging was begun 2.5 hr posttransfection.</span></p></li><li><p><span>(A–C) At 0 min () dox, CFP-lac repressor marks the locus (A) and MS2-YFP is diffusely distributed throughout the nucleus (B).</span></p></li><li><p><span>(D–F) 2.5 hr after the addition of Dox, the locus is highly decondensed and CFP-SKL is seen in the cytoplasmic peroxisomes (D).</span></p><ul><li><span>MS2-YFP accumulates at the site of the decondensed locus and is present in a particulate pattern throughout the nucleoplasm (E).</span></li></ul></li><li><p><span>(G–L) Image stacks of cells expressing pSV2-CFP-lac repressor (pseudocolored red) and EYFP-rtTA-N1 (pseudocolored green) were collected and deconvolved in cells fixed 5 min (G–I) and 60 min (J–L) after the induction of transcription.</span></p><ul><li><span>Single sections from deconvolved stacks are shown.</span></li></ul></li><li><p><span>(M–U) Factors involved in gene expression colocalize with the decondensed locus.</span></p><ul><li><span>YFP-RNA polymerase II (M–O), YFP-SF2/ASF (P–R), and Cstf64 (S–U) are present at the active locus. Scale bar is equal to 5m.</span></li><li><span>Scale bar in enlarged insets is equal to 1m.</span></li></ul></li></ul><p>&nbsp;</p><p><img src="https://39363.org/IMAGE_BUCKET/1632358228393-469014086.jpeg" style="zoom:67%;" /></p><center>Kinetics of RNA Synthesis</center><ul><li><p><span>(A–O) Still images from a time series of U2OS 2-6-3 cells during transcriptional activation showing the relationship between the chromatin of the locus, marked by CFP-lac repressor, and the RNA, marked by MS2-YFP.</span></p><ul><li><span>Also see Supplemental Data and Supplemental Movie S1 available on Cell website. Scale bar is equal to 5  m.</span></li></ul></li><li><p><span>(P) Quantitative analysis and modeling of RNA levels at the locus.</span></p><ul><li><span>The intensity of the MS2YFP signal at the locus as it associates with the synthesized RNA was measured every 2.5 min.</span></li><li><span>There is an initial linear increase after the induction of transcription which slows over time (following the form A</span><em><span>(1-exp(-B</span></em><span>t))).</span></li><li><span>The fit shows a deviation from this increase at 140 min after which a decrease predominates.</span></li><li><span>There is a delay in the increase at 50 min.</span></li></ul></li></ul><p><img src="https://39363.org/IMAGE_BUCKET/1632358309100-494947238.jpeg" style="zoom:67%;" /></p><center>Figure 4 - Characterization of the Condensed Heterochromatic Locus</center><ul><li><span>YFP-HP1 (A-C), YFP-HP1  (D-F), and YFPHP1 (G–I) colocalize with the condensed locus, marked by CFP-lac repressor and the histone H3 is trimethylated on lysine 9 (H3 trimeK9) (J-L).</span></li><li><span>The H3 lysine 9 modification is not detected after the induction of transcription (M–O; 2.5 hr postdox).</span></li><li><span>The histone H3 K9 methyltransferases YFP-Suv39h1 (P–R) and YFP-G9a-L (S–U) are present at the condensed locus.</span></li><li><span>Scale bar is equal to 5  m.</span></li></ul><p><img src="https://39363.org/IMAGE_BUCKET/1632358374906-18796627.jpeg" style="zoom:67%;" /></p><center>Figure 5 - Chromatin Immunoprecipitation (ChIP) Analysis of Heterochromatin Proteins and Modifications on the Inactive Gene Expression Plasmid</center><ul><li><span>Diagram (A) depicts the gene expression plasmid and the location of the primer pairs used: (a) promoter, (b) beginning of globin intron/ exon module, (c) end of globin intron/exon module, (d) and (e) bacterial plasmid sequence.</span></li><li><span>(B) Results of the ChIP analysis showing the localization of HP1, HP1  , histone H3 di- and tri-MeK9, and Eu-HMTase1 on the gene expression plasmid.</span></li><li><span>(C) The levels of the associated factors and modifications are plotted.</span></li></ul><p><img src="https://39363.org/IMAGE_BUCKET/1632358433874-786146381.jpeg" style="zoom:67%;" /></p><center>Figure 6 - Dynamics of YFP-HP1 Depletion from the Locus during Transcriptional Activation</center><ul><li><span>YFP-HP1 colocalizes with the condensed locus (0 min, A–C) and the condensed regions during the early time points of transcriptional activation (D–F, 17.5 min).</span></li><li><span>It is seen in punctate structures at the 30 min time point (G–I) and appears smooth and diffuse by 50 min (J–L).</span></li><li><span>180 min postinduction, a dark region (HP1 depleted) that colocalizes with the decondensed locus, is seen in the YFP-HP1 image.</span></li><li><span>Also see Supplemental Data and Supplemental Movie S2 available on Cell website.</span></li><li><span>Scale bar is equal to 5m.</span></li></ul><p><img src="https://39363.org/IMAGE_BUCKET/1632358557125-797537709.jpeg" style="zoom:67%;" /></p><center>Figure 7. Analysis of the Deposition of the Histone Variant, H3.3, at the Locus during Transcriptional Activation</center><ul><li><span>H3.3-YFP is not enriched at the condensed locus (A–C).</span></li><li><span>A small spot adjacent to the locus is seen immediately after induction (D–F; 7.5 min).</span></li><li><span>Significant deposition begins around the periphery of the decondensing locus (G–I, 40 min; J–L, 75 min) and eventually H3.3-YFP appears in a concentrated region that does not completely colocalize with CFP-lac repressor (pseudocolored red) (M–O; 180 min).</span></li><li><span>Also see Supplemental Data and Supplemental Movie S3 available on Cell website. Scale bar is equal to 5m.</span></li></ul><p>&nbsp;</p><center><h1>Supplemental Data</h1></center><h2 id='supplemental-experimental-procedures'><span>Supplemental Experimental Procedures</span></h2><h3 id='construction-of-p3216pecms2\nβ'><span>Construction of p3216PECMS2</span><mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img" focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: -0.439ex;"><defs><path id="MJX-89-TEX-I-1D6FD" d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)"><g data-mml-node="math"><g data-mml-node="mi"><use data-c="1D6FD" xlink:href="#MJX-89-TEX-I-1D6FD"></use></g></g></g></svg><mjx-assistive-mml unselectable="on" display="inline"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi>β</mi></math></mjx-assistive-mml></mjx-container><script type="math/tex">\beta</script></h3><ul><li><span>p3216PECMS2β consists of 32 units of the lac operator, 16 units of the tetracycline-responsive element (TRE), the CMV minimal promoter (P), CFP with peroxisomal targeting signal-1 (SKL) fused to the C-terminus (C) and a rabbit β-globin intron/exon module (β).</span></li><li><span>p3216PECMS2β contains ECFP (EC) instead of CFP (C) and 24 translational operators (MS2 repeats) from the bacteriophage MS2 between ECFP-SKL and the rabbit β-globin intron/exon module.</span></li><li><span>pECFP-SKL was made by subcloning the HaeIII/KpnI fragment encoding 25 C-terminal amino acids of rat acyl-CoA oxidase into the blunted EcoRI/KpnI sites of pECFP-C1.</span></li><li><span>The sites between XhoI and EcoRI were removed by double digestion, ﬁlled in with Klenow fragment and ligated.</span></li><li><span>An Aor51HI/PstI fragment was excised from pECFP-SKL and used to replace CFP-SKL in p16PCβ and p3216PCβ, resulting in p16PECβ and p3216PECβ, respectively.</span></li><li><span>The former is composed of XhoI-TRE-P-Aor51HI-AgeI-ECFP-SKL-PstI-BamHI-β (contains internal ApaI)-HindIII.</span></li><li><span>The later is composed of XhoI-lac operator-XhoI-TRE-P-Aor51HI-AgeI-ECFP-SKL-PstI-β (contains internal ApaI)-HindIII.</span></li><li><span>The MS2 stem loop repeats were cut from pLacZ-24-βact (Fusco et al., 2003) as a BamHI/BglII fragment and inserted into the BamHI site of p16PECβ in the correct orientation resulting in p16PECMS2β.</span></li><li><span>The ECFPSKLMS2β fragment was cut from p16PECMS2β with AgeI and ApaI and replaced the corresponding part of p3216PECβ to make p3216PECMS2β.</span></li></ul><h3 id='ms2-coat-protein-constructs'><span>MS2 Coat Protein Constructs</span></h3><ul><li><span>The MS2 coat protein deleted of the 13 residues of the FG loop (mutant dlFG) important for capsid assembly and containing the V291 mutation shown to enhance RNA binding independent of a capsid-assembly defect was used to make the MS2-YFP fusion proteins.</span></li><li><span>Oligonucleotides were designed to insert the SV40 NLS in frame at the C-terminus of either YFP or the MS2 coat protein which cause the fusion protein to be retained in the nucleus until they are transported to the cytoplasm in association with mRNA transcripts.</span></li></ul><h3 id='southern-blotting'><span>Southern Blotting</span></h3><ul><li><span>The approximate copy number of plasmids integrated into the 2-6-3 cell genome was determined by comparing the intensity of a fragment from digested 2-6-3 genomic DNA to known quantities of the plasmid.</span></li><li><span>Genomic DNA was isolated using standard procedures.</span></li><li><span>Due to the aneuploidy of the U2OS cell line, DNA concentration was determined as the number of cells/µl.</span></li><li><span>DNA from 10,000 cells was digested with NcoI, which cuts at the 5′ end of ECFP and within the β-globin intron (2.4 kb fragment).</span></li><li><span>Genomic DNA from the parental U2OS cell line was digested with NcoI and mixed with known quantities of digested plasmid corresponding to 100 (22 pg), 200 (33 pg), 300 (44 pg), and 400 (66 pg) copies/cell.</span></li><li><span>Blotting was done according to standard protocols. A PCR probe from the ECFP coding region (41–482 nt) was used for detection</span></li></ul><h3 id='northern-blotting'><span>Northern Blotting</span></h3><ul><li><span>Cells were electroporated with pTet-On (2 μg) and sheared salmon sperm DNA (40 μg) and dox (1 μg/ml) was added 2.5 hr later.</span></li><li><span>Total RNA was isolated at the 0, 5, 10, 15, and 30 min time points using TRI-reagent (MRC Research Lab, Cincinnati, OH). 6–7 μg of RNA was loaded for each time point and blotting was done according to standard protocols.</span></li><li><span>An MS2 repeat fragment was used to probe the 0–30 min blot and ECFP (41–482 nt) was used for the 0–120 min blot.</span></li><li><span>After stripping the MS2 probe, the membrane was rehybridized with a β-actin probe to control for equal loading.</span></li></ul><h3 id='immunoﬂuoresence'><span>Immunoﬂuoresence</span></h3><ul><li><span>Cells were ﬁxed in 2% formaldehyde in 1× PBS pH 7.4 for 10 min, rinsed in 1× PBS and permeabilized in 0.5% Triton in 1× PBS on ice for 5 min.</span></li><li><span>Cells were washed 2× for 5 min each with 1× PBS and then for 20 min in 1× PBS + 0.5 % BSA.</span></li><li><span>Cells were incubated in 1˚ antibody (diluted in 1× PBS + 0.5 % BSA) for 1 hr, washed 3× for 10 min each in 1× PBS then incubated in 2˚ antibody (diluted in 1× PBS + 0.5 % BSA) for 1 hr, washed 3× for 10 min in 1× PBS and mounted in 90% glycerol, 10% PBS, 10 mg/ml p-phenylene-diamine (Aldrich, Milwaukee, WI) [∼pH 9].</span></li></ul><p>&nbsp;</p><p><img src="https://39363.org/IMAGE_BUCKET/1632360949794-741520924.jpeg" style="zoom:67%;" /></p><center>Figure S1</center><ul><li><span>Northern blot analysis of RNA isolated 0, 30, and 120 minutes after the addition of doxycycline showing that RNA species with molecular weights both higher and lower than the 3.4 kb pre-mRNA and 2.8 kb spliced products are produced at later time points.</span></li><li><span>The probe recognizes the CFP sequence.</span></li><li><span>Ethidium bromide staining of 28S RNA shows equal loading of RNA.</span></li></ul><p>&nbsp;</p><p><img src="https://39363.org/IMAGE_BUCKET/1632360996125-154286511.jpeg" style="zoom:67%;" /></p><center>Figure S2</center><ul><li><span>RNA ﬂuorescence in situ hybridization (RNA FISH) with a probe made by nick-translation of the pTK-Hyg plasmid shows that the drug resistant plasmid integrated into the array of inducible transcription units and is expressed at the periphery of the condensed locus.</span></li><li><span>Image stacks were collected and deconvolved and projections are shown. (A) YFP-lac repressor marks the locus, (B) Texas-red RNA FISH probe localizes RNA transcribed from pTK-Hyg and (C) merge.</span></li><li><span>Scale bar represents 5 µm. Scale bar in enlarged insets represents 1 µm.</span></li></ul><p>&nbsp;</p><p><img src="https://39363.org/IMAGE_BUCKET/1632361040239-442833849.jpeg" style="zoom:67%;" /></p><center>Figure S3</center><ul><li><span>RNA ﬂuorescence in situ hybridization (RNA FISH) with an oligo probe to sequence in the β-globin intron conﬁrmed that pre-mRNA (C) is present at the site of the decondensed locus (A) and the MS2-YFP accumulation (B).</span></li><li><span>Scale bar represents 5 µm.</span></li></ul><p>&nbsp;</p><p><img src="https://39363.org/IMAGE_BUCKET/1632361070912-285919663.jpeg" style="zoom:67%;" /></p><center>Figure S4</center><ul><li><p><span>Immunoblot analysis of YFP constructs used in the imaging studies.</span></p></li><li><p><span>Blot was probed with a monoclonal anti-GFP antibody.</span></p><p>&nbsp;</p></li></ul><p><img src="https://39363.org/IMAGE_BUCKET/1632361098098-389479131.jpeg" referrerpolicy="no-referrer"></p><center>Figure S5</center><ul><li><span>Endogenous HP1α, β, and γ colocalize with the condensed locus. 2-6-3 cells expressing YFP lac repressor were labeled with antibodies to HP1α (A–C), HP1β (D–F) and HP1γ (G–I) and all three endogenous isoforms of HP1 colocalize.</span></li><li><span>Scale bar represents 5 µm.</span></li></ul><p>&nbsp;</p><iframe width="560" height="560" src="https://www.youtube.com/embed/ifT5ybIFcT0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe><iframe width="560" height="560" src="https://www.youtube.com/embed/7bQ5Hx_B7qo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe><iframe width="560" height="560" src="https://www.youtube.com/embed/Tcvgpw5lXO8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe><p>&nbsp;</p></div></div>
</body>
</html>