<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>
<title>23.1 - 23.2 - Notes</title><style type='text/css'>html {overflow-x: initial !important;}:root { --bg-color: #ffffff; --text-color: #333333; --select-text-bg-color: #B5D6FC; --select-text-font-color: auto; --monospace: "Lucida Console",Consolas,"Courier",monospace; }
html { font-size: 14px; background-color: var(--bg-color); color: var(--text-color); font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; -webkit-font-smoothing: antialiased; }
body { margin: 0px; padding: 0px; height: auto; bottom: 0px; top: 0px; left: 0px; right: 0px; font-size: 1rem; line-height: 1.42857143; overflow-x: hidden; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; tab-size: 4; background-position: inherit inherit; background-repeat: inherit inherit; }
iframe { margin: auto; }
a.url { word-break: break-all; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { text-shadow: none; background: var(--select-text-bg-color); color: var(--select-text-font-color); }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; white-space: normal; overflow-x: visible; padding-top: 40px; }
#write.first-line-indent p { text-indent: 2em; }
#write.first-line-indent li p, #write.first-line-indent p * { text-indent: 0px; }
#write.first-line-indent li { margin-left: 2em; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
.typora-export .footnote-line, .typora-export li, .typora-export p { white-space: pre-wrap; }
.typora-export .task-list-item input { pointer-events: none; }
@media screen and (max-width: 500px) { 
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  #write { padding-left: 20px; padding-right: 20px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
#write li > figure:last-child { margin-bottom: 0.5rem; }
#write ol, #write ul { position: relative; }
img { max-width: 100%; vertical-align: middle; image-orientation: from-image; }
button, input, select, textarea { color: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-caps: inherit; font-weight: inherit; font-stretch: inherit; line-height: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
*, ::after, ::before { box-sizing: border-box; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p, #write pre { width: inherit; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p { position: relative; }
p { line-height: inherit; }
h1, h2, h3, h4, h5, h6 { break-after: avoid-page; break-inside: avoid; orphans: 4; }
p { orphans: 4; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
.md-math-block, .md-rawblock, h1, h2, h3, h4, h5, h6, p { margin-top: 1rem; margin-bottom: 1rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: 700; font-style: italic; }
a { cursor: pointer; }
sup.md-footnote { padding: 2px 4px; background-color: rgba(238, 238, 238, 0.7); color: rgb(85, 85, 85); border-top-left-radius: 4px; border-top-right-radius: 4px; border-bottom-right-radius: 4px; border-bottom-left-radius: 4px; cursor: pointer; }
sup.md-footnote a, sup.md-footnote a:hover { color: inherit; text-transform: inherit; text-decoration: inherit; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; }
figure { overflow-x: auto; margin: 1.2em 0px; max-width: calc(100% + 16px); padding: 0px; }
figure > table { margin: 0px; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 32px; }
.CodeMirror-gutters { border-right-width: 0px; background-color: inherit; }
.CodeMirror-linenumber { }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
#write pre { white-space: pre-wrap; }
#write.fences-no-line-wrapping pre { white-space: pre; }
#write pre.ty-contain-cm { white-space: normal; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; position: relative !important; background-position: inherit inherit; background-repeat: inherit inherit; }
.md-diagram-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
#write .md-fences.mock-cm { white-space: pre-wrap; }
.md-fences.md-fences-with-lineno { padding-left: 0px; }
#write.fences-no-line-wrapping .md-fences.mock-cm { white-space: pre; overflow-x: auto; }
.md-fences.mock-cm.md-fences-with-lineno { padding-left: 8px; }
.CodeMirror-line, twitterwidget { break-inside: avoid; }
.footnotes { opacity: 0.8; font-size: 0.9rem; margin-top: 1em; margin-bottom: 1em; }
.footnotes + .footnotes { margin-top: 0px; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; line-height: normal; font-weight: 400; text-align: left; box-sizing: content-box; direction: ltr; background-position: 0px 0px; background-repeat: initial initial; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li .mathjax-block, li p { margin: 0.5rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child, li > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; white-space: normal; }
#write .footnote-line { white-space: pre-wrap; }
@media print { 
  body, html { border: 1px solid transparent; height: 99%; break-after: avoid; break-before: avoid; font-variant-ligatures: no-common-ligatures; }
  #write { margin-top: 0px; padding-top: 0px; border-color: transparent !important; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  html.blink-to-pdf { font-size: 13px; }
  .typora-export #write { break-after: avoid; }
  .typora-export #write::after { height: 0px; }
  .is-mac table { break-inside: avoid; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 0.8rem; white-space: pre-wrap; background-color: rgb(204, 204, 204); display: block; overflow-x: hidden; background-position: initial initial; background-repeat: initial initial; }
p > .md-image:only-child:not(.md-img-error) img, p > img:only-child { display: block; margin: auto; }
#write.first-line-indent p > .md-image:only-child:not(.md-img-error) img { left: -2em; position: relative; }
p > .md-image:only-child { display: inline-block; width: 100%; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.md-math-block { width: 100%; }
.md-math-block:not(:empty)::after { display: none; }
[contenteditable="true"]:active, [contenteditable="true"]:focus, [contenteditable="false"]:active, [contenteditable="false"]:focus { outline: 0px; box-shadow: none; }
.md-task-list-item { position: relative; list-style-type: none; }
.task-list-item.md-task-list-item { padding-left: 0px; }
.md-task-list-item > input { position: absolute; top: 0px; left: 0px; margin-left: -1.2em; margin-top: calc(1em - 10px); border: none; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-top-left-radius: 10px; border-top-right-radius: 10px; border-bottom-right-radius: 10px; border-bottom-left-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc-content::after, .md-toc::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); }
.md-toc-item a { text-decoration: none; }
.md-toc-inner:hover { text-decoration: underline; }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: 700; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) { 
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
code, pre, samp, tt { font-family: var(--monospace); }
kbd { margin: 0px 0.1em; padding: 0.1em 0.6em; font-size: 0.8em; color: rgb(36, 39, 41); background-color: rgb(255, 255, 255); border: 1px solid rgb(173, 179, 185); border-top-left-radius: 3px; border-top-right-radius: 3px; border-bottom-right-radius: 3px; border-bottom-left-radius: 3px; box-shadow: rgba(12, 13, 14, 0.2) 0px 1px 0px, rgb(255, 255, 255) 0px 0px 0px 2px inset; white-space: nowrap; vertical-align: middle; background-position: initial initial; background-repeat: initial initial; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: var(--monospace); }
code { text-align: left; }
a.md-print-anchor { white-space: pre !important; border: none !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: 0px !important; text-shadow: initial !important; background-position: 0px 0px !important; background-repeat: initial initial !important; }
.md-inline-math .MathJax_SVG .noError { display: none !important; }
.html-for-mac .inline-math-svg .MathJax_SVG { vertical-align: 0.2px; }
.md-math-block .MathJax_SVG_Display { text-align: center; margin: 0px; position: relative; text-indent: 0px; max-width: none; max-height: none; min-height: 0px; min-width: 100%; width: auto; overflow-y: hidden; display: block !important; }
.MathJax_SVG_Display, .md-inline-math .MathJax_SVG_Display { width: auto; margin: inherit; display: inline-block !important; }
.MathJax_SVG .MJX-monospace { font-family: var(--monospace); }
.MathJax_SVG .MJX-sans-serif { font-family: sans-serif; }
.MathJax_SVG { display: inline; font-style: normal; font-weight: 400; line-height: normal; zoom: 90%; text-indent: 0px; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0px; min-height: 0px; border: 0px; padding: 0px; margin: 0px; }
.MathJax_SVG * { transition: none; }
.MathJax_SVG_Display svg { vertical-align: middle !important; margin-bottom: 0px !important; margin-top: 0px !important; }
.os-windows.monocolor-emoji .md-emoji { font-family: "Segoe UI Symbol", sans-serif; }
.md-diagram-panel > svg { max-width: 100%; }
[lang="flow"] svg, [lang="mermaid"] svg { max-width: 100%; height: auto; }
[lang="mermaid"] .node text { font-size: 1rem; }
table tr th { border-bottom-width: 0px; }
video { max-width: 100%; display: block; margin: 0px auto; }
iframe { max-width: 100%; width: 100%; border: none; }
.highlight td, .highlight tr { border: 0px; }
svg[id^="mermaidChart"] { line-height: 1em; }
mark { background-color: rgb(255, 255, 0); color: rgb(0, 0, 0); background-position: initial initial; background-repeat: initial initial; }
.md-html-inline .md-plain, .md-html-inline strong, mark .md-inline-math, mark strong { color: inherit; }
mark .md-meta { color: rgb(0, 0, 0); opacity: 0.3 !important; }
@media print { 
  .typora-export h1, .typora-export h2, .typora-export h3, .typora-export h4, .typora-export h5, .typora-export h6 { break-inside: avoid; }
}


/* Flowchart variables */
/* Sequence Diagram variables */
/* Gantt chart variables */
/* state colors */
.label {
  
  color: #333; }

.label text {
  fill: #333; }

.node rect,
.node circle,
.node ellipse,
.node polygon {
  fill: #BDD5EA;
  stroke: #9370DB;
  stroke-width: 1px; }

.node .label {
  text-align: center; }

.node.clickable {
  cursor: pointer; }

.arrowheadPath {
  fill: lightgrey; }

.edgePath .path {
  stroke: lightgrey;
  stroke-width: 1.5px; }

.edgeLabel {
  background-color: #e8e8e8;
  text-align: center; }

.cluster rect {
  fill: #6D6D65;
  stroke: rgba(255, 255, 255, 0.25);
  stroke-width: 1px; }

.cluster text {
  fill: #F9FFFE; }

div.mermaidTooltip {
  position: absolute;
  text-align: center;
  max-width: 200px;
  padding: 2px;
  
  font-size: 12px;
  background: #6D6D65;
  border: 1px solid rgba(255, 255, 255, 0.25);
  border-radius: 2px;
  pointer-events: none;
  z-index: 100; }

.actor {
  stroke: #81B1DB;
  fill: #BDD5EA; }

text.actor {
  fill: black;
  stroke: none; }

.actor-line {
  stroke: lightgrey; }

.messageLine0 {
  stroke-width: 1.5;
  stroke-dasharray: '2 2';
  stroke: lightgrey; }

.messageLine1 {
  stroke-width: 1.5;
  stroke-dasharray: '2 2';
  stroke: lightgrey; }

#arrowhead {
  fill: lightgrey; }

.sequenceNumber {
  fill: white; }

#sequencenumber {
  fill: lightgrey; }

#crosshead path {
  fill: lightgrey !important;
  stroke: lightgrey !important; }

.messageText {
  fill: lightgrey;
  stroke: none; }

.labelBox {
  stroke: #81B1DB;
  fill: #BDD5EA; }

.labelText {
  fill: #323D47;
  stroke: none; }

.loopText {
  fill: lightgrey;
  stroke: none; }

.loopLine {
  stroke-width: 2;
  stroke-dasharray: '2 2';
  stroke: #81B1DB; }

.note {
  stroke: rgba(255, 255, 255, 0.25);
  fill: #fff5ad; }

.noteText {
  fill: black;
  stroke: none;
  
  font-size: 14px; }

.activation0 {
  fill: #f4f4f4;
  stroke: #666; }

.activation1 {
  fill: #f4f4f4;
  stroke: #666; }

.activation2 {
  fill: #f4f4f4;
  stroke: #666; }

/** Section styling */
.section {
  stroke: none;
  opacity: 0.2; }

.section0 {
  fill: rgba(255, 255, 255, 0.3); }

.section2 {
  fill: #EAE8B9; }

.section1,
.section3 {
  fill: white;
  opacity: 0.2; }

.sectionTitle0 {
  fill: #F9FFFE; }

.sectionTitle1 {
  fill: #F9FFFE; }

.sectionTitle2 {
  fill: #F9FFFE; }

.sectionTitle3 {
  fill: #F9FFFE; }

.sectionTitle {
  text-anchor: start;
  font-size: 11px;
  text-height: 14px;
   }

/* Grid and axis */
.grid .tick {
  stroke: lightgrey;
  opacity: 0.3;
  shape-rendering: crispEdges; }

.grid path {
  stroke-width: 0; }

/* Today line */
.today {
  fill: none;
  stroke: #DB5757;
  stroke-width: 2px; }

/* Task styling */
/* Default task */
.task {
  stroke-width: 2; }

.taskText {
  text-anchor: middle;
   }

.taskText:not([font-size]) {
  font-size: 11px; }

.taskTextOutsideRight {
  fill: #323D47;
  text-anchor: start;
  font-size: 11px;
   }

.taskTextOutsideLeft {
  fill: #323D47;
  text-anchor: end;
  font-size: 11px; }

/* Special case clickable */
.task.clickable {
  cursor: pointer; }

.taskText.clickable {
  cursor: pointer;
  fill: #003163 !important;
  font-weight: bold; }

.taskTextOutsideLeft.clickable {
  cursor: pointer;
  fill: #003163 !important;
  font-weight: bold; }

.taskTextOutsideRight.clickable {
  cursor: pointer;
  fill: #003163 !important;
  font-weight: bold; }

/* Specific task settings for the sections*/
.taskText0,
.taskText1,
.taskText2,
.taskText3 {
  fill: #323D47; }

.task0,
.task1,
.task2,
.task3 {
  fill: #BDD5EA;
  stroke: rgba(255, 255, 255, 0.5); }

.taskTextOutside0,
.taskTextOutside2 {
  fill: lightgrey; }

.taskTextOutside1,
.taskTextOutside3 {
  fill: lightgrey; }

/* Active task */
.active0,
.active1,
.active2,
.active3 {
  fill: #81B1DB;
  stroke: rgba(255, 255, 255, 0.5); }

.activeText0,
.activeText1,
.activeText2,
.activeText3 {
  fill: #323D47 !important; }

/* Completed task */
.done0,
.done1,
.done2,
.done3 {
  stroke: grey;
  fill: lightgrey;
  stroke-width: 2; }

.doneText0,
.doneText1,
.doneText2,
.doneText3 {
  fill: #323D47 !important; }

/* Tasks on the critical line */
.crit0,
.crit1,
.crit2,
.crit3 {
  stroke: #E83737;
  fill: #E83737;
  stroke-width: 2; }

.activeCrit0,
.activeCrit1,
.activeCrit2,
.activeCrit3 {
  stroke: #E83737;
  fill: #81B1DB;
  stroke-width: 2; }

.doneCrit0,
.doneCrit1,
.doneCrit2,
.doneCrit3 {
  stroke: #E83737;
  fill: lightgrey;
  stroke-width: 2;
  cursor: pointer;
  shape-rendering: crispEdges; }

.milestone {
  transform: rotate(45deg) scale(0.8, 0.8); }

.milestoneText {
  font-style: italic; }

.doneCritText0,
.doneCritText1,
.doneCritText2,
.doneCritText3 {
  fill: #323D47 !important; }

.activeCritText0,
.activeCritText1,
.activeCritText2,
.activeCritText3 {
  fill: #323D47 !important; }

.titleText {
  text-anchor: middle;
  font-size: 18px;
  fill: #323D47;
   }

g.classGroup text {
  fill: #9370DB;
  stroke: none;
  
  font-size: 10px; }
  g.classGroup text .title {
    font-weight: bolder; }

g.classGroup rect {
  fill: #BDD5EA;
  stroke: #9370DB; }

g.classGroup line {
  stroke: #9370DB;
  stroke-width: 1; }

.classLabel .box {
  stroke: none;
  stroke-width: 0;
  fill: #BDD5EA;
  opacity: 0.5; }

.classLabel .label {
  fill: #9370DB;
  font-size: 10px; }

.relation {
  stroke: #9370DB;
  stroke-width: 1;
  fill: none; }

#compositionStart {
  fill: #9370DB;
  stroke: #9370DB;
  stroke-width: 1; }

#compositionEnd {
  fill: #9370DB;
  stroke: #9370DB;
  stroke-width: 1; }

#aggregationStart {
  fill: #BDD5EA;
  stroke: #9370DB;
  stroke-width: 1; }

#aggregationEnd {
  fill: #BDD5EA;
  stroke: #9370DB;
  stroke-width: 1; }

#dependencyStart {
  fill: #9370DB;
  stroke: #9370DB;
  stroke-width: 1; }

#dependencyEnd {
  fill: #9370DB;
  stroke: #9370DB;
  stroke-width: 1; }

#extensionStart {
  fill: #9370DB;
  stroke: #9370DB;
  stroke-width: 1; }

#extensionEnd {
  fill: #9370DB;
  stroke: #9370DB;
  stroke-width: 1; }

.commit-id,
.commit-msg,
.branch-label {
  fill: lightgrey;
  color: lightgrey;
   }

.pieTitleText {
  text-anchor: middle;
  font-size: 25px;
  fill: #eee;
}

g.stateGroup text {
  stroke: none;
  font-size: 10px;
}

g.stateGroup circle {
  fill: white !important;
  stroke: white !important;
}

g.stateGroup .state-title {
  font-weight: bolder;
  fill: black; }

g.stateGroup rect {
  fill: #ececff;
  stroke: #9370DB; }

g.stateGroup line {
  stroke: #9370DB;
  stroke-width: 1; }

.transition {
  stroke: #9370DB;
  stroke-width: 1;
  fill: none; }

.stateGroup .composit {
  fill: #555;
  border-bottom: 1px; }

.state-note {
  stroke: rgba(255, 255, 255, 0.25);
  fill: #fff5ad; }
  .state-note text {
    fill: black;
    stroke: none;
    font-size: 10px; }

.stateLabel .box {
  stroke: none;
  stroke-width: 0;
  fill: #BDD5EA;
  opacity: 0.5; }

.stateLabel text {
  fill: black;
  font-size: 10px;
  font-weight: bold;
}

.cluster-label {
  color:black;
}

.statediagram-cluster rect {
  fill: #BDD5EA;
  stroke: #9370DB; 
  stroke-width: 1px;
}
.statediagram-cluster rect.outer {
  rx: 5px;
  ry: 5px;
}
.statediagram-state .divider {
  stroke: #9370DB; 
}

.statediagram-state .title-state {
  rx: 5px;
  ry: 5px;
}
.statediagram-cluster.statediagram-cluster .inner {
  fill: white;
}
.statediagram-cluster.statediagram-cluster-alt .inner {
  fill: #e0e0e0;
}

.statediagram-cluster .inner {
  rx:0;
  ry:0;
}

.statediagram-state rect.basic {
  rx: 5px;
  ry: 5px;
}
.statediagram-state rect.divider {
  stroke-dasharray: 10,10;
  fill: #efefef;
}

.note-edge {
  stroke-dasharray: 5;
}

.statediagram-note rect {
  stroke: var(--cluster-border);
  fill: #fff5ad;
  stroke-width: 1px;
  rx: 0;
  ry: 0;
}

.node circle.state-start {
  fill: black;
  stroke: black;
}
.node circle.state-end {
  fill: black;
  stroke: white;
  stroke-width: 1.5
}
#statediagram-barbEnd {
  fill: #9370DB; 
}

/* CSS Document */

/** code highlight */

.cm-s-inner .cm-variable,
.cm-s-inner .cm-operator,
.cm-s-inner .cm-property {
    color: #b8bfc6;
}

.cm-s-inner .cm-keyword {
    color: #C88FD0;
}

.cm-s-inner .cm-tag {
    color: #7DF46A;
}

.cm-s-inner .cm-attribute {
    color: #7575E4;
}

.CodeMirror div.CodeMirror-cursor {
    border-left: 1px solid #b8bfc6;
    z-index: 3;
}

.cm-s-inner .cm-string {
    color: #D26B6B;
}

.cm-s-inner .cm-comment,
.cm-s-inner.cm-comment {
    color: #DA924A;
}

.cm-s-inner .cm-header,
.cm-s-inner .cm-def,
.cm-s-inner.cm-header,
.cm-s-inner.cm-def {
    color: #8d8df0;
}

.cm-s-inner .cm-quote,
.cm-s-inner.cm-quote {
    color: #57ac57;
}

.cm-s-inner .cm-hr {
    color: #d8d5d5;
}

.cm-s-inner .cm-link {
    color: #d3d3ef;
}

.cm-s-inner .cm-negative {
    color: #d95050;
}

.cm-s-inner .cm-positive {
    color: #50e650;
}

.cm-s-inner .cm-string-2 {
    color: #f50;
}

.cm-s-inner .cm-meta,
.cm-s-inner .cm-qualifier {
    color: #b7b3b3;
}

.cm-s-inner .cm-builtin {
    color: #f3b3f8;
}

.cm-s-inner .cm-bracket {
    color: #997;
}

.cm-s-inner .cm-atom,
.cm-s-inner.cm-atom {
    color: #84B6CB;
}

.cm-s-inner .cm-number {
    color: #64AB8F;
}

.cm-s-inner .cm-variable {
    color: #b8bfc6;
}

.cm-s-inner .cm-variable-2 {
    color: #9FBAD5;
}

.cm-s-inner .cm-variable-3 {
    color: #1cc685;
}

.CodeMirror-selectedtext,
.CodeMirror-selected {
    background: #4a89dc;
    color: #fff !important;
    text-shadow: none;
}

.CodeMirror-gutters {
    border-right: none;
}

/* CSS Document */

/** markdown source **/
.cm-s-typora-default .cm-header, 
.cm-s-typora-default .cm-property
{
    color: #cebcca;
}

.CodeMirror.cm-s-typora-default div.CodeMirror-cursor{
    border-left: 3px solid #b8bfc6;
}

.cm-s-typora-default .cm-comment {
    color: #9FB1FF;
}

.cm-s-typora-default .cm-string {
    color: #A7A7D9
}

.cm-s-typora-default .cm-atom, .cm-s-typora-default .cm-number {
    color: #848695;
    font-style: italic;
}

.cm-s-typora-default .cm-link {
    color: #95B94B;
}

.cm-s-typora-default .CodeMirror-activeline-background {
    background: rgba(51, 51, 51, 0.72);
}

.cm-s-typora-default .cm-comment, .cm-s-typora-default .cm-code {
	color: #8aa1e1;
}@import "";
@import "";
@import "";

:root {
    --bg-color:  #363B40;
    --side-bar-bg-color: #2E3033;
    --text-color: #F4F5F6;

    --select-text-bg-color:#4a89dc;

    --item-hover-bg-color: #0a0d16;
    --control-text-color: #b7b7b7;
    --control-text-hover-color: #eee;
    --window-border: 1px solid #555;

    --active-file-bg-color: rgb(34, 34, 34);
    --active-file-border-color: #8d8df0;

    --primary-color: #a3d5fe;

    --active-file-text-color: white;
    --item-hover-bg-color: #70717d;
    --item-hover-text-color: white;
    --primary-color: #6dc1e7;

    --rawblock-edit-panel-bd: #333;

    --search-select-bg-color: #428bca;
}

html {
    font-size: 16px;
}

html,
body {
    -webkit-text-size-adjust: 100%;
    -ms-text-size-adjust: 100%;
    background: #363B40;
    background: var(--bg-color);
    fill: currentColor;
    line-height: 1.625rem;
}

#write {
    max-width: 914px;
}


@media only screen and (min-width: 1400px) {
	#write {
		max-width: 1024px;
	}
}

@media only screen and (min-width: 1800px) {
	#write {
		max-width: 1200px;
	}
}

html,
body,
button,
input,
select,
textarea,
div.code-tooltip-content {
    color: #F4F5F6;
    border-color: transparent;
}

div.code-tooltip,
.md-hover-tip .md-arrow:after {
    background: #333;
}

.popover.bottom > .arrow:after {
    border-bottom-color: #333;
}

html,
body,
button,
input,
select,
textarea {
    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
}

hr {
    height: 2px;
    border: 0;
    margin: 24px 0 !important;
}

h1,
h2,
h3,
h4,
h5,
h6 {
    font-family: "Lucida Grande", "Corbel", sans-serif;
    font-weight: normal;
    clear: both;
    -ms-word-wrap: break-word;
    word-wrap: break-word;
    margin: 0;
    padding: 0;
    color: #DEDEDE
}

h1 {
    font-size: 2.5rem;
    /* 36px */
    line-height: 2.75rem;
    /* 40px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1.5px;
}

h2 {
    font-size: 1.63rem;
    /* 24px */
    line-height: 1.875rem;
    /* 30px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h3 {
    font-size: 1.17rem;
    /* 18px */
    line-height: 1.5rem;
    /* 24px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h4 {
    font-size: 1.12rem;
    /* 16px */
    line-height: 1.375rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    color: white;
}

h5 {
    font-size: 0.97rem;
    /* 16px */
    line-height: 1.25rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    font-weight: bold;
}

h6 {
    font-size: 0.93rem;
    /* 16px */
    line-height: 1rem;
    /* 16px */
    margin-bottom: 0.75rem;
    color: white;
}

@media (min-width: 980px) {
    h3.md-focus:before,
    h4.md-focus:before,
    h5.md-focus:before,
    h6.md-focus:before {
        color: #ddd;
        border: 1px solid #ddd;
        border-radius: 3px;
        position: absolute;
        left: -1.642857143rem;
        top: .357142857rem;
        float: left;
        font-size: 9px;
        padding-left: 2px;
        padding-right: 2px;
        vertical-align: bottom;
        font-weight: normal;
        line-height: normal;
    }

    h3.md-focus:before {
        content: 'h3';
    }

    h4.md-focus:before {
        content: 'h4';
    }

    h5.md-focus:before {
        content: 'h5';
        top: 0px;
    }

    h6.md-focus:before {
        content: 'h6';
        top: 0px;
    }
}

a {
    text-decoration: none;
    outline: 0;
}

a:hover {
    outline: 0;
}

a:focus {
    outline: thin dotted;
}

sup.md-footnote {
    background-color: #555;
    color: #ddd;
}

p {
    -ms-word-wrap: break-word;
    word-wrap: break-word;
}

p,
ul,
dd,
ol,
hr,
address,
pre,
table,
iframe,
.wp-caption,
.wp-audio-shortcode,
.wp-video-shortcode {
    margin-top: 0;
    margin-bottom: 1.5rem;
    /* 24px */
}

li > blockquote {
	margin-bottom: 0;
}

audio:not([controls]) {
    display: none;
}

[hidden] {
    display: none;
}

::-moz-selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

*.in-text-selection,
::selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

ul,
ol {
    padding: 0 0 0 1.875rem;
    /* 30px */
}

ul {
    list-style: square;
}

ol {
    list-style: decimal;
}

ul ul,
ol ol,
ul ol,
ol ul {
    margin: 0;
}

b,
th,
dt,
strong {
    font-weight: bold;
}

i,
em,
dfn,
cite {
    font-style: italic;
}

blockquote {
    padding-left: 1.875rem;
    margin: 0 0 1.875rem 1.875rem;
    border-left: solid 2px #474d54;
    padding-left: 30px;
    margin-top: 35px;
}

pre,
code,
kbd,
tt,
var {
    font-size: 0.875rem;
    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
}

code,
tt,
var {
    background: rgba(0, 0, 0, 0.05);
}

kbd {
    padding: 2px 4px;
    font-size: 90%;
    color: #fff;
    background-color: #333;
    border-radius: 3px;
    box-shadow: inset 0 -1px 0 rgba(0,0,0,.25);
}

pre.md-fences {
    padding: 10px 10px 10px 30px;
    margin-bottom: 20px;
    background: #333;
}

.CodeMirror-gutters {
    background: #333;
    border-right: 1px solid transparent;
}

.enable-diagrams pre.md-fences[lang="sequence"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="flow"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="mermaid"] .code-tooltip {
    bottom: -2.2em;
    right: 4px;
}

code,
kbd,
tt,
var {
    padding: 2px 5px;
}

table {
    max-width: 100%;
    width: 100%;
    border-collapse: collapse;
    border-spacing: 0;
}

th,
td {
    padding: 5px 10px;
    vertical-align: top;
}

a {
    -webkit-transition: all .2s ease-in-out;
    transition: all .2s ease-in-out;
}

hr {
    background: #474d54;
    /* variable */
}

h1 {
    margin-top: 2em;
}

a {
    color: #e0e0e0;
    text-decoration: underline;
}

a:hover {
    color: #fff;
}

.md-inline-math script {
    color: #81b1db;
}

b,
th,
dt,
strong {
    color: #DEDEDE;
    /* variable */
}

mark {
    background: #D3D40E;
}

blockquote {
    color: #9DA2A6;
}

table a {
    color: #DEDEDE;
    /* variable */
}

th,
td {
    border: solid 1px #474d54;
    /* variable */
}

.task-list {
    padding-left: 0;
}

.md-task-list-item {
    padding-left: 1.25rem;
}

.md-task-list-item > input {
    top: auto;
}

.md-task-list-item > input:before {
    content: "";
    display: inline-block;
    width: 0.875rem;
    height: 0.875rem;
    vertical-align: middle;
    text-align: center;
    border: 1px solid #F4F5F6;
    background-color: #363B40;
    margin-top: -0.4rem;
}

.md-task-list-item > input:checked:before,
.md-task-list-item > input[checked]:before {
    content: '\221A';
    /*◘*/
    font-size: 0.625rem;
    line-height: 0.625rem;
    color: #DEDEDE;
}

/** quick open **/
.auto-suggest-container {
    border: 0px;
    background-color: #525C65;
}

#typora-quick-open {
    background-color: #525C65;
}

#typora-quick-open input{
    background-color: #525C65;
    border: 0;
    border-bottom: 1px solid grey;
}

.typora-quick-open-item {
    background-color: inherit;
    color: inherit;
}

.typora-quick-open-item.active,
.typora-quick-open-item:hover {
    background-color: #4D8BDB;
    color: white;
}

.typora-quick-open-item:hover {
    background-color: rgba(77, 139, 219, 0.8);
}

.typora-search-spinner > div {
  background-color: #fff;
}

#write pre.md-meta-block {
    border-bottom: 1px dashed #ccc;
    background: transparent;
    padding-bottom: 0.6em;
    line-height: 1.6em;
}

.btn,
.btn .btn-default {
    background: transparent;
    color: #F4F5F6;
}

.ty-table-edit {
    border-top: 1px solid gray;
    background-color: #363B40;
}

.popover-title {
    background: transparent;
}

.md-image>.md-meta {
    color: #BBBBBB;
    background: transparent;
}

.md-expand.md-image>.md-meta {
    color: #DDD;
}

#write>h3:before,
#write>h4:before,
#write>h5:before,
#write>h6:before {
    border: none;
    border-radius: 0px;
    color: #888;
    text-decoration: underline;
    left: -1.4rem;
    top: 0.2rem;
}

#write>h3.md-focus:before {
    top: 2px;
}

#write>h4.md-focus:before {
    top: 2px;
}

.md-toc-item {
    color: #A8C2DC;
}

#write div.md-toc-tooltip {
    background-color: #363B40;
}

.dropdown-menu .btn:hover,
.dropdown-menu .btn:focus,
.md-toc .btn:hover,
.md-toc .btn:focus {
    color: white;
    background: black;
}

#toc-dropmenu {
    background: rgba(50, 54, 59, 0.93);
    border: 1px solid rgba(253, 253, 253, 0.15);
}

#toc-dropmenu .divider {
    background-color: #9b9b9b;
}

.outline-expander:before {
    top: 2px;
}

#typora-sidebar {
    box-shadow: none;
    border-right: 1px dashed;
    border-right: none;
}

.sidebar-tabs {
    border-bottom:0;
}

#typora-sidebar:hover .outline-title-wrapper {
    border-left: 1px dashed;
}

.outline-title-wrapper .btn {
    color: inherit;
}

.outline-item:hover {
    border-color: #363B40;
    background-color: #363B40;
    color: white;
}

h1.md-focus .md-attr,
h2.md-focus .md-attr,
h3.md-focus .md-attr,
h4.md-focus .md-attr,
h5.md-focus .md-attr,
h6.md-focus .md-attr,
.md-header-span .md-attr {
    color: #8C8E92;
    display: inline;
}

.md-comment {
    color: #5a95e3;
    opacity: 1;
}

.md-inline-math svg {
    color: #F4F5F6;
}

#math-inline-preview .md-arrow:after {
    background: black;
}

.modal-content {
    background: var(--bg-color);
    border: 0;
}

.modal-title {
    font-size: 1.5em;
}

.modal-content input {
    background-color: rgba(26, 21, 21, 0.51);
    color: white;
}

.modal-content .input-group-addon {
    color: white;
}

.modal-backdrop {
    background-color: rgba(174, 174, 174, 0.7);
}

.modal-content .btn-primary {
    border-color: var(--primary-color);
}

.md-table-resize-popover {
    background-color: #333;
}

.form-inline .input-group .input-group-addon {
    color: white;
}

#md-searchpanel {
    border-bottom: 1px dashed grey;
}

/** UI for electron */

.context-menu,
#spell-check-panel,
#footer-word-count-info {
    background-color: #42464A;
}

.context-menu.dropdown-menu .divider,
.dropdown-menu .divider {
    background-color: #777777;
}

footer {
    color: inherit;
}

@media (max-width: 1000px) {
    footer {
        border-top: none;
    }
    footer:hover {
        color: inherit;
    }
}

#file-info-file-path .file-info-field-value:hover {
    background-color: #555;
    color: #dedede;
}

.megamenu-content,
.megamenu-opened header {
    background: var(--bg-color);
}

.megamenu-menu-panel h2,
.megamenu-menu-panel h1,
.long-btn {
    color: inherit;
}

.megamenu-menu-panel input[type='text'] {
    background: inherit;
    border: 0;
    border-bottom: 1px solid;
}

#recent-file-panel-action-btn {
    background: inherit;
    border: 1px grey solid;
}

.megamenu-menu-panel .dropdown-menu > li > a {
    color: inherit;
    background-color: #2F353A;
    text-decoration: none;
}

.megamenu-menu-panel table td:nth-child(1) {
    color: inherit;
    font-weight: bold;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(1) {
    color: white;
}

.modal-footer .btn-default,
.modal-footer .btn-primary,
.modal-footer .btn-default:not(:hover) {
    border: 1px solid;
    border-color: transparent;
}

.btn-default:hover, .btn-default:focus, .btn-default.focus, .btn-default:active, .btn-default.active, .open > .dropdown-toggle.btn-default {
    color: white;
    border: 1px solid #ddd;
    background-color: inherit;
}

.modal-header {
    border-bottom: 0;
}

.modal-footer {
    border-top: 0;
}

#recent-file-panel tbody tr:nth-child(2n-1) {
    background-color: transparent !important;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(2) {
    color: inherit;
}

.megamenu-menu-panel .btn {
    border: 1px solid #eee;
    background: transparent;
}

.mouse-hover .toolbar-icon.btn:hover,
#w-full.mouse-hover,
#w-pin.mouse-hover {
    background-color: inherit;
}

.typora-node::-webkit-scrollbar {
    width: 5px;
}

.typora-node::-webkit-scrollbar-thumb:vertical {
    background: rgba(250, 250, 250, 0.3);
}

.typora-node::-webkit-scrollbar-thumb:vertical:active {
    background: rgba(250, 250, 250, 0.5);
}

#w-unpin {
    background-color: #4182c4;
}

#top-titlebar, #top-titlebar * {
    color: var(--item-hover-text-color);
}

.typora-sourceview-on #toggle-sourceview-btn,
#footer-word-count:hover,
.ty-show-word-count #footer-word-count {
    background: #333333;
}

#toggle-sourceview-btn:hover {
    color: #eee;
    background: #333333;
}

/** focus mode */
.on-focus-mode .md-end-block:not(.md-focus):not(.md-focus-container) * {
    color: #686868 !important;
}

.on-focus-mode .md-end-block:not(.md-focus) img,
.on-focus-mode .md-task-list-item:not(.md-focus-container)>input {
    opacity: #686868 !important;
}

.on-focus-mode li[cid]:not(.md-focus-container){
    color: #686868;
}

.on-focus-mode .md-fences.md-focus .CodeMirror-code>*:not(.CodeMirror-activeline) *,
.on-focus-mode .CodeMirror.cm-s-inner:not(.CodeMirror-focused) * {
    color: #686868 !important;
}

.on-focus-mode .md-focus,
.on-focus-mode .md-focus-container {
    color: #fff;
}

.on-focus-mode #typora-source .CodeMirror-code>*:not(.CodeMirror-activeline) * {
    color: #686868 !important;
}


/*diagrams*/
#write .md-focus .md-diagram-panel {
    border: 1px solid #ddd;
    margin-left: -1px;
    width: calc(100% + 2px);
}

/*diagrams*/
#write .md-focus.md-fences-with-lineno .md-diagram-panel {
    margin-left: auto;
}

.md-diagram-panel-error {
    color: #f1908e;
}

.active-tab-files #info-panel-tab-file,
.active-tab-files #info-panel-tab-file:hover,
.active-tab-outline #info-panel-tab-outline,
.active-tab-outline #info-panel-tab-outline:hover {
    color: #eee;
}

.sidebar-footer-item:hover,
.footer-item:hover {
    background: inherit;
    color: white;
}

.ty-side-sort-btn.active,
.ty-side-sort-btn:hover,
.selected-folder-menu-item a:after {
    color: white;
}

#sidebar-files-menu {
    border:solid 1px;
    box-shadow: 4px 4px 20px rgba(0, 0, 0, 0.79);
    background-color: var(--bg-color);
}

.file-list-item {
    border-bottom:none;
}

.file-list-item-summary {
    opacity: 1;
}

.file-list-item.active:first-child {
    border-top: none;
}

.file-node-background {
    height: 32px;
}

.file-library-node.active>.file-node-content,
.file-list-item.active {
    color: white;
    color: var(--active-file-text-color);
}

.file-library-node.active>.file-node-background{
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}
.file-list-item.active {
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}

#ty-tooltip {
    background-color: black;
    color: #eee;
}

.md-task-list-item>input {
    margin-left: -1.3em;
    margin-top: 0.3rem;
    -webkit-appearance: none;
}

.md-mathjax-midline {
    background-color: #57616b;
    border-bottom: none;
}

footer.ty-footer {
    border-color: #656565;
}

.ty-preferences .btn-default {
    background: transparent;
}
.ty-preferences .btn-default:hover {
    background: #57616b;
}

.ty-preferences select {
    border: 1px solid #989698;
    height: 21px;
}

.ty-preferences .nav-group-item.active {
    background: var(--item-hover-bg-color);
}

.ty-preferences input[type="search"] {
    border-color: #333;
    background: #333;
    line-height: 22px;
    border-radius: 6px;
    color: white;
}

.ty-preferences input[type="search"]:focus {
    box-shadow: none;
}

[data-is-directory="true"] .file-node-content {
    margin-bottom: 0;
}

.file-node-title {
    line-height: 22px;
}

.html-for-mac .file-node-open-state, .html-for-mac .file-node-icon {
    line-height: 26px;
}

::-webkit-scrollbar-thumb {
    background: rgba(230, 230, 230, 0.30);
}

::-webkit-scrollbar-thumb:active {
    background: rgba(230, 230, 230, 0.50);
}

#typora-sidebar:hover div.sidebar-content-content::-webkit-scrollbar-thumb:horizontal {
    background: rgba(230, 230, 230, 0.30);
}

.nav-group-item:active {
    background-color: #474d54;
}

.md-search-hit {
    background: rgba(199, 140, 60, 0.81);
    color: #eee;
}

.md-search-hit * {
    color: #eee;
}

#md-searchpanel input {
    color: white;
}

.export-detail,
.export-item.active,
.export-items-list-control {
    background: #d6d6d4
}


</style>
</head>
<body class='typora-export'>
<div id='write'  class=''><center><h1>Chapter 23.1 - 23.2</h1></center><p>&nbsp;</p><center><h2><span style="color:#35a166">1.) DNA Repair</span></h2></center><h3><span style="color:#63C0F5">Types and Consequences of DNA Damage</span></h3><ul><li><p><span>Unprogrammed chemical changes occur in all biological macromolecules because of </span></p><ul><li><span>environmental damage</span></li><li><span>errors in synthesis.</span></li></ul></li><li><p><span>For most biomolecules, including RNA, protein, and membrane lipids, the effects of these changes are minimized by turnover and replacement of the altered molecules.</span></p></li><li><p><span>DNA is distinctive, however, in that its information content must be transmitted virtually intact from one cell to another during cell division or the reproduction of an organism. </span></p><ul><li><span>Thus, DNA requires metabolic stability. </span></li></ul></li><li><p><span>This stability is maintained in two ways</span></p><ul><li><span>By a highly accurate replication process</span></li><li><span>By mechanisms for correcting genetic mistakes when DNA suffers damage.</span></li></ul></li><li><p><span>In Chapter 22 we described mechanisms used to ensure high replication fidelity. </span></p></li><li><p><span>Here we discuss the kinds of environmental and endogenous damage that occur and the processes that repair the damage. </span></p></li><li><p><span>Figure 23.2 identifies the major kinds of DNA damage that arise through either replication errors or the actions of environmental agents. </span></p></li></ul><p><img src="https://39363.org/IMAGE_BUCKET/1605501305710-916244329.jpeg" style="zoom:75%;" /></p><blockquote><p><strong><span>Connection</span></strong><span> One of the principal causes of cancer is genomic instability resulting from the defective repair of DNA damage. </span></p></blockquote><ul><li><p><span>The figure also identifies each of the major processes that can repair particular types of damage. </span></p></li><li><p><span>Because unrepaired damage is mutagenic and cancer arises from genomic instability (Chapter 20), the figure also identifies forms of cancer that result from defects in particular repair processes.</span></p></li><li><p><span>Some of the most prominent endogenous DNA-damaging reactions include : </span></p><ul><li><span>(1) depurination, resulting from cleavage of the glycosidic bond between deoxyribose and a purine base; </span></li><li><span>(2) deamination, usually the hydrolytic conversion of a DNA-cytosine residue to uracil </span></li><li><span>(3) oxidation, notably the oxidation of guanine to 8-oxoguanine or of thymine to thymine glycol</span></li><li><span>(4) nonenzymatic methylation by S-adenosylmethionine.</span></li><li><span>Figure 23.3 shows the frequency with which they occur in mammalian cells.</span></li></ul></li><li><p><span>Environmental DNA-damaging agents include : </span></p><ul><li><span>ionizing radiation</span></li><li><span>ultraviolet radiation</span></li><li><span>DNA methylating reagents, such as N-methylN′-nitro-N-nitrosoguanidine (MNNG; next page)</span></li><li><span>DNA crosslinking reagents, such as the anticancer drug cisplatin</span></li><li><span>Bulky hydrocarbons, such as benzo[a]pyrene, one of the carcinogenic agents in tobacco smoke.</span></li></ul></li><li><p><span>As we discuss the repair of spontaneous or environmental DNA damage, we will see that some repair processes are accurate (i.e., the original DNA sequence is restored), while some lead to mutations (i.e., the repair process is inaccurate). </span></p></li><li><p><span>As mentioned above and in Chapter 20, cancer results from an accumulation of somatic cell mutations. </span></p><ul><li><span>Therefore, the mechanisms of DNA repair are under intense study as determinants of an animals susceptibility to cancer.</span></li></ul></li><li><p><span>Our earliest understanding of DNA repair came from study of the lethal and mutagenic effects of ultraviolet light on bacteria or viruses. </span></p></li><li><p><span>How do we know that DNA is the target for UV light? </span></p><ul><li><span>Organisms were found to be  killed or mutagenized most efficiently by light of wavelength 260 nm, the absorption maximum for DNA.</span></li><li><span>Analysis of the </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015D32'><span>photoproducts</span></a><span>, or altered DNA constituents after irradiation, showed the most prominent to be intrastrand dimers consisting of two bases (usually thymine) joined either by a cyclobutane ring involving carbons 5 and 6 or by a bond from C4 of one pyrimidine to C6 of the next (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P700101482500000000000000000497B'><span>Figure 23.4</span></a><span>).</span></li><li><span>These </span><strong><span>thymine dimers</span></strong><span> were identified early as biologically significant photoproducts because the relative abundance of thymine dimers in irradiated DNA correlated most closely with death or mutagenesis of irradiated phages or bacteria.</span></li><li><span>Thus, the ability of an organism to survive ultraviolet irradiation was partially due to its ability to remove thymine dimers from its DNA.</span></li><li><span>The cyclobutane structure draws the adjacent thymine residues together, distorting the helix in a way that blocks replication past this site, while the 6–4 dimer is strongly mutagenic because it induces inaccurate replication.</span></li></ul></li></ul><blockquote><p><strong><span>Concept</span></strong><span> Cyclobutane thymine dimer is the most lethal photoproduct in UV light–irradiated DNA, and 6–4 dimer is the most mutagenic</span></p></blockquote><center><h2><span style="color:#35a166">2.) Direct repair of Damaged DNA Bases: Photoreactivation and Alkyltransferases</span></h2></center><ul><li><span>Of the half dozen well-understood DNA repair processes, most involve removal of the damaged nucleotides, along with several adjacent  residues, followed by replacement of the excised region using information encoded in the complementary (undamaged) strand. </span></li><li><span>However, at least two processes involve reactions that </span><em><span>directly change</span></em><span> the damaged bases, rather than removing them.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> DNA can be repaired directly, by changing a damaged base to a normal one,  or indirectly, by replacing a DNA segment containing the damaged  nucleotide.</span></p></blockquote><h3><span style="color:#63C0F5">Photoreactivation</span></h3><ul><li><span>In some organisms, not including mammals, DNA damage can be repaired directly, by visible light irradiation at about 370 nm, in a process called </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015D35'><span>photoreactivation</span></a><span>.</span></li><li><span>This involves the action of the enzyme </span><strong><span>DNA photolyase</span></strong><span>. </span></li><li><span>The enzyme binds to DNA in a light-independent process, specifically at the site of pyrimidine dimers.</span></li><li><span>In the presence of visible-wavelength light, the bonds linking the pyrimidine rings are broken, after which the enzyme can dissociate in the dark.</span></li></ul><ul><li><span>Clues to the photolyase mechanism have come with the finding that the enzyme contains two chromophores. </span></li><li><span>One chromophore is bound flavin adenine dinucleotide, deprotonated and in the reduced state (FADH−)</span></li><li><span>The second in many photolyases is 5,10-methenyltetrahydrofolate (introduced in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch18_pg0001.xhtml#P7001014825000000000000000003D65'><span>Chapter 18</span></a><span>). </span></li><li><span>Mechanistic studies suggest a process akin to photosynthesis, with the second chromophore functioning as a light-harvesting factor and transmitting light energy by fluorescent resonance energy transfer to FADH−.</span></li><li><span>Hence, FADH− functions like the photochemical reaction center, transferring an electron to the dimer and breaking the pyrimidine–pyrimidine bonds by a free radical mechanism.</span></li><li><span>The crystal structure of the </span><em><span>E. coli</span></em><span> photolyase shows 5,10-methenyltetrahydrofolate bound at the surface, between the N-terminal and C-terminal domains, with FADH− bound deeply within the C-terminal domain (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P700101482500000000000000000498D'><span>Figure 23.5a</span></a><span>). </span></li><li><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P700101482500000000000000000498D'><span>Figure 23.5b</span></a><span> shows the probable reaction pathway.</span></li></ul><h3><span style="color:#63C0F5">O<sup>6</sup> - Alkylguanine Alkyltransferase</span></h3><ul><li><span>Like UV irradiation, exposure of DNA to a methylating or ethylating reagent yields modified DNA bases, some of which cause mutagenesis or death if not repaired.</span></li><li><span>Some alkylating agents are used in cancer chemotherapy because of their ability to block DNA replication and, hence, cell  proliferation.</span></li><li><span>Some alkylating agents are used as mutagens in the laboratory, including N-methyl-N</span><em><span>′</span></em><span>-nitro-N-nitrosoguanidine (MNNG, page </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P700101482500000000000000000497A'><span>717</span></a><span>) and others shown here.</span></li><li><span>MNNG and other alkylating agents react primarily with purines. </span></li><li><span>The most highly mutagenic of these products, O6-methylguanine, has a high probability of pairing with thymine when the modified strand replicates (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P700101482500000000000000000499B'><span>Figure 23.6</span></a><span>). </span></li><li><span>Thus, alkylation of a DNA-guanine stimulates a GC → AT transition mutation (see </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P700101482500000000000000000499B'><span>Figure 23.6b</span></a><span>) where mG is a methylguanine residue.</span></li></ul><ul><li><span>Repair of this type of damage involves action of O6-alkylguanine alkyltransferase, which transfers a methyl or ethyl group from an O6-methylguanine or O6-ethylguanine residue to a cysteine residue in the active site of the protein.</span></li><li><span>Remarkably, this “enzyme,” which is widely distributed in bacteria and  eukaryotes, can function only once. </span></li><li><span>Having become alkylated, it cannot remove the alkyl group, and the protein molecule cannot repeat the process.</span></li><li><span>However, the alkylated form of the protein is a transcriptional activator, which stimulates transcription of the gene encoding the  alkyltransferase.</span></li><li><span>It thereby allows the cell to adapt to alkylation damage by using the alkylated protein to signal the cell to produce more of the protein needed to repair the damage.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> Direct repair enzymes include photolyase, which uses light energy to  pyrimidine dimers, and alkyltransferases, “enzymes” that are inactivated after just one catalytic cycle.</span></p></blockquote><center><h2><span style="color:#35a166">3.) Nucleotide Excision Repair: Excinucleases</span></h2></center><ul><li><span> is an enzyme system capable of repairing thymine dimers created in DNA by UV irradiation.</span></li><li><span>Unlike photoreactivation, this process can take place in the dark.</span></li><li><span>The enzyme system involved, which in </span><em><span>E. coli</span></em><span> includes the products of genes </span><em><span>uvrA, uvrB,</span></em><span> and </span><em><span>uvrC,</span></em><span> also acts upon a wide range of other DNA lesions that may be quite bulky, such as those created by large alkyl or aryl groups that distort the DNA double helix. </span></li><li><span>Similar systems exist in mammalian cells and in  yeast, so the process is probably universal.</span></li></ul><ul><li><span>As shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049A7'><span>Figure 23.7</span></a><span> for </span><em><span>E. coli,</span></em><span> the three-subunit UvrABC enzyme recognizes a lesion (a thymine dimer in the example shown) and, with the help of ATP hydrolysis, forces DNA to bend, step </span><strong><span>1</span></strong><span>. </span></li><li><span>Dissociation of UvrA and its replacement by UvrC, step </span><strong><span>2</span></strong><span>, leads to cleavage of the damaged strand at two sites—seven nucleotides  to the 5′ side of the damaged site and four nucleotides to the 3′ side. </span></li><li><span>The resulting gap in the damaged DNA strand is ~11 nucleotides in length, with a 3′ hydroxyl group and a 5′ phosphate at the ends, step </span><strong><span>3</span></strong><span>. </span></li><li><span>Polymerase and ligase action then replaces the damaged 11-mer with new  DNA, using the undamaged DNA strand as a template, step </span><strong><span>4</span></strong><span>.</span></li><li><span>Helicase II, the product of the </span><em><span>uvrD</span></em><span> gene, is also required, presumably to unwind and remove the excised oligonucleotide, which is ultimately broken down by other enzymes. </span></li><li><span>Because 11 nucleotides amount to about one turn of the DNA helix, the enzyme may catalyze both cleavages from the same DNA binding site. </span></li><li><span>The UvrABC enzyme is not a classical endonuclease because it cuts at two distinct sites</span></li><li><span>The term </span><strong><span>excinuclease</span></strong><span> has been proposed, denoting its role in </span><em><span>exci</span></em><span>sion repair. </span></li><li><span>This system is also involved in repairing the DNA damage that results when two strands covalently crosslink to each other.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> Excision repair involves endonuclease cleavage on both sides of a damaged site, followed by replacement synthesis.</span></p></blockquote><ul><li><span>Excision repair also occurs in mammalian cells, as shown by the presence of a human excinuclease that cleaves at positions −22 and +6 relative to a thymine dimer.</span></li><li><span>Unlike bacteria, the process in humans requires two different endonucleases—one cutting on the 5′ side and one on the 3′  side. </span></li><li><span>Excision repair in humans originally came to light through studies of a rare genetic disease called </span><strong><span>xeroderma pigmentosum (XP)</span></strong><span>. </span></li><li><span>XP is actually a family of diseases in which one or more enzymes of the excision pathway are deficient. </span></li><li><span>In affected humans there is no known way to treat the condition. </span></li><li><span>The biological consequences of XP include extreme sensitivity to sunlight and a high incidence of skin cancers.  </span></li><li><span>Although overexposure to the ultraviolet rays in sunlight increases the risk of skin cancer for all humans, the greatly increased frequency of skin cancer in XP patients highlights the importance of UV repair pathways to mammals.</span></li></ul><blockquote><p><strong><span>Connection</span></strong><span> Serious clinical consequences, notably, enhanced cancer susceptibility, result from genetic abnormalities of NER.</span></p></blockquote><ul><li><span>Later studies of nucleotide excision repair showed that active genes (those undergoing transcription) are preferred substrates for excision repair,  and within these genes the template DNA strand is preferentially repaired. </span></li><li><span>This </span><strong><span>transcription-coupled repair</span></strong><span> may initiate when a transcribing RNA polymerase becomes stalled at the site of a DNA lesion. </span></li><li><span>Coupling transcription with repair helps to ensure the integrity of genes that are actually being used. </span></li><li><span>In mammalian cells, transcription-coupled repair is a specialized mode of NER that requires additional proteins. </span></li><li><em><span>Cockayne’s syndrome</span></em><span> results from genetic defects in one or more of the human enzymes of transcription-coupled repair. </span></li><li><span>Children with this condition age  prematurely and usually die of aging-related symptoms by age 12. </span></li><li><span>Another human gene, BRCA1, discovered by Mary-Claire King, is also implicated in transcription-coupled NER. </span></li><li><span>Mutations in BRCA1, and the related BRCA2  gene, greatly increase the risk of breast and ovarian cancer. </span></li><li><span>The BRCA  genes became more widely known to the public in 2013, when actress Angelina Jolie underwent a preventive double mastectomy because she carried BRCA mutations and had a family history of breast cancer deaths.</span></li></ul><ul><li><span>As noted earlier, NER also corrects DNA that has been damaged by the formation of bulky DNA adducts. </span></li><li><span>Many environmental carcinogens give rise to such adducts. </span></li><li><span>In the absence of repair, or following error-prone  repair, these adducts can lead to mutations. </span></li><li><span>For example, polycyclic aromatic hydrocarbons (PAHs) are prevalent organic pollutants formed by incomplete combustion (e.g., by burning charcoal in a backyard grill or  by smoking). </span></li><li><span>As with many environmental carcinogens, PAHs are not directly reactive with DNA. </span></li><li><span>Indeed, their lack of chemical reactivity accounts for their persistence in the environment. </span></li><li><span>However, PAHs can undergo </span><em><span>metabolic activation</span></em><span> (i.e., </span><em><span>biotransformation</span></em><span>), thus leading to reactive electrophilic intermediates that can bind to DNA and other macromolecules. </span></li><li><span>These reactions can occur in the liver and in other organs. </span></li><li><span>The best-characterized route for the metabolic activation of PAHs is the pathway for benzo[a]pyrene metabolism (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049B4'><span>Figure 23.8</span></a><span>). </span></li><li><span>First, oxygenation catalyzed by a cytochrome P450 generates an epoxide intermediate. </span></li><li><span>This epoxide is hydrolyzed, in a reaction catalyzed by </span><em><span>epoxide hydrolase, to</span></em><span> give a dihydrodiol product. </span></li><li><span>Finally, a second cytochrome P450-catalyzed oxygenation produces the highly reactive benzo[a]pyrene 7,8-dihydrodiol-9,10-epoxide (BPDE). </span></li><li><span>BPDE is highly reactive with DNA,  forming bulky covalent adducts primarily, but not exclusively, with  guanine.</span></li></ul><blockquote><p><strong><span>Connection</span></strong><span> The carcinogenicity of polycyclic aromatic hydrocarbons (PAHs) stems from  their ability to react with DNA bases and form mutagenic intermediates.</span></p></blockquote><center><h2><span style="color:#35a166">4.) Base excision repair: DNA N-Glycosylases</span></h2></center><ul><li><span>In addition to NER, there is another form of excision repair, </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015A23'><span>base excision repair (BER)</span></a><span>. </span></li><li><span>Like NER, BER also removes one or more nucleotides from a site of base damage.</span></li><li><span>However, this process begins with enzymatic cleavage of the glycosidic bond between the damaged base and deoxyribose.</span></li></ul><h3><span style="color:#63C0F5">Replacement of Uracil in DNA by BER</span></h3><ul><li><span>One of the best-understood BER systems scans DNA to remove uracil. </span></li><li><span>Uracil can base-pair with adenine in a DNA duplex, and DNA polymerases can readily accept deoxyuridine triphosphate as a substrate in place of  thymidine triphosphate. </span></li><li><span>Yet cells possess an elaborate two-stage process that prevents dUMP residues from accumulating in DNA. </span></li><li><span>The first stage, described in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch19_pg0001.xhtml#P7001014825000000000000000004056'><span>Chapter 19</span></a><span>, involves cleavage by dUTPase of dUTP to dUMP, thereby minimizing the  dUTP pool and its use as a replication substrate. </span></li><li><span>The second stage involves uracil-DNA N-glycosylase (Ung), an enzyme that removes any dUMP residues in DNA that might have arisen either through deamination of a dCMP residue or through incorporation of a dUTP that escaped the action of dUTPase.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> A base excision repair process removes uracil residues in DNA, whether  they arose through the deamination of cytosine residues or the  incorporation of deoxyuridine nucleotides instead of thymidine  nucleotides.</span></p></blockquote><ul><li><span>As shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049C6'><span>Figure 23.9</span></a><span>, Ung hydrolytically cleaves the glycosidic bond between N-1 of uracil and C-1′ of deoxyribose. </span></li><li><span>This yields free uracil and DNA with an </span><strong><span>apyrimidinic site</span></strong><span> (i.e., a sugar residue lacking an attached pyrimidine). </span></li><li><span>Another enzyme, </span><strong><span>apyrimidinic endonuclease</span></strong><span> (AP endonuclease), recognizes this site and cleaves the phosphodiester bond on the 5′ side of the deoxyribose moiety. </span></li><li><span>This is followed in bacteria by the nick translation activity of DNA polymerase I (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0004.xhtml#P700101482500000000000000000476F'><span>Figure 22.14</span></a><span>), which inserts dTTP as a replacement for the dUMP that was removed,  displacing the deoxyribose phosphate residue in the AP site. </span></li><li><span>This is removed by a </span><strong><span>deoxyribose-5′-phosphatase</span></strong><span>. </span></li><li><span>The resulting nick is sealed by DNA ligase.</span></li></ul><ul><li><span>Ung, like all DNA glycosylases examined to date, acts by flipping the target base out of the DNA duplex and binding it in a pocket where cleavage  occurs. </span></li><li><span>Structural analysis of human UNG shows that the pocket is small enough to exclude purine bases. </span></li><li><span>More important, as shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049CD'><span>Figure 23.10</span></a><span>, the pocket excludes thymine because of negative steric interaction between the thymine methyl group at C5 and Tyr-147. (By convention,  eukaryotic genes and gene products are denoted with capital letters;  thus, </span><em><span>UNG</span></em><span> is the human gene that encodes the protein UNG. In bacteria, gene </span><em><span>ung</span></em><span> encodes protein Ung.)</span></li></ul><ul><li><span>Why do cells go to all this trouble just to replace a nucleotide that does not affect the information encoded in DNA?</span></li><li><span>Almost certainly, U-A base pairs are not the true target of this DNA repair system. </span></li><li><span>Uracil residues in DNA also arise through spontaneous deamination of cytosine residues (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P7001014825000000000000000004970'><span>Figure 23.3</span></a><span>). </span></li><li><span>This alteration </span><em><span>does</span></em><span> change the genetic sense because it converts a G-C base pair to a G-U  pair, and in a subsequent round of replication the U-containing strand would give rise to an A-T base pair—a GC-to-AT </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015E58'><span>transition</span></a><span> (change of a purine-pyrimidine pair to another purine-pyrimidine). </span></li><li><span>The uracil repair system prevents this mutation but does not discriminate between uracils paired with adenines or with guanines. </span></li><li><span>Consistent with this model is the </span><strong><span>hypermutable</span></strong><span> phenotype displayed by mutant bacteria lacking an active Ung. </span></li><li><span>Such strains exhibit elevated rates of spontaneous mutagenesis, resulting from the accumulation of DNA dUMP residues paired with dGMP.</span></li></ul><ul><li><span>The replacement of U by T in DNA is one example of BER. </span></li><li><span>Another example, described next, involves repair of damage caused by reactive oxygen  species.</span></li></ul><h3><span style="color:#63C0F5">Repair of Oxidative Damage to DNA</span></h3><ul><li><span>Most cells contain several DNA-N-glycosylases, including those specific for the alkylated bases N-methyladenine, 3-methyladenine, and 7-methylguanine. </span></li><li><span>Of particular interest is the BER process used to repair oxidative DNA damage. </span></li><li><span>As was indicated in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P7001014825000000000000000004969'><span>Figure 23.2</span></a><span>, one of the most abundant of the numerous oxidation products resulting from DNA exposure to ROS is 8-oxoguanine. </span></li><li><span>This is a strongly mutagenic alteration because 8-oxoguanine pairs readily with adenine.</span></li></ul><ul><li><span>Oxidation of a G paired with C gives rise to a C-oxoG base pair. </span></li><li><span>A subsequent round of replication readily gives rise to an A-oxoG pair, which then becomes A-T in the next round. </span></li><li><span>Hence, the C-oxoG mispair can be an intermediate in a </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015E61'><span>transversion</span></a><span> mutation C‐G → C‐oxoG → A‐oxoG → A‐T. (A </span><em><span>transversion</span></em><span> changes a pyrimidine-purine base pair to a purine-pyrimidine pair.) </span></li><li><span>In </span><em><span>E. coli</span></em><span>, three genes encode proteins that minimize oxidative mutagenesis: </span><em><span>mutM, mutY,</span></em><span> and </span><em><span>mutT.</span></em><span> (The “mut” designation indicates that mutations in any one of these genes have the effect of increasing the spontaneous mutation rate.) </span></li><li><span>All three gene products have mammalian homologs. MutM and MutY  are both DNA-base glycosylases. </span></li><li><span>MutM (or its mammalian homolog, OGG1), which we describe here, cleaves 8-oxoguanine from DNA to initiate BER.  </span></li><li><span>The repair process, similar to DNA-uracil repair, results in replacement of oxoG by G.</span></li></ul><ul><li><span>Like UNG, described earlier, human OGG1 DNA glycosylase distinguishes target oxoG bases from nontarget bases such as guanine by excluding nontarget bases from the active site where cleavage occurs. </span></li><li><span>Structural analysis of human OGG1 revealed that 8-oxoguanine is able to enter the binding pocket, while guanine is completely excluded due to both attractive and repulsive forces (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049DD'><span>Figure 23.11</span></a><span>).</span></li></ul><center><h2><span style="color:#35a166">5.) Mismatch Repair</span></h2></center><ul><li><span>Mismatches, or non-Watson–Crick base pairs in a DNA duplex, can arise through replication errors, through deamination of 5-methylcytosine in DNA to yield thymine, or through recombination between DNA segments that  are not completely homologous. </span></li><li><span>In addition, mismatches result when DNA  polymerase encounters a short repeated sequence and slides along its  template, creating short loops or bulges in duplex DNA (see </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0003.xhtml#P7001014825000000000000000004728'><span>Figure 22.7</span></a><span>). </span></li><li><span>We best understand the correction of replication errors, so that is what we describe here.</span></li></ul><ul><li><span>If DNA polymerase introduces an incorrect nucleotide, creating a  non-Watson–Crick base pair, the error is usually corrected by the  polymerase-associated 3′ exonuclease activity (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0001.xhtml#P70010148250000000000000000046BB'><span>Chapter 22</span></a><span>). </span></li><li><span>If the error is not corrected immediately, the fully replicated DNA will contain a mismatch at that site.</span></li><li><span>This error can be corrected by </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015C98'><span>mismatch repair</span></a><span>. </span></li><li><span>In </span><em><span>E. coli</span></em><span>, the proteins that participate include the products of genes </span><em><span>mutH, mutL,</span></em><span> and </span><em><span>mutS,</span></em><span> plus an enzyme called helicase II.</span></li></ul><ul><li><span>The mismatch correction system scans newly replicated DNA, looking for both mismatched bases and single-base insertions or deletions. </span></li><li><span>MutS binds to DNA at the site of the mismatch, followed by the binding of MutL and  then MutH. MutS is a “motor protein,” which uses the energy of ATP hydrolysis to pull DNA from both directions until it reaches the site at which repair is to begin. </span></li><li><span>When it finds an appropriate signal, part of one strand containing the mismatched region is cut out and replaced (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049EB'><span>Figure 23.12</span></a><span>).</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> The mismatch repair system in bacteria uses DNA methylation to identify the strand that has a mispaired nucleotide.</span></p></blockquote><ul><li><span>How does the mismatch repair system recognize the correct strand to repair? </span></li><li><span>If it chose either strand randomly, it would choose incorrectly half  the time and there would be negligible gain in replication accuracy. </span></li><li><span>The mismatch repair enzymes identify the newly replicated strand because for a short period that DNA is unmethylated. </span></li><li><span>In </span><em><span>E. coli</span></em><span>, the sequence –GATC– is crucial because that is the site methylated soon after replication, by the product of the </span><em><span>dam</span></em><span> gene (DNA adenine methylase). </span></li><li><span>The mismatch repair enzymes look for  –GATC– sequences that are not methylated. </span></li><li><span>Recognition of an unmethylated GATC can target that strand for mismatch correction at a site as far as 1 kbp between the mismatch and the GATC site, in either direction. </span></li><li><span>Once the methylation system has acted on all GATC sites in the daughter strand, it is too late for the mismatch repair system to recognize the more recently synthesized DNA strand, and any advantage in total DNA replication fidelity is lost. </span></li><li><span>When the system functions properly, it has the effect of increasing overall replication fidelity by about 100-fold—from about 1 error in 107 base pairs replicated to about 1 in 109 or better.</span></li></ul><ul><li><span>As shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049EB'><span>Figure 23.12</span></a><span>, the MutHLS complex moves along DNA in both directions until it encounters the nearest 5′-GATC sequence. </span></li><li><span>An endonuclease activity of  MutH then cleaves on the 5′ side of the G in the unmethylated strand.</span></li><li><span>At that point helicase II unwinds the DNA, moving back past the mismatch, followed by an exonuclease that digests the displaced single strand. </span></li><li><span>The resultant gap is filled by DNA polymerase III holoenzyme and DNA  ligase, working in concert with SSB.</span></li></ul><ul><li><p><span>A similar mismatch repair system exists in eukaryotic cells. </span></p></li><li><p><span>The mismatch recognition step is somewhat more complex, however, because three  different MutS homologs (MSH proteins) are involved—MSH2, MSH3, and MSH6. </span></p></li><li><p><span>These three proteins  form heterodimers, with specificities for different types of mismatches. </span></p></li><li><p><span>Still unknown is the mechanism by which eukaryotic systems recognize  and initiate repair on the newly replicated DNA strand;</span></p><ul><li><span>it is clear that DNA methylation is not involved.</span></li></ul></li><li><p><span>Recently, it has been proposed that ribonucleotides incorporated by imperfect specificity of DNA polymerase (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0001.xhtml#P70010148250000000000000000046BB'><span>Chapter 22</span></a><span>) could provide such a signal. </span></p></li><li><p><span>In the parental strands, any ribonucleotide incorporation would have been repaired.</span></p></li></ul><ul><li><span>Just as is seen with bacteria, mutations in eukaryotic genes that control mismatch repair confer a mutator phenotype, raising spontaneous mutation rates at all loci. </span></li><li><span>How do such mutations affect the biology of human cells? </span></li><li><span>As mentioned in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch20_pg0001.xhtml#P700101482500000000000000000427B'><span>Chapter 20</span></a><span>, the progression of a normal cell to a cancer cell involves the accumulation of multiple mutations in tumor cell precursors. </span></li><li><span>Cells  lacking mismatch repair activity do have elevated mutation rates, so it was a logical development when mutations in mismatch repair genes were found in tumor cells from individuals with a heritable cancer predisposition called HNPCC (heritable nonpolyposis colon cancer). </span></li><li><span>To date, germ-line mutations in the genes for five different mismatch repair proteins have been found to be associated with HNPCC. </span></li><li><span>The most common forms of the disease involve mutations affecting either hMLH1 (human Mut L homolog) or hMHS2 (human MutS homolog). </span></li><li><span>The cancer predisposition is inherited in an autosomal dominant fashion, suggesting that most affected  individuals are heterozygous, with one wild-type and one nonfunctional allele. </span></li><li><span>Mismatch repair is normal until a somatic cell mutation inactivates the one functional allele and mismatch repair capacity is  essentially abolished, with a consequent increase in spontaneous mutagenesis. </span></li><li><span>It is unclear why mutations affecting mismatch repair are  associated specifically with tumors in the colon.</span></li></ul><blockquote><p><strong><span>Connection</span></strong><span> Genetic deficiencies in mammalian mismatch repair are often associated with colorectal cancer.</span></p></blockquote><ul><li><span>Tumor cells from those affected with HNPCC exhibit </span><strong><span>microsatellite instability</span></strong><span>—a phenomenon in which there are large numbers of mutations in regions of the genome containing repeats of single-, double-, and triple-nucleotide sequences, usually with large increases in the numbers of repeating units in such sequences. </span></li><li><span>These data suggest that the product and template strands can normally slip at such sites, so that DNA polymerase copies a short repeating sequence more than once, or else skips a segment. </span></li><li><span>This creates a heteroduplex with a short loop, as we showed in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0001.xhtml#P70010148250000000000000000046BB'><span>Chapter 22</span></a><span> (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0003.xhtml#P7001014825000000000000000004728'><span>Figure 22.7</span></a><span>) for replication errors leading to Huntington’s disease. </span></li><li><span>Normally, a replication error of this type would be corrected by mismatch repair, but such errors persist and accumulate in a cell lacking normal mismatch repair. </span></li><li><span>Studies of this type have given scientists insight into the  nature of cancer as a progressive genetic disease.</span></li></ul><center><h2><span style="color:#35a166">6.) Double-Strand Break Repair</span></h2></center><ul><li><span>A double-strand break (DSB), a lesion that can be caused by ionizing radiation or by replication stalling, is the most lethal form of DNA  damage because it destroys the physical integrity of a chromosome. </span></li><li><span>DSBs occur about 50 times per mammalian cell cycle. </span></li><li><span>Moreover, double-stranded DNA breaks occur naturally during meiotic recombination. </span></li><li><span>DSBs can be repaired by either of two processes—</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015BE5'><span>homologous recombination</span></a><span> (HR), using sequence information from an undamaged sister chromatid, or </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015CCB'><span>nonhomologous end joining</span></a><span> (NHEJ). (From the standpoint of evolution, </span><em><span>homologous</span></em><span> means “derived from a common ancestor,” but we use it in a broader sense, meaning “having similar sequences.”)</span></li></ul><ul><li><span>NHEJ is more efficient, but if DNA ends are not rejoined at the precise  sites where breakage occurred, genetic information becomes lost or scrambled. </span></li><li><span>By contrast, HR normally repairs the broken site precisely, but it can occur only during the S or G2 cell-cycle phases, when a homologous chromosome is available. </span></li><li><span>Both processes begin with several signaling proteins associating at the  severed ends. </span></li><li><span>A principal player in vertebrates is a protein kinase called ATM (ataxia telangiectasia mutated).</span></li></ul><ul><li><span>An early event in both HR and NHEJ is phosphorylation of a variant form of H2 histone, H2AX, in nucleosomes near the break. </span></li><li><span>This may be part of a chromatin remodeling process that helps to move core particles out of  the way, exposing DNA ends for processing.</span></li></ul><ul><li><span>Double-strand break repair is better understood for homologous recombination, and that is what we outline here. </span></li><li><span>Although it is simple to visualize conceptually, as shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P7001014825000000000000000004A01'><span>Figure 23.13</span></a><span>, </span></li><li><span>HR is a complex process, involving extensive signaling reactions as part of the </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015AD5'><span>DNA damage response</span></a><span> (see page </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P7001014825000000000000000004A0A'><span>725</span></a><span>). </span></li><li><span>As studied in yeast, a signaling complex called ATM activates a nuclease in another complex called MRN, as well as signaling downstream effectors via an associated protein kinase activity. </span></li><li><span>The MRN-associated nuclease trims away 5′ DNA ends, leaving 3′-terminated single-stranded ends, which become coated with RPA (replication protein A; single-strand binding protein). </span></li><li><span>The BRCA2 protein (page </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P70010148250000000000000000049A6'><span>720</span></a><span>) is involved in this process.</span></li><li><span>RNA polymerase II transcribes the  single-stranded DNA ends, and ribonuclease H degrades the RNA in the hybrid. </span></li><li><span>RPA then coats the 3′ DNA ends. </span></li><li><span>Not shown is the binding of Rad51, a protein that scans the single-stranded DNA ends and the homologous undamaged chromosome, seeking regions of sequence homology,  thereby facilitating base pairing and subsequent ligation. </span></li><li><span>Rad51 is the eukaryotic counterpart of bacterial RecA, whose action is described on  page </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A48'><span>728</span></a><span>.</span></li></ul><blockquote><p><strong><span>Connection</span></strong><span> Genetic defects involving double-stranded break repair, particularly in genes  BRCA1 and BRCA2, are risk factors for breast or ovarian cancer.</span></p></blockquote><ul><li><span>Human mutations are known that affect several of the proteins involved in DSB repair. </span></li><li><span>As mentioned previously, mutations in either BRCA1 or BRCA2 are risk factors for developing breast or ovarian cancer. </span></li><li><span>BRCA2 mutations are also associated with Fanconi’s anemia, a rare  condition characterized by bone marrow failure. </span></li><li><span>ATM received its name from the deficiency syndrome ataxia telangiectasia, which involves  premature aging, cerebellar degeneration, and enhanced cancer  susceptibility.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> Double-strand DNA breaks can be repaired either by homologous recombination (HR) or  by nonhomologous end joining (NHEJ), a process that does not require DNA sequence homology at the ends being joined.</span></p></blockquote><center><h2><span style="color:#35a166">7.) Daughter-Strand Gap Repair</span></h2></center><ul><li><span>Bacteria also use HR as a way to repair DNA damage. </span></li><li><span>However, because bacteria have but a single chromosome, the undamaged duplex needed as a repair template is the replicated portion of an incompletely replicated chromosome. </span></li><li><span>This process, called </span><strong><span>daughter-strand gap repair</span></strong><span>, comes into play if DNA damage is sufficient to exceed the cell’s capacity for photoreactivation or excision repair. </span></li><li><span>The process is schematized in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P7001014825000000000000000004A10'><span>Figure 23.14</span></a><span>.</span></li></ul><ul><li><span>When a replisome encounters a thymine dimer or a bulky lesion, it cannot  replicate past this site. </span></li><li><span>A recombination-related process can take place, not to repair the damage, but to allow replication to continue, saving the damaged site for subsequent repair by another process.  </span></li><li><span>Daughter-strand gap repair depends on the RecA protein mentioned earlier. </span></li><li><span>RecA, like its eukaryotic counterpart, Rad51, catalyzes </span><strong><span>strand pairing</span></strong><span>, or </span><strong><span>strand assimilation</span></strong><span>—the joining of two different DNAs by homologous base pairing.</span></li></ul><center><h2><span style="color:#35a166">8.) Translesion Synthesis and the DNA Damage Response</span></h2></center><ul><li><span>With the exception of direct-reversal repair processes like photoreactivation, DNA repair requires the action of a polymerase to fill in a single-strand gap in the DNA duplex. </span></li><li><span>If the gap contains a  damaged structure, like a thymine dimer or a bulky lesion, a replicative polymerase is unable to copy past the damaged site, and the repair process aborts. </span></li><li><span>A new set of damage-tolerant DNA polymerases, inducible by DNA damage, comes into play. </span></li><li><span>Recall from </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0001.xhtml#P70010148250000000000000000046BB'><span>Chapter 22</span></a><span> that bacteria contain five DNA polymerases, only two of which are  involved in replication, while eukaryotic cells have as many as nine  damage-inducible polymerases. </span></li><li><span>Most of these enzymes have low fidelity, partly because they lack 3′ exonucleolytic proofreading and partly because their catalytic sites are larger and more open than those of high-fidelity replicative polymerases. </span></li><li><span>The enzymes are inducible as a result of DNA damage, but the control mechanisms differ markedly between bacterial and eukaryotic organisms.</span></li></ul><ul><li><span>The </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015E03'><span>SOS response</span></a><span> in bacteria is a metabolic alarm system that helps the cell to save  itself in the presence of potentially lethal stresses. </span></li><li><span>Inducers of the  SOS response include DNA damaging agents such as ultraviolet irradiation, thymine starvation, and inactivation of genes essential to DNA replication. </span></li><li><span>Responses include mutagenesis, filamentation (in which  cells elongate by growth but don’t divide), and activated excision repair. Mutagenesis occurs because, under SOS conditions, the gaps that  are formed are filled by inaccurate DNA polymerases. </span></li><li><span>In fact, this  process is the principal pathway by which ultraviolet light stimulates mutagenesis in bacteria.</span></li></ul><ul><li><span>Induction of the SOS response by UV light or other DNA damage activates transcription of about 40 genes in </span><em><span>E. coli.</span></em><span> </span></li><li><span>We discuss the mechanism of transcriptional activation in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch26_pg0001.xhtml#P700101482500000000000000000500E'><span>Chapter 26</span></a><span>. </span></li><li><span>Among these SOS-inducible genes are those encoding DNA polymerases IV and V. </span></li><li><span>Both of these enzymes are highly error-prone. </span></li><li><span>After SOS  activation, the processing of two gene products (UmuC and UmuD) gives  rise to a pol V molecule. </span></li><li><span>This associates with RecA bound to ATP, giving the active form of the enzyme. </span></li><li><span>When the replisome containing pol III holoenzyme stalls at a damage site, pol V replaces it and binds to the  sliding clamp. </span></li><li><span>Once the damaged site has been copied (inaccurately), pol V somehow steps aside, and pol III holoenzyme completes the replication process. Pol IV functions similarly, but we know less about it.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> DNA can be repaired after replication, either by recombination or by inducible error-prone repair. Both processes require RecA.</span></p></blockquote><ul><li><span>Considering that most mutations are probably deleterious, what is the advantage to the cell of accumulating mutations during DNA repair? </span></li><li><span>Probably none, except that the alternative would be for the cell to die. </span></li><li><span>In other words, extensive mutagenesis is a worthwhile price for the cell to pay, if a massive UV dose has overwhelmed all other repair pathways and if error-prone replication past a dimer is necessary for the cell to survive. </span></li><li><span>As noted earlier, translesion synthesis is inducible in eukaryotic cells as well. </span></li><li><span>Recall from </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0001.xhtml#P70010148250000000000000000046BB'><span>Chapter 22</span></a><span> that eukaryotic cells were formerly thought to have five DNA polymerases. </span></li><li><span>Since the late 1990s, a remarkable variety of new DNA polymerases has been discovered in eukaryotic cells, and the total number of human DNA polymerases now stands at 15. </span></li><li><span>Four of the newly  discovered polymerases—Pol </span><em><span>η</span></em><span>, Pol </span><em><span>ι</span></em><span>, Pol </span><em><span>κ</span></em><span>, and REV1—are specialized to replicate past specific types of DNA  damage. </span></li><li><span>These enzymes all have generally low fidelity, with the  exception of pol </span><em><span>η</span></em><span>, which can replicate past thymine dimers in an essentially error-free manner.</span></li></ul><ul><li><span>Induction of some or all of the repair polymerases in eukaryotic cells is part of the DNA damage response, a complex set of events involving a number of signaling pathways, which we introduced on page </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P7001014825000000000000000004A0A'><span>725</span></a><span>. </span></li><li><span>At least 700 proteins undergo phosphorylation during this process,  catalyzed by ATM and ATR and other kinases. </span></li><li><span>Depending on the severity of DNA damage, the cell type, and the phase of the cell cycle, the cellular responses may activate DNA repair pathways, cell-cycle arrest, or programmed cell death (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015A07'><span>apoptosis</span></a><span>). </span></li><li><span>We have previously mentioned p53 (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch20_pg0001.xhtml#P700101482500000000000000000427B'><span>Chapter 20</span></a><span>), an effector that acts as a transcription factor whose effects include either cell-cycle arrest, until damage has been repaired, or apoptosis  if damage is too extensive for repair. </span></li><li><span>Other effectors include cyclin-dependent kinases, protein kinases that regulate the cell cycle.</span></li></ul><blockquote><p><strong><span>Connection</span></strong><span> Inaccurate replication past damaged sites in DNA leads to mutation, and this is  the primary pathway by which UV irradiation causes skin cancers.</span></p></blockquote><center><h2><span style="color:#35a166">9.) Recombination</span></h2></center><ul><li><span>Population genetics teaches us that the survival of a species depends on its  ability to maintain genetic diversity, so that individuals can vary in their ability to respond to unforeseen environmental pressures.  </span></li><li><span>Diversity is maintained through mutation, which alters single genes or small groups of genes in an individual, and recombination, which redistributes the contents of a genome among various progeny during  reproduction.</span></li></ul><ul><li><span>As initially described by cytogeneticists, recombination results from crossing over between paired homologous chromosomes during meiosis in eukaryotes.</span></li><li><span>Recombination, however, encompasses more processes and biological functions than those involved in sexual reproduction.  </span></li><li><span>Strictly speaking, </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015DA8'><span>recombination</span></a><span> is any process that involves the formation of new DNA from distinct DNA molecules, such that genetic information from each parental DNA molecule is present in the new molecules. </span></li><li><span>The double-strand break repair and daughter-strand gap repair processes described earlier are forms of recombination. </span></li><li><span>So, too, is the formation of recombinant genotypes in  bacterial conjugation—and, of course, recombinant DNA molecules  constructed in vitro.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> Recombination is any process that creates end-to-end joining from two different DNA molecules.</span></p></blockquote><ul><li><span>We begin our discussion with </span><strong><span>site-specific recombination</span></strong><span>, which was initially studied as a model for homologous recombination; </span></li><li><span>It is important for the insight it provides into infection of cells by  some viruses, including retroviruses.</span></li></ul><center><h2><span style="color:#35a166">10.) Site-Specific Recombination</span></h2></center><ul><li><span>Many DNA viruses maintain a long-term relationship with the cells that they infect by inserting their genome into one or more chromosomes of the infected cell. </span></li><li><span>Retroviruses carry out a similar process, which was  outlined for HIV in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch20_pg0001.xhtml#P700101482500000000000000000427B'><span>Chapters 20</span></a><span> and </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch22_pg0001.xhtml#P70010148250000000000000000046BB'><span>22</span></a><span>; in this process, the RNA genome makes a DNA copy, which can then become inserted into host-cell chromosomes. </span></li><li><span>The mechanism of viral genome integration is best understood for bacteriophage </span><em><span>λ</span></em><span>, and that is what we describe here.</span></li></ul><ul><li><span>Phage T4 is a </span><em><span>lytic</span></em><span> phage. </span></li><li><span>After infecting a bacterial cell, a cycle of virus multiplication always occurs, followed by lysis of the cell, which releases hundreds of new viral particles.</span></li><li><span>Phage λ, by contrast, is a </span><em><span>temperate</span></em><span> phage, which does not always kill its host. Infection can lead either to a lytic growth cycle, like T4, or a </span><em><span>lysogenic</span></em><span> response, in which the viral chromosome becomes linearly inserted into  the bacterial chromosome; once inserted, it can remain in a quiescent  state, called a </span><em><span>prophage,</span></em><span> for many generations. </span></li><li><span>A change in environmental conditions can trigger the expression of genes that direct excision of the viral chromosome from that of the host and turn on genes leading to virus multiplication and eventual lysis of the cell. </span></li><li><span>Understanding these processes contributed importantly to our understanding of transcriptional regulation, which we will discuss in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch26_pg0001.xhtml#P700101482500000000000000000500E'><span>Chapter 26</span></a><span>.</span></li></ul><ul><li><span>The phage </span><em><span>λ</span></em><span> genome, as it exists in phage particles, is a linear duplex DNA molecule with short single-stranded tails at each 5′ end. </span></li><li><span>These are called </span><em><span>cohesive ends,</span></em><span> or sticky ends, because they are complementary in sequence. </span></li><li><span>Hence, the molecule can circularize by base pairing between the ends. </span></li><li><span>The resultant nicks are closed by DNA ligase. In an infection leading to lysogeny, the covalently closed </span><em><span>λ</span></em><span> circle becomes linearly inserted into the </span><em><span>E. coli</span></em><span> host-cell chromosome, as schematized in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A31'><span>Figure 23.15</span></a><span>. </span></li><li><span>A virally encoded protein called </span><em><span>integrase</span></em><span> recognizes specific sequences in the viral and bacterial chromosomes, which are brought together by their joint attraction for integrase.</span></li></ul><ul><li><span>Unlike homologous recombination, which requires several hundred base pairs of sequence homology in order to occur, site-specific recombination requires fewer than two dozen homologous base pairs. </span></li><li><span>In the case of phage </span><em><span>λ</span></em><span> (see </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A31'><span>Figure 23.15</span></a><span>), sequences in the phage and bacterial genomes, called </span><em><span>attP</span></em><span> and </span><em><span>attB,</span></em><span> respectively, bind to specific sites on the integrase molecule, which  catalyzes strand breaking and resealing reactions that linearly insert  the viral chromosome into the chromosome of the infected </span><em><span>E. coli</span></em><span> cell. </span></li><li><span>In this process, the sites of DNA recognition, breakage, and joining are brought about by specific DNA–protein interactions rather  than by the DNA–DNA interactions via extensive sequence homology that categorize homologous recombination.</span></li></ul><center><h2><span style="color:#35a166">11.) Homologous Recombination</span></h2></center><h3><span style="color:#63C0F5">Breaking and Joining of Chromosomes</span></h3><ul><li><span>The most straightforward way to accomplish recombination is to break and  rejoin DNA molecules. </span></li><li><span>However, if recombination occurs this way, the  sites of breakage must be precisely the same on both recombining chromosomes for intact genes to be regenerated. </span></li><li><span>Some researchers favored alternative mechanisms, but in 1961 Matthew Meselson and Jean Weigle showed that recombination in fact does occur via breakage and rejoining of chromosomes. </span></li><li><span>The demonstration involved a Meselson–Stahl type of experiment (see </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch04_pg0004.xhtml#P70010148250000000000000000012BE'><span>Figure 4.17</span></a><span>). </span></li><li><em><span>E. coli</span></em><span> was infected with two genetically distinct </span><em><span>λ</span></em><span> phage populations, one of which had been density labeled by growth in 13C–15N medium. </span></li><li><span>The phage particles resulting from this mixed infection were centrifuged to equilibrium in a cesium chloride gradient. </span></li><li><span>Phages with recombinant genotypes were recovered from all parts of the gradient, whereas nonrecombinant phages were uniformly light or heavy. </span></li><li><span>This result means that recombinant phages contained DNA derived from both parents,  which could have occurred only by breaking and rejoining DNA strands.</span></li></ul><ul><li><span>Recombination is stimulated by processes that nick or break DNA strands, such as  thymidine starvation or UV irradiation. </span></li><li><span>This suggested that single-stranded DNA, or free DNA ends, play a role in initiating recombination.</span></li></ul><h3><span style="color:#63C0F5">Models for Recombination</span></h3><ul><li><span>In 1964, Robin Holliday proposed a model for homologous recombination between duplex DNA molecules. </span></li><li><span>Detailed in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A41'><span>Figure 23.16</span></a><span>, Holliday’s model continues to dominate thinking and experiments about this process.</span></li></ul><ul><li><span>Holliday proposed that recombination begins with nicking at the same site on two paired chromosomes (step </span><strong><span>1</span></strong><span>). </span></li><li><span>Partial unwinding of the duplexes is followed by </span><strong><span>strand invasion</span></strong><span>, in which a free single-strand 3′ end from one duplex pairs with its  unbroken complementary strand in the other duplex, and vice versa (step </span><strong><span>2</span></strong><span>). </span></li><li><span>Enzymatic ligation generates a crossed-strand intermediate, called a </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/glossary.xhtml#P7001014825000000000000000015BDF'><span>Holliday junction</span></a><span> (step </span><strong><span>3</span></strong><span>). </span></li><li><span>The crossed-strand structure can move in either direction by duplex unwinding and rewinding (branch migration, step </span><strong><span>4</span></strong><span>). </span></li><li><span>The Holliday junction “resolves” itself into two unbroken duplexes by a process of strand breaking and rejoining. </span></li><li><span>The process leading to recombination begins with isomerization of the Holliday structure (step </span><strong><span>5</span></strong><span>), followed by strand breakage, so that the strands that break (in step </span><strong><span>9</span></strong><span>) are those that were </span><em><span>not</span></em><span> broken in step 1. </span></li><li><span>Resolution of the resulting structure (steps </span><strong><span>10</span></strong><span> and </span><strong><span>11</span></strong><span>) generates two chromosomes recombinant for DNA flanking the region and  each containing a heteroduplex region (each strand from a different  parental molecule). </span></li><li><span>However, if the original crossed strands (those that </span><em><span>were</span></em><span> broken in step 1) break and rejoin (steps </span><strong><span>6</span></strong><span>, </span><strong><span>7</span></strong><span> and </span><strong><span>8</span></strong><span>), the products are nonrecombinant duplexes, each containing a  heteroduplex region (i.e., nonrecombinant with respect to the outside  markers A and Z).</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> Homologous recombination breaks and rejoins chromosomes.</span></p></blockquote><ul><li><span>Considerable evidence now supports the central tenets of the Holliday model, including electron microscopic visualization of Holliday junctions (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A4C'><span>Figure 23.17</span></a><span>). </span></li><li><span>However, a major problem with the Holliday model was the requirement that two duplex DNAs undergo single-strand nicking at precisely the same points. </span></li><li><span>How might that problem be resolved? </span></li><li><span>Matthew Meselson and Charles Radding proposed that recombination could start with a single nick. The 3′ end of the nick invades the unbroken duplex and initiates strand displacement synthesis, with the displaced single strand then pairing with its partner in the unbroken strand, as indicated in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A52'><span>Figure 23.18</span></a><span>.</span></li></ul><h3><span style="color:#63C0F5">Proteins Involved in Homologous Recombination</span></h3><ul><li><span>The Holliday and Meselson–Radding models explained most of the existing data on homologous recombination between paired chromosomes, particularly as studied in lower eukaryotes such as yeast. </span></li><li><span>Moreover, the models could easily be adapted to explain daughter-strand gap repair or the recombination that occurs in bacteria after transformation or conjugation. Some of the proteins thought to participate, notably DNA  polymerase, DNA ligase, and single-strand DNA-binding protein, had been characterized and shown to participate in recombination.</span></li></ul><ul><li><span>What other proteins might participate in recombination if the models are largely correct? </span></li><li><span>To answer this question, we turn back to </span><em><span>E. coli</span></em><span> and its phages and the characteristics of bacterial mutants defective in recombination. </span></li><li><span>Mutations conferring a recombination-defective (</span><em><span>rec</span></em><span>−) phenotype map in several loci, and two important proteins are  responsible for most bacterial recombination events. </span></li><li><span>One of these gene  products, the RecA protein, was introduced earlier. </span></li><li><span>The other protein is called exonuclease V, or the RecBCD nuclease.</span></li></ul><ul><li><span>RecA is a multifunctional protein with molecular weight of about 38,000. </span></li><li><span>As we will see in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch26_pg0001.xhtml#P700101482500000000000000000500E'><span>Chapter 26</span></a><span>, it participates in regulating the SOS response. In recombination, it promotes the ATP-dependent pairing of homologous strands, as mentioned  earlier in connection with daughter-strand gap repair.</span></li></ul><ul><li><span>As schematized in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A5D'><span>Figure 23.19</span></a><span> (left panel), the process begins with a reaction between RecA and  single-stranded (ss) DNA, to give a RecA-ssDNA filament.</span></li><li><span>RecA wraps  about ssDNA as a multisubunit right-handed helix, with six RecA monomers per turn. </span></li><li><span>Once ssDNA is bound, the filament searches double-stranded  (ds) DNA, seeking sequences complementary to those in the single strand already bound. </span></li><li><span>In this process, dsDNA is also taken up within the filament, giving a “joint molecule” (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A5D'><span>Figure 23.19</span></a><span>, center panel). </span></li><li><span>Binding to dsDNA requires ATP, but that ATP need not undergo hydrolysis. </span></li><li><span>By contrast, movement of the ssDNA-protein complex with respect to the dsDNA </span><em><span>does</span></em><span> require ATP hydrolysis. </span></li><li><span>During this process, dsDNA becomes underwound and stretched to about 1.5 times its normal length. </span></li><li><span>The complex moves in a 5′-to-3′ direction along the initially bound ssDNA. During the movement, a triple-stranded structure transiently forms, with ssDNA (the red strand in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A5D'><span>Figure 23.19</span></a><span>) wound in the minor groove of the dsDNA. </span></li><li><span>The ssDNA continually tests the antiparallel strand in dsDNA (yellow) for sequence complementarity.</span></li><li><span>It is not clear how this occurs, but evidently short oligonucleotide sequences swing out from the duplex structure and can pair with ssDNA if sequence complementarity is found.</span></li></ul><ul><li><span>Once complementarity is established, branch migration occurs, with simultaneous strand exchange. </span></li><li><span>A duplex is formed between the red strand (ssDNA originally) and the yellow strand (complementary to the red  strand), while the displaced (green) strand is spooled out, away from the complex. </span></li><li><span>ATP hydrolysis during this period may promote rotation of  the DNA within the filament, facilitating the release of the displaced  strand.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> RecA, a multifunctional bacterial enzyme, uses ATP to promote the pairing of homologous DNA sequences.</span></p></blockquote><ul><li><span>Recombination occurs preferentially at or near particular DNA sequences. </span></li><li><span>In </span><em><span>E. coli,</span></em><span> recombination is favored near a particular octanucleotide sequence,  5′-GCTGGTCC, called Chi (for crossover hotspot instigator). </span></li><li><span>How does  this site act to stimulate recombination? </span></li><li><span>The RecBCD protein, a multifunctional heterotrimeric enzyme encoded by the </span><em><span>recB, recC,</span></em><span> and </span><em><span>recD</span></em><span> genes, displays sequence specificity for Chi. </span></li><li><span>This enzyme binds at a  double-strand break on duplex DNA and uses two helicase activities—RecB  and RecD—to unwind and partially degrade the DNA.</span></li></ul><ul><li><span>As shown from the crystal structure of a RecBCD-DNA complex, both helicases are in contact with DNA (</span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A68'><span>Figure 23.20</span></a><span>). </span></li><li><span>The RecD helicase activity is higher than that of RecB, so as the protein moves, the 3′ end is displaced as a single-stranded loop ahead of RecB and becomes coated with SSB protein. </span></li><li><span>Both strands are degraded by associated nucleases, but because of the differential speeds of the  protein motors, more of the 3′ end is saved as the loop. </span></li><li><span>When the enzyme reaches Chi, the protein pauses briefly, and a sequence-specific interaction causes RecBCD to switch speeds and change its preferred polarity of DNA degradation. </span></li><li><span>As RecB moves faster, the 3′-terminated  loop is reeled in by RecB and coated with RecA. </span></li><li><span>An associated nuclease releases the bound 3′ end (not shown), freeing it for strand invasion of a neighboring duplex.</span></li></ul><ul><li><span>Once a Holliday junction is formed, branch migration is essential for eventual formation of recombinant structures, as was shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A41'><span>Figure 23.16</span></a><span>. </span></li><li><span>This is largely the responsibility of three other proteins. </span></li><li><span>In </span><em><span>E. coli</span></em><span>, these three proteins are products of the </span><em><span>ruvA, ruvB,</span></em><span> and </span><em><span>ruvC</span></em><span> genes. </span></li><li><span>RuvA is a DNA-binding protein, whose specificity directs it toward the four-stranded Holliday structure. </span></li><li><span>RuvB protein is an ATP-requiring motor protein, which binds to two opposed arms of the  junction. </span></li><li><span>In the model shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0003.xhtml#P7001014825000000000000000004A70'><span>Figure 23.21</span></a><span>, which is based on crystal structures of the isolated RuvA and RuvB proteins, the two RuvB molecules act as twin pumps, rotating the two arms in opposite directions. </span></li><li><span>This forces branch migration by driving the rotational movement of the other two strands toward the junction. </span></li><li><span>Eventually, RuvC binds and begins the resolution of the Holliday structure by nicking two strands.</span></li></ul><blockquote><p><strong><span>Concept</span></strong><span> RecBCD, a multifunctional enzyme, unwinds and rewinds DNA, with one strand  being unwound more rapidly and converted to a single-stranded 3′ end.</span></p></blockquote><ul><li><span>Although Ruv protein homologs have not yet been detected in eukaryotic cells, much of the biochemistry of homologous recombination in eukaryotes is similar to what is described here. </span></li><li><span>In particular, the RAD51 protein of both human cells and yeast has a strand-pairing activity similar to that of RecA, and the two proteins show extensive sequence homology. </span></li><li><span>As noted earlier, an essential function of homologous recombination in eukaryotic cells is the repair of double-strand breaks, both during meiosis and as a result of DNA damage. </span></li><li><span>A broken chromosome can use the sequence information in its homolog to reconstruct the original DNA sequence at the site of the break, as was shown in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0002.xhtml#P7001014825000000000000000004A10'><span>Figure 23.14</span></a><span>.</span></li></ul><ul><li><span>Understanding the biochemistry of recombination is of far more than academic interest. </span></li><li><span>In recent years, scientists have learned how to direct the insertion of DNA into specific sites in mammalian genomes by use of  homologous recombination. </span></li><li><span>This targeted insertion allows the creation of a “knockout mouse,” in which any desired gene can be inactivated to allow investigation of the biological function of the deleted gene (see </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0005.xhtml#P7001014825000000000000000004B71'><span>Tools of Biochemistry 23A</span></a><span>). </span></li><li><span>The recent development of CRISPR-Cas9 technology has made it even simpler to obliterate any desired gene function (also described in </span><a href='https://etext-ise.pearson.com/eps/pearson-reader/api/item/f3bdb57a-3dee-4d1c-8832-7c5d915b21f7/1/file/appling-bcac-2e_eText_v5_060118/OPS/xhtml/ch23_pg0005.xhtml#P7001014825000000000000000004B71'><span>Tools of Biochemistry 23A</span></a><span>).</span></li></ul><blockquote><p><strong><span>Connection</span></strong><span> The ability to knock out the expression of any gene in a mouse by targeted  homologous recombination has led to important insights into human  diseases including cancer, obesity, heart disease, diabetes, arthritis,  aging, and Parkinson’s disease.</span></p></blockquote></div>
</body>
</html>