<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>

<style type='text/css'>html {overflow-x: initial !important;}:root { --bg-color: #ffffff; --text-color: #333333; --select-text-bg-color: #B5D6FC; --select-text-font-color: auto; --monospace: "Lucida Console",Consolas,"Courier",monospace; --title-bar-height: 20px; }
.mac-os-11 { --title-bar-height: 28px; }
html { font-size: 14px; background-color: var(--bg-color); color: var(--text-color); font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; -webkit-font-smoothing: antialiased; }
body { margin: 0px; padding: 0px; height: auto; inset: 0px; font-size: 1rem; line-height: 1.42857143; overflow-x: hidden; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; tab-size: 4; background-position: inherit; background-repeat: inherit; }
iframe { margin: auto; }
a.url { word-break: break-all; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { text-shadow: none; background: var(--select-text-bg-color); color: var(--select-text-font-color); }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; white-space: normal; overflow-x: visible; padding-top: 36px; }
#write.first-line-indent p { text-indent: 2em; }
#write.first-line-indent li p, #write.first-line-indent p * { text-indent: 0px; }
#write.first-line-indent li { margin-left: 2em; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
.typora-export .footnote-line, .typora-export li, .typora-export p { white-space: pre-wrap; }
.typora-export .task-list-item input { pointer-events: none; }
@media screen and (max-width: 500px) { 
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  #write { padding-left: 20px; padding-right: 20px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
#write li > figure:last-child { margin-bottom: 0.5rem; }
#write ol, #write ul { position: relative; }
img { max-width: 100%; vertical-align: middle; image-orientation: from-image; }
button, input, select, textarea { color: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-caps: inherit; font-weight: inherit; font-stretch: inherit; line-height: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
*, ::after, ::before { box-sizing: border-box; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p, #write pre { width: inherit; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p { position: relative; }
p { line-height: inherit; }
h1, h2, h3, h4, h5, h6 { break-after: avoid-page; break-inside: avoid; orphans: 4; }
p { orphans: 4; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
.md-math-block, .md-rawblock, h1, h2, h3, h4, h5, h6, p { margin-top: 1rem; margin-bottom: 1rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: 700; font-style: italic; }
a { cursor: pointer; }
sup.md-footnote { padding: 2px 4px; background-color: rgba(238, 238, 238, 0.7); color: rgb(85, 85, 85); border-top-left-radius: 4px; border-top-right-radius: 4px; border-bottom-right-radius: 4px; border-bottom-left-radius: 4px; cursor: pointer; }
sup.md-footnote a, sup.md-footnote a:hover { color: inherit; text-transform: inherit; text-decoration: inherit; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; }
figure { overflow-x: auto; margin: 1.2em 0px; max-width: calc(100% + 16px); padding: 0px; }
figure > table { margin: 0px; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 32px; }
.CodeMirror-gutters { border-right-width: 0px; background-color: inherit; }
.CodeMirror-linenumber { }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
#write pre { white-space: pre-wrap; }
#write.fences-no-line-wrapping pre { white-space: pre; }
#write pre.ty-contain-cm { white-space: normal; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; position: relative !important; background-position: inherit; background-repeat: inherit; }
.md-fences-adv-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
#write .md-fences.mock-cm { white-space: pre-wrap; }
.md-fences.md-fences-with-lineno { padding-left: 0px; }
#write.fences-no-line-wrapping .md-fences.mock-cm { white-space: pre; overflow-x: auto; }
.md-fences.mock-cm.md-fences-with-lineno { padding-left: 8px; }
.CodeMirror-line, twitterwidget { break-inside: avoid; }
.footnotes { opacity: 0.8; font-size: 0.9rem; margin-top: 1em; margin-bottom: 1em; }
.footnotes + .footnotes { margin-top: 0px; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; line-height: normal; font-weight: 400; text-align: left; box-sizing: content-box; direction: ltr; background-position: 0px 0px; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li .mathjax-block, li p { margin: 0.5rem 0px; }
li blockquote { margin: 1rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child, li > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; white-space: normal; }
#write .footnote-line { white-space: pre-wrap; }
@media print { 
  body, html { border: 1px solid transparent; height: 99%; break-after: avoid; break-before: avoid; font-variant-ligatures: no-common-ligatures; }
  #write { margin-top: 0px; padding-top: 0px; border-color: transparent !important; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  .typora-export #write { break-after: avoid; }
  .typora-export #write::after { height: 0px; }
  .is-mac table { break-inside: avoid; }
  .typora-export-show-outline .typora-export-sidebar { display: none; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 0.8rem; white-space: pre-wrap; background-color: rgb(204, 204, 204); display: block; overflow-x: hidden; }
p > .md-image:only-child:not(.md-img-error) img, p > img:only-child { display: block; margin: auto; }
#write.first-line-indent p > .md-image:only-child:not(.md-img-error) img { left: -2em; position: relative; }
p > .md-image:only-child { display: inline-block; width: 100%; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.md-math-block { width: 100%; }
.md-math-block:not(:empty)::after { display: none; }
.MathJax_ref { fill: currentcolor; }
[contenteditable="true"]:active, [contenteditable="true"]:focus, [contenteditable="false"]:active, [contenteditable="false"]:focus { outline: 0px; box-shadow: none; }
.md-task-list-item { position: relative; list-style-type: none; }
.task-list-item.md-task-list-item { padding-left: 0px; }
.md-task-list-item > input { position: absolute; top: 0px; left: 0px; margin-left: -1.2em; margin-top: calc(1em - 10px); border: none; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-top-left-radius: 10px; border-top-right-radius: 10px; border-bottom-right-radius: 10px; border-bottom-left-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc-content::after, .md-toc::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); }
.md-toc-item a { text-decoration: none; }
.md-toc-inner:hover { text-decoration: underline; }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: 700; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) { 
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
code, pre, samp, tt { font-family: var(--monospace); }
kbd { margin: 0px 0.1em; padding: 0.1em 0.6em; font-size: 0.8em; color: rgb(36, 39, 41); background-color: rgb(255, 255, 255); border: 1px solid rgb(173, 179, 185); border-top-left-radius: 3px; border-top-right-radius: 3px; border-bottom-right-radius: 3px; border-bottom-left-radius: 3px; box-shadow: rgba(12, 13, 14, 0.2) 0px 1px 0px, rgb(255, 255, 255) 0px 0px 0px 2px inset; white-space: nowrap; vertical-align: middle; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: var(--monospace); }
code { text-align: left; }
a.md-print-anchor { white-space: pre !important; border: none !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: 0px !important; text-shadow: initial !important; background-position: 0px 0px !important; }
.os-windows.monocolor-emoji .md-emoji { font-family: "Segoe UI Symbol", sans-serif; }
.md-diagram-panel > svg { max-width: 100%; }
[lang="flow"] svg, [lang="mermaid"] svg { max-width: 100%; height: auto; }
[lang="mermaid"] .node text { font-size: 1rem; }
table tr th { border-bottom-width: 0px; }
video { max-width: 100%; display: block; margin: 0px auto; }
iframe { max-width: 100%; width: 100%; border: none; }
.highlight td, .highlight tr { border: 0px; }
mark { background-color: rgb(255, 255, 0); color: rgb(0, 0, 0); }
.md-html-inline .md-plain, .md-html-inline strong, mark .md-inline-math, mark strong { color: inherit; }
.md-expand mark .md-meta { opacity: 0.3 !important; }
mark .md-meta { color: rgb(0, 0, 0); }
@media print { 
  .typora-export h1, .typora-export h2, .typora-export h3, .typora-export h4, .typora-export h5, .typora-export h6 { break-inside: avoid; }
}
.md-diagram-panel .messageText { stroke: none !important; }
.md-diagram-panel .start-state { fill: var(--node-fill); }
.md-diagram-panel .edgeLabel rect { opacity: 1 !important; }
.md-require-zoom-fix { height: auto; margin-top: 16px; margin-bottom: 16px; }
.md-require-zoom-fix foreignObject { font-size: var(--mermaid-font-zoom); }
.md-fences.md-fences-math { font-size: 1em; }
.md-fences-advanced:not(.md-focus) { padding: 0px; white-space: nowrap; border: 0px; }
.md-fences-advanced:not(.md-focus) { background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; background-position: inherit; background-repeat: inherit; }
.typora-export-show-outline .typora-export-content { max-width: 1440px; margin: auto; display: flex; flex-direction: row; }
.typora-export-sidebar { width: 300px; font-size: 0.8rem; margin-top: 80px; margin-right: 18px; }
.typora-export-show-outline #write { --webkit-flex: 2; flex: 2 1 0%; }
.typora-export-sidebar .outline-content { position: fixed; top: 0px; max-height: 100%; overflow: hidden auto; padding-bottom: 30px; padding-top: 60px; width: 300px; }
@media screen and (max-width: 1024px) { 
  .typora-export-sidebar, .typora-export-sidebar .outline-content { width: 240px; }
}
@media screen and (max-width: 800px) { 
  .typora-export-sidebar { display: none; }
}
.outline-content li, .outline-content ul { margin-left: 0px; margin-right: 0px; padding-left: 0px; padding-right: 0px; list-style: none; }
.outline-content ul { margin-top: 0px; margin-bottom: 0px; }
.outline-content strong { font-weight: 400; }
.outline-expander { width: 1rem; height: 1.428571429rem; position: relative; display: table-cell; vertical-align: middle; cursor: pointer; padding-left: 4px; }
.outline-expander::before { content: ''; position: relative; font-family: Ionicons; display: inline-block; font-size: 8px; vertical-align: middle; }
.outline-item { padding-top: 3px; padding-bottom: 3px; cursor: pointer; }
.outline-expander:hover::before { content: ''; }
.outline-h1 > .outline-item { padding-left: 0px; }
.outline-h2 > .outline-item { padding-left: 1em; }
.outline-h3 > .outline-item { padding-left: 2em; }
.outline-h4 > .outline-item { padding-left: 3em; }
.outline-h5 > .outline-item { padding-left: 4em; }
.outline-h6 > .outline-item { padding-left: 5em; }
.outline-label { cursor: pointer; display: table-cell; vertical-align: middle; text-decoration: none; color: inherit; }
.outline-label:hover { text-decoration: underline; }
.outline-item:hover { border-color: rgb(245, 245, 245); background-color: var(--item-hover-bg-color); }
.outline-item:hover { margin-left: -28px; margin-right: -28px; border-left-width: 28px; border-left-style: solid; border-left-color: transparent; border-right-width: 28px; border-right-style: solid; border-right-color: transparent; }
.outline-item-single .outline-expander::before, .outline-item-single .outline-expander:hover::before { display: none; }
.outline-item-open > .outline-item > .outline-expander::before { content: ''; }
.outline-children { display: none; }
.info-panel-tab-wrapper { display: none; }
.outline-item-open > .outline-children { display: block; }
.typora-export .outline-item { padding-top: 1px; padding-bottom: 1px; }
.typora-export .outline-item:hover { margin-right: -8px; border-right-width: 8px; border-right-style: solid; border-right-color: transparent; }
.typora-export .outline-expander::before { content: "+"; font-family: inherit; top: -1px; }
.typora-export .outline-expander:hover::before, .typora-export .outline-item-open > .outline-item > .outline-expander::before { content: '−'; }
.typora-export-collapse-outline .outline-children { display: none; }
.typora-export-collapse-outline .outline-item-open > .outline-children, .typora-export-no-collapse-outline .outline-children { display: block; }
.typora-export-no-collapse-outline .outline-expander::before { content: "" !important; }
.typora-export-show-outline .outline-item-active > .outline-item .outline-label { font-weight: 700; }
.md-inline-math-container mjx-container { zoom: 0.95; }


:root {
  --mermaid-theme: night;
}

[lang='mermaid'] .label {
  color: #333;
}

/* CSS Document */

/** code highlight */

.cm-s-inner .cm-variable,
.cm-s-inner .cm-operator,
.cm-s-inner .cm-property {
    color: #b8bfc6;
}

.cm-s-inner .cm-keyword {
    color: #C88FD0;
}

.cm-s-inner .cm-tag {
    color: #7DF46A;
}

.cm-s-inner .cm-attribute {
    color: #7575E4;
}

.CodeMirror div.CodeMirror-cursor {
    border-left: 1px solid #b8bfc6;
    z-index: 3;
}

.cm-s-inner .cm-string {
    color: #D26B6B;
}

.cm-s-inner .cm-comment,
.cm-s-inner.cm-comment {
    color: #DA924A;
}

.cm-s-inner .cm-header,
.cm-s-inner .cm-def,
.cm-s-inner.cm-header,
.cm-s-inner.cm-def {
    color: #8d8df0;
}

.cm-s-inner .cm-quote,
.cm-s-inner.cm-quote {
    color: #57ac57;
}

.cm-s-inner .cm-hr {
    color: #d8d5d5;
}

.cm-s-inner .cm-link {
    color: #d3d3ef;
}

.cm-s-inner .cm-negative {
    color: #d95050;
}

.cm-s-inner .cm-positive {
    color: #50e650;
}

.cm-s-inner .cm-string-2 {
    color: #f50;
}

.cm-s-inner .cm-meta,
.cm-s-inner .cm-qualifier {
    color: #b7b3b3;
}

.cm-s-inner .cm-builtin {
    color: #f3b3f8;
}

.cm-s-inner .cm-bracket {
    color: #997;
}

.cm-s-inner .cm-atom,
.cm-s-inner.cm-atom {
    color: #84B6CB;
}

.cm-s-inner .cm-number {
    color: #64AB8F;
}

.cm-s-inner .cm-variable {
    color: #b8bfc6;
}

.cm-s-inner .cm-variable-2 {
    color: #9FBAD5;
}

.cm-s-inner .cm-variable-3 {
    color: #1cc685;
}

.CodeMirror-selectedtext,
.CodeMirror-selected {
    background: #4a89dc;
    color: #fff !important;
    text-shadow: none;
}

.CodeMirror-gutters {
    border-right: none;
}

/* CSS Document */

/** markdown source **/
.cm-s-typora-default .cm-header, 
.cm-s-typora-default .cm-property
{
    color: #cebcca;
}

.CodeMirror.cm-s-typora-default div.CodeMirror-cursor{
    border-left: 3px solid #b8bfc6;
}

.cm-s-typora-default .cm-comment {
    color: #9FB1FF;
}

.cm-s-typora-default .cm-string {
    color: #A7A7D9
}

.cm-s-typora-default .cm-atom, .cm-s-typora-default .cm-number {
    color: #848695;
    font-style: italic;
}

.cm-s-typora-default .cm-link {
    color: #95B94B;
}

.cm-s-typora-default .CodeMirror-activeline-background {
    background: rgba(51, 51, 51, 0.72);
}

.cm-s-typora-default .cm-comment, .cm-s-typora-default .cm-code {
	color: #8aa1e1;
}@import "";
@import "";
@import "";

:root {
    --bg-color:  #363B40;
    --side-bar-bg-color: #2E3033;
    --text-color: #b8bfc6;

    --select-text-bg-color:#4a89dc;

    --item-hover-bg-color: #0a0d16;
    --control-text-color: #b7b7b7;
    --control-text-hover-color: #eee;
    --window-border: 1px solid #555;

    --active-file-bg-color: rgb(34, 34, 34);
    --active-file-border-color: #8d8df0;

    --primary-color: #a3d5fe;

    --active-file-text-color: white;
    --item-hover-bg-color: #70717d;
    --item-hover-text-color: white;
    --primary-color: #6dc1e7;

    --rawblock-edit-panel-bd: #333;

    --search-select-bg-color: #428bca;
}

html {
    font-size: 16px;
    -webkit-font-smoothing: antialiased;
}

html,
body {
    -webkit-text-size-adjust: 100%;
    -ms-text-size-adjust: 100%;
    background: #363B40;
    background: var(--bg-color);
    fill: currentColor;
    line-height: 1.625rem;
}

#write {
    max-width: 914px;
}


@media only screen and (min-width: 1400px) {
    #write {
        max-width: 1024px;
    }
}

@media only screen and (min-width: 1800px) {
    #write {
        max-width: 1200px;
    }
}

html,
body,
button,
input,
select,
textarea,
div.code-tooltip-content {
    color: #b8bfc6;
    border-color: transparent;
}

div.code-tooltip,
.md-hover-tip .md-arrow:after {
    background: #333;
}

.native-window #md-notification {
    border: 1px solid #70717d;
}

.popover.bottom > .arrow:after {
    border-bottom-color: #333;
}

html,
body,
button,
input,
select,
textarea {
    font-family: "Helvetica Neue", Helvetica, Arial, 'Segoe UI Emoji', sans-serif;
}

hr {
    height: 2px;
    border: 0;
    margin: 24px 0 !important;
}

h1,
h2,
h3,
h4,
h5,
h6 {
    font-family: "Lucida Grande", "Corbel", sans-serif;
    font-weight: normal;
    clear: both;
    -ms-word-wrap: break-word;
    word-wrap: break-word;
    margin: 0;
    padding: 0;
    color: #DEDEDE
}

h1 {
    font-size: 2.5rem;
    /* 36px */
    line-height: 2.75rem;
    /* 40px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1.5px;
}

h2 {
    font-size: 1.63rem;
    /* 24px */
    line-height: 1.875rem;
    /* 30px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h3 {
    font-size: 1.17rem;
    /* 18px */
    line-height: 1.5rem;
    /* 24px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h4 {
    font-size: 1.12rem;
    /* 16px */
    line-height: 1.375rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    color: white;
}

h5 {
    font-size: 0.97rem;
    /* 16px */
    line-height: 1.25rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    font-weight: bold;
}

h6 {
    font-size: 0.93rem;
    /* 16px */
    line-height: 1rem;
    /* 16px */
    margin-bottom: 0.75rem;
    color: white;
}

@media (min-width: 980px) {
    h3.md-focus:before,
    h4.md-focus:before,
    h5.md-focus:before,
    h6.md-focus:before {
        color: #ddd;
        border: 1px solid #ddd;
        border-radius: 3px;
        position: absolute;
        left: -1.642857143rem;
        top: .357142857rem;
        float: left;
        font-size: 9px;
        padding-left: 2px;
        padding-right: 2px;
        vertical-align: bottom;
        font-weight: normal;
        line-height: normal;
    }

    h3.md-focus:before {
        content: 'h3';
    }

    h4.md-focus:before {
        content: 'h4';
    }

    h5.md-focus:before {
        content: 'h5';
        top: 0px;
    }

    h6.md-focus:before {
        content: 'h6';
        top: 0px;
    }
}

a {
    text-decoration: none;
    outline: 0;
}

a:hover {
    outline: 0;
}

a:focus {
    outline: thin dotted;
}

sup.md-footnote {
    background-color: #555;
    color: #ddd;
}

p {
    -ms-word-wrap: break-word;
    word-wrap: break-word;
}

p,
ul,
dd,
ol,
hr,
address,
pre,
table,
iframe,
.wp-caption,
.wp-audio-shortcode,
.wp-video-shortcode {
    margin-top: 0;
    margin-bottom: 1.5rem;
    /* 24px */
}

audio:not([controls]) {
    display: none;
}

[hidden] {
    display: none;
}

::-moz-selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

*.in-text-selection,
::selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

ul,
ol {
    padding: 0 0 0 1.875rem;
    /* 30px */
}

ul {
    list-style: square;
}

ol {
    list-style: decimal;
}

ul ul,
ol ol,
ul ol,
ol ul {
    margin: 0;
}

b,
th,
dt,
strong {
    font-weight: bold;
}

i,
em,
dfn,
cite {
    font-style: italic;
}

/*blockquote {
    padding-left: 1.875rem;
    margin: 0 0 1.875rem 1.875rem;
    border-left: solid 2px #474d54;
    padding-left: 30px;
    margin-top: 35px;
}*/

blockquote {
    padding-left: 1.875rem;
    margin: 0 0 1.875rem 1.875rem;
    padding-left: 30px;
    margin-top: 35px;
    border-left: 2px solid #D5C5C8;
    background: #474d54;
    background-color: #474d54;
    color: #B197FC;
}

/*#772D8B , #2660A4 , #D90368 , #048BA8*/
/*https://coolors.co/d6ffb7-61cc9e-048ba8-ffbfa0-d5c5c8*/
/*blockquote blockquote { border-left: 3px solid #61CC9E }*/
blockquote blockquote { border-left: 2px solid #D6FFB7; border-bottom:none; }
blockquote blockquote blockquote { border-left: 2px solid #FFBFA0; border-bottom:none; }
blockquote blockquote blockquote blockquote { border-left: 2px solid #048BA8; border-bottom:none; }
blockquote blockquote blockquote blockquote blockquote { border-left: 2px solid #D90368; border-bottom:none; }

blockquote .md-math-block {
    color: #FFEBFF;
}

blockquote h2 {
    color: #EBD6A6;
}

blockquote h6 {
    border-radius:2em;
    opacity: 0.9;
    background-color:#B1C6FC;
    color:black;
    hite-space:normal;
    display:inline-block;
    padding:0.2em;
}


pre,
code,
kbd,
tt,
var {
    font-size: 0.875em;
    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
}

code,
tt,
var {
    background: rgba(0, 0, 0, 0.05);
}

kbd {
    padding: 2px 4px;
    font-size: 90%;
    color: #fff;
    background-color: #333;
    border-radius: 3px;
    box-shadow: inset 0 -1px 0 rgba(0,0,0,.25);
}

pre.md-fences {
    padding: 10px 10px 10px 30px;
    margin-bottom: 20px;
    background: #333;
}

.CodeMirror-gutters {
    background: #333;
    border-right: 1px solid transparent;
}

.enable-diagrams pre.md-fences[lang="sequence"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="flow"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="mermaid"] .code-tooltip {
    bottom: -2.2em;
    right: 4px;
}

code,
kbd,
tt,
var {
    padding: 2px 5px;
}

table {
    max-width: 100%;
    width: 100%;
    border-collapse: collapse;
    border-spacing: 0;
}

th,
td {
    padding: 5px 10px;
    vertical-align: top;
}

a {
    -webkit-transition: all .2s ease-in-out;
    transition: all .2s ease-in-out;
}

hr {
    background: #474d54;
    /* variable */
}

h1 {
    margin-top: 2em;
}

a {
    color: #e0e0e0;
    text-decoration: underline;
}

a:hover {
    color: #fff;
}

.md-inline-math script {
    color: #81b1db;
}

b,
th,
dt,
strong {
    color: #DEDEDE;
    /* variable */
}

mark {
    background: #D3D40E;
}

/*blockquote {
    color: #9DA2A6;
}
*/
table a {
    color: #DEDEDE;
    /* variable */
}

th,
td {
    border: solid 1px #474d54;
    /* variable */
}

.task-list {
    padding-left: 0;
}

.md-task-list-item {
    padding-left: 1.25rem;
}

.md-task-list-item > input {
    top: auto;
}

.md-task-list-item > input:before {
    content: "";
    display: inline-block;
    width: 0.875rem;
    height: 0.875rem;
    vertical-align: middle;
    text-align: center;
    border: 1px solid #b8bfc6;
    background-color: #363B40;
    margin-top: -0.4rem;
}

.md-task-list-item > input:checked:before,
.md-task-list-item > input[checked]:before {
    content: '\221A';
    /*◘*/
    font-size: 0.625rem;
    line-height: 0.625rem;
    color: #DEDEDE;
}

/** quick open **/
.auto-suggest-container {
    border: 0px;
    background-color: #525C65;
}

#typora-quick-open {
    background-color: #525C65;
}

#typora-quick-open input{
    background-color: #525C65;
    border: 0;
    border-bottom: 1px solid grey;
}

.typora-quick-open-item {
    background-color: inherit;
    color: inherit;
}

.typora-quick-open-item.active,
.typora-quick-open-item:hover {
    background-color: #4D8BDB;
    color: white;
}

.typora-quick-open-item:hover {
    background-color: rgba(77, 139, 219, 0.8);
}

.typora-search-spinner > div {
  background-color: #fff;
}

#write pre.md-meta-block {
    border-bottom: 1px dashed #ccc;
    background: transparent;
    padding-bottom: 0.6em;
    line-height: 1.6em;
}

.btn,
.btn .btn-default {
    background: transparent;
    color: #b8bfc6;
}

.ty-table-edit {
    border-top: 1px solid gray;
    background-color: #363B40;
}

.popover-title {
    background: transparent;
}

.md-image>.md-meta {
    color: #BBBBBB;
    background: transparent;
}

.md-expand.md-image>.md-meta {
    color: #DDD;
}

#write>h3:before,
#write>h4:before,
#write>h5:before,
#write>h6:before {
    border: none;
    border-radius: 0px;
    color: #888;
    text-decoration: underline;
    left: -1.4rem;
    top: 0.2rem;
}

#write>h3.md-focus:before {
    top: 2px;
}

#write>h4.md-focus:before {
    top: 2px;
}

.md-toc-item {
    color: #A8C2DC;
}

#write div.md-toc-tooltip {
    background-color: #363B40;
}

.dropdown-menu .btn:hover,
.dropdown-menu .btn:focus,
.md-toc .btn:hover,
.md-toc .btn:focus {
    color: white;
    background: black;
}

#toc-dropmenu {
    background: rgba(50, 54, 59, 0.93);
    border: 1px solid rgba(253, 253, 253, 0.15);
}

#toc-dropmenu .divider {
    background-color: #9b9b9b;
}

.outline-expander:before {
    top: 2px;
}

#typora-sidebar {
    box-shadow: none;
    border-right: 1px dashed;
    border-right: none;
}

.sidebar-tabs {
    border-bottom:0;
}

#typora-sidebar:hover .outline-title-wrapper {
    border-left: 1px dashed;
}

.outline-title-wrapper .btn {
    color: inherit;
}

.outline-item:hover {
    border-color: #363B40;
    background-color: #363B40;
    color: white;
}

h1.md-focus .md-attr,
h2.md-focus .md-attr,
h3.md-focus .md-attr,
h4.md-focus .md-attr,
h5.md-focus .md-attr,
h6.md-focus .md-attr,
.md-header-span .md-attr {
    color: #8C8E92;
    display: inline;
}

.md-comment {
    color: #5a95e3;
    opacity: 1;
}

.md-inline-math svg {
    color: #b8bfc6;
}

#math-inline-preview .md-arrow:after {
    background: black;
}

.modal-content {
    background: var(--bg-color);
    border: 0;
}

.modal-title {
    font-size: 1.5em;
}

.modal-content input {
    background-color: rgba(26, 21, 21, 0.51);
    color: white;
}

.modal-content .input-group-addon {
    color: white;
}

.modal-backdrop {
    background-color: rgba(174, 174, 174, 0.7);
}

.modal-content .btn-primary {
    border-color: var(--primary-color);
}

.md-table-resize-popover {
    background-color: #333;
}

.form-inline .input-group .input-group-addon {
    color: white;
}

#md-searchpanel {
    border-bottom: 1px dashed grey;
}

/** UI for electron */

.context-menu,
#spell-check-panel,
#footer-word-count-info {
    background-color: #42464A;
}

.context-menu.dropdown-menu .divider,
.dropdown-menu .divider {
    background-color: #777777;
    opacity: 1;
}

footer {
    color: inherit;
}

@media (max-width: 1000px) {
    footer {
        border-top: none;
    }
    footer:hover {
        color: inherit;
    }
}

#file-info-file-path .file-info-field-value:hover {
    background-color: #555;
    color: #dedede;
}

.megamenu-content,
.megamenu-opened header {
    background: var(--bg-color);
}

.megamenu-menu-panel h2,
.megamenu-menu-panel h1,
.long-btn {
    color: inherit;
}

.megamenu-menu-panel input[type='text'] {
    background: inherit;
    border: 0;
    border-bottom: 1px solid;
}

#recent-file-panel-action-btn {
    background: inherit;
    border: 1px grey solid;
}

.megamenu-menu-panel .dropdown-menu > li > a {
    color: inherit;
    background-color: #2F353A;
    text-decoration: none;
}

.megamenu-menu-panel table td:nth-child(1) {
    color: inherit;
    font-weight: bold;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(1) {
    color: white;
}

.modal-footer .btn-default,
.modal-footer .btn-primary,
.modal-footer .btn-default:not(:hover) {
    border: 1px solid;
    border-color: transparent;
}

.btn-primary {
    color: white;
}

.btn-default:hover, .btn-default:focus, .btn-default.focus, .btn-default:active, .btn-default.active, .open > .dropdown-toggle.btn-default {
    color: white;
    border: 1px solid #ddd;
    background-color: inherit;
}

.modal-header {
    border-bottom: 0;
}

.modal-footer {
    border-top: 0;
}

#recent-file-panel tbody tr:nth-child(2n-1) {
    background-color: transparent !important;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(2) {
    color: inherit;
}

.megamenu-menu-panel .btn {
    border: 1px solid #eee;
    background: transparent;
}

.mouse-hover .toolbar-icon.btn:hover,
#w-full.mouse-hover,
#w-pin.mouse-hover {
    background-color: inherit;
}

.typora-node::-webkit-scrollbar {
    width: 5px;
}

.typora-node::-webkit-scrollbar-thumb:vertical {
    background: rgba(250, 250, 250, 0.3);
}

.typora-node::-webkit-scrollbar-thumb:vertical:active {
    background: rgba(250, 250, 250, 0.5);
}

#w-unpin {
    background-color: #4182c4;
}

#top-titlebar, #top-titlebar * {
    color: var(--item-hover-text-color);
}

.typora-sourceview-on #toggle-sourceview-btn,
#footer-word-count:hover,
.ty-show-word-count #footer-word-count {
    background: #333333;
}

#toggle-sourceview-btn:hover {
    color: #eee;
    background: #333333;
}

/** focus mode */
.on-focus-mode .md-end-block:not(.md-focus):not(.md-focus-container) * {
    color: #686868 !important;
}

.on-focus-mode .md-end-block:not(.md-focus) img,
.on-focus-mode .md-task-list-item:not(.md-focus-container)>input {
    opacity: #686868 !important;
}

.on-focus-mode li[cid]:not(.md-focus-container){
    color: #686868;
}

.on-focus-mode .md-fences.md-focus .CodeMirror-code>*:not(.CodeMirror-activeline) *,
.on-focus-mode .CodeMirror.cm-s-inner:not(.CodeMirror-focused) * {
    color: #686868 !important;
}

.on-focus-mode .md-focus,
.on-focus-mode .md-focus-container {
    color: #fff;
}

.on-focus-mode #typora-source .CodeMirror-code>*:not(.CodeMirror-activeline) * {
    color: #686868 !important;
}


/*diagrams*/
#write .md-focus .md-diagram-panel {
    border: 1px solid #ddd;
    margin-left: -1px;
    width: calc(100% + 2px);
}

/*diagrams*/
#write .md-focus.md-fences-with-lineno .md-diagram-panel {
    margin-left: auto;
}

.md-diagram-panel-error {
    color: #f1908e;
}

.active-tab-files #info-panel-tab-file,
.active-tab-files #info-panel-tab-file:hover,
.active-tab-outline #info-panel-tab-outline,
.active-tab-outline #info-panel-tab-outline:hover {
    color: #eee;
}

.sidebar-footer-item:hover,
.footer-item:hover {
    background: inherit;
    color: white;
}

.ty-side-sort-btn.active,
.ty-side-sort-btn:hover,
.selected-folder-menu-item a:after {
    color: white;
}

#sidebar-files-menu {
    border:solid 1px;
    box-shadow: 4px 4px 20px rgba(0, 0, 0, 0.79);
    background-color: var(--bg-color);
}

.file-list-item {
    border-bottom:none;
}

.file-list-item-summary {
    opacity: 1;
}

.file-list-item.active:first-child {
    border-top: none;
}

.file-node-background {
    height: 32px;
}

.file-library-node.active>.file-node-content,
.file-list-item.active {
    color: white;
    color: var(--active-file-text-color);
}

.file-library-node.active>.file-node-background{
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}
.file-list-item.active {
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}

#ty-tooltip {
    background-color: black;
    color: #eee;
}

.md-task-list-item>input {
    margin-left: -1.3em;
    margin-top: 0.3rem;
    -webkit-appearance: none;
}

.md-mathjax-midline {
    background-color: #57616b;
    border-bottom: none;
}

footer.ty-footer {
    border-color: #656565;
}

.ty-preferences .btn-default {
    background: transparent;
}
.ty-preferences .btn-default:hover {
    background: #57616b;
}

.ty-preferences select {
    border: 1px solid #989698;
    height: 21px;
}

.ty-preferences .nav-group-item.active,
.export-item.active,
.export-items-list-control,
.export-detail {
    background: var(--item-hover-bg-color);
}

.ty-preferences input[type="search"] {
    border-color: #333;
    background: #333;
    line-height: 22px;
    border-radius: 6px;
    color: white;
}

.ty-preferences input[type="search"]:focus {
    box-shadow: none;
}

[data-is-directory="true"] .file-node-content {
    margin-bottom: 0;
}

.file-node-title {
    line-height: 22px;
}

.html-for-mac .file-node-open-state, .html-for-mac .file-node-icon {
    line-height: 26px;
}

::-webkit-scrollbar-thumb {
    background: rgba(230, 230, 230, 0.30);
}

::-webkit-scrollbar-thumb:active {
    background: rgba(230, 230, 230, 0.50);
}

#typora-sidebar:hover div.sidebar-content-content::-webkit-scrollbar-thumb:horizontal {
    background: rgba(230, 230, 230, 0.30);
}

.nav-group-item:active {
    background-color: #474d54 !important;
}

.md-search-hit {
    background: rgba(199, 140, 60, 0.81);
    color: #eee;
}

.md-search-hit * {
    color: #eee;
}

#md-searchpanel input {
    color: white;
}

.modal-backdrop.in {
    opacity: 1;
    backdrop-filter: blur(1px);
}

.clear-btn-icon {
    top: 8px;
}

 :root {--mermaid-font-zoom:1em ;}  @media print { @page {margin: 0 0 0 0;} body.typora-export {padding-left: 0; padding-right: 0;} #write {padding:0;}}
</style><title>Chapter 3 - Figures</title>
</head>
<body class='typora-export'><div class='typora-export-content'>
<div id='write'  class=''><center><h1>Chapter 3 - Figures</h1></center><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721038978-87360933.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-1 - <span class="title">A peptide bond</span></span></center>
<p>This <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5034/">covalent bond</a> forms when the carbon atom from the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4912/">carboxyl group</a> of one <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> shares electrons with the nitrogen atom <i>(blue)</i> from the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4809/">amino group</a> of a second amino acid. As indicated, a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> of water is lost in this <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5015/">condensation reaction</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721039183-211504980.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-2 - <span class="title">The structural components of a protein</span></span></center>
<p>A <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> consists of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5659/">polypeptide backbone</a> with attached side chains. Each type of protein differs in its sequence and number of amino acids; therefore, it is the sequence of the chemically different side chains that makes each protein distinct. The two ends of a polypeptide chain are chemically different: the end carrying the free <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4809/">amino group</a> (NH<sub>3</sub>
<sup>+</sup>, also written NH<sub>2</sub>) is the amino terminus, or N-terminus, and that carrying the free <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4912/">carboxyl group</a> (COO<sup>–</sup>, also written COOH) is the carboxyl terminus or C-terminus. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> sequence of a protein is always presented in the N-to-C direction, reading from left to right.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721039405-318171724.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-3 - <span class="title">The 20 amino acids found in proteins</span></span></center>
<p>Both three-letter and one-letter abbreviations are listed. As shown, there are equal numbers of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5653/">polar</a> and nonpolar side chains. For their atomic structures, see <a href="https://www.ncbi.nlm.nih.gov/books/NBK26830/box/A393/?report=objectonly" rid-ob="figobA393" target="object">Panel 3-1</a> (pp. 132–133).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721039631-53381543.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-4 - <span class="title">Steric limitations on the bond angles in a polypeptide chain</span></span></center>
<p>(A) Each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> contributes three bonds <i>(red)</i> to the backbone of the chain. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5605/">peptide bond</a> is planar <i>(gray shading)</i> and does not permit rotation. By contrast, rotation can occur about the C<sub>α</sub>–C bond, whose angle of rotation is called psi (ψ), and about the N–C<sub>α</sub> bond, whose angle of rotation is called phi (ϕ). By convention, an R group is often used to denote an amino acid <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5793/">side chain</a> <i>(green circles).</i> (B) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> of the main-chain atoms in a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> is <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by one pair of ϕ and ψ angles for each amino acid; because of steric collisions between atoms within each amino acid, most pairs of ϕ and ψ angles do not occur. In this so-called Ramachandran plot, each dot represents an observed pair of angles in a protein. (B, from J. Richardson, <i>Adv</i>. <i>Prot</i>. <i>Chem</i>. 34:174–175, 1981. © Academic Press.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721039834-705093537.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-5 - <span class="title">Three types of noncovalent bonds that help proteins fold</span></span></center>
<p>Although a single one of these bonds is quite weak, many of them often form together to create a strong bonding arrangement, as in the example shown. As in the previous figure, R is used as a general designation for an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5793/">side chain</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721040006-213859205.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-6 - <span class="title">How a protein folds into a compact conformation</span></span></center>
<p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5653/">polar</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> side chains tend to gather on the outside of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>, where they can interact with water; the nonpolar amino acid side chains are buried on the inside to form a tightly packed hydrophobic core of atoms that are hidden from water. In this schematic drawing, the protein contains only about 30 amino acids.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721040182-88875653.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-7 - <span class="title">Hydrogen bonds in a protein molecule</span></span></center>
<p>Large numbers of hydrogen bonds form between adjacent regions of the folded <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chain and help stabilize its three-dimensional shape. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> depicted is a portion of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5416/">lysozyme</a>, and the hydrogen bonds between the three possible pairs of partners have been differently colored, as indicated. (After C.<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5365/">K</a>. Mathews and K.E. van Holde, Biochemistry. Redwood City, CA: Benjamin/Cummings, 1996.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721040352-226255550.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-8 - <span class="title">The refolding of a denatured protein</span></span></center>
<p>(A) This experiment demonstrates that the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> is <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> solely by its <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> sequence. (B) The structure of urea. Urea is very soluble in water and unfolds proteins at high concentrations, where there is about one urea <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> for every six water molecules.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721040585-436559263.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-9 - <span class="title">The regular conformation of the polypeptide backbone observed in the α helix and the β sheet</span></span></center>
<p>(A, B, and C) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4755/">α helix</a>. The N–H of every <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5605/">peptide bond</a> is hydrogen-bonded to the C=O of a neighboring peptide bond located four peptide bonds away in the same chain. (D, E, and F) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4866/">β sheet</a>. In this example, adjacent peptide chains run in opposite (<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4834/">antiparallel</a>) directions. The individual <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chains (strands) in a β sheet are held together by hydrogen-bonding between peptide bonds in different strands, and the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> side chains in each strand alternately project above and below the plane of the sheet. (A) and (D) show all the atoms in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5659/">polypeptide backbone</a>, but the amino acid side chains are truncated and denoted by R. In contrast, (B) and (E) show the backbone atoms only, while (C) and (F) display the shorthand symbols that are used to represent the α helix and the β sheet in ribbon drawings of proteins (see <a href="https://www.ncbi.nlm.nih.gov/books/NBK26830/box/A400/?report=objectonly" rid-ob="figobA400" target="object">Panel 3-2</a>B).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721040869-476616012.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-10 - <span class="title">Two types of β sheet structures</span></span></center>
<p>(A) An <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4834/">antiparallel</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4866/">β sheet</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A402/?report=objectonly" rid-figpopup="figA402" rid-ob="figobA402" target="object">Figure 3-9D</a>). (B) A parallel β sheet. Both of these structures are common in proteins.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721041033-899890420.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-11 - <span class="title">The structure of a coiled-coil</span></span></center>
<p>(A) A single <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4755/">α helix</a>, with successive <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> side chains labeled in a sevenfold sequence, “abcdefg” (from bottom to top). Amino acids “a” and “d” in such a sequence lie close together on the cylinder surface, forming a “stripe” (<i>red</i>) that winds slowly around the α helix. Proteins that form coiled-coils typically have nonpolar amino acids at positions “a” and “d.” Consequently, as shown in (B), the two α helices can wrap around each other with the nonpolar side chains of one α helix interacting with the nonpolar side chains of the other, while the more <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5305/">hydrophilic</a> amino acid side chains are left exposed to the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4840/">aqueous</a> environment. (C) The atomic structure of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5002/">coiled-coil</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by x-ray crystallography. The <i>red</i> side chains are nonpolar.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721041239-345164958.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-12 - <span class="title">A protein formed from four domains</span></span></center>
<p>In the Src <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> shown, two of the domains form a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5691/">protein kinase</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a>, while the SH2 and SH3 domains perform regulatory functions. (A) A ribbon model, with ATP <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a> in <i>red</i>. (B) A spacing-filling model, with ATP substrate in <i>red</i>. Note that the site that binds ATP is positioned at the interface of the two domains that form the kinase. The detailed structure of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5792/">SH2 domain</a> is illustrated in <a href="https://www.ncbi.nlm.nih.gov/books/NBK26830/box/A400/?report=objectonly" rid-ob="figobA400" target="object">Panel 3-2</a> (pp. 138–139).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721041447-615101226.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-13 - <span class="title">Ribbon models of three different protein domains</span></span></center>
<p>(A) Cytochrome <i>b</i><sub>562</sub>, a single-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> involved in <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5118/">electron</a> transport in mitochondria. This protein is composed almost entirely of α helices. (B) The NAD-binding domain of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> lactic dehydrogenase, which is composed of a mixture of α helices and β sheets. (C) The variable domain of an immunoglobulin (antibody) <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5395/">light chain</a>, composed of a sandwich of two β sheets. In these examples, the α helices are shown in <i>green</i>, while strands organized as β sheets are denoted by <i>red arrows</i>.</p><p>Note that the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chain generally traverses back and forth across the entire <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a>, making sharp turns only at the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> surface. It is the protruding loop regions <i>(yellow)</i> that often form the binding sites for other molecules. (Adapted from drawings courtesy of Jane Richardson.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721041673-995769506.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-14 - <span class="title">The conformations of two serine proteases compared</span></span></center>
<p>The backbone conformations of elastase and chymotrypsin. Although only those amino acids in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chain shaded in <i>green</i> are the same in the two proteins, the two conformations are very similar nearly everywhere. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/">active site</a> of each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> is circled in <i>red;</i> this is where the peptide bonds of the proteins that serve as substrates are bound and cleaved by hydrolysis. The serine proteases derive their name from the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> serine, whose <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5793/">side chain</a> is part of the active site of each enzyme and directly participates in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/">cleavage</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721041872-687968139.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-15 - <span class="title">A comparison of a class of DNA-binding domains, called homeodomains, in a pair of proteins from two organisms separated by more than a billion years of evolution</span></span></center>
<p>(A) A ribbon model of the structure common to both proteins. (B) A trace of the α-carbon positions. The three-dimensional structures shown were <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by x-ray crystallography for the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5937/">yeast</a> α2 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> <i>(green)</i> and the <i>Drosophila</i> engrailed protein <i>(red).</i> (C) A comparison of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> sequences for the region of the proteins shown in (A) and (B). <i>Black dots</i> mark sites with identical amino acids. <i>Orange dots</i> indicate the position of a three amino acid insert in the α2 protein. (Adapted from C. Wolberger et al., <i>Cell</i> 67:517–528, 1991.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721042091-484108003.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-16 - <span class="title">Percentage of total genes containing one or more copies of the indicated protein domain, as derived from complete genome sequences</span></span></center>
<p>Note that one of the three domains selected, the immunoglobulin <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a>, has been a relatively late addition, and its relative abundance has increased in the vertebrate lineage. The estimates of human <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/">gene</a> numbers are approximate.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721042262-349431611.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-17 - <span class="title">The use of short signature sequences to find homologous protein domains</span></span></center>
<p>The two short sequences of 15 and 9 amino acids shown <i>(green)</i> can be used to search large databases for a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5689/">protein domain</a> that is found in many proteins, the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5792/">SH2 domain</a>. Here, the first 50 amino acids of the SH2 domain of 100 amino acids is compared for the human and <i>Drosophila</i> Src protein (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A407/?report=objectonly" rid-figpopup="figA407" rid-ob="figobA407" target="object">Figure 3-12</a>). In the computer-generated sequence comparison <i>(yellow row),</i> exact matches between the human and <i>Drosophila</i> proteins are noted by the one-letter abbreviation for the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a>; the positions with a similar but nonidentical amino acid are denoted by +, and nonmatches are blank. In this diagram, wherever one or both proteins contain an exact match to a position in the <i>green</i> sequences, both aligned sequences are colored <i>red</i>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721042397-683628580.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-18 - <span class="title">Domain shuffling</span></span></center>
<p>An extensive shuffling of blocks of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> sequence (protein domains) has occurred during protein evolution. Those portions of a protein denoted by the same shape and color in this diagram are evolutionarily related. Serine proteases like chymotrypsin are formed from two domains <i>(brown).</i> In the three other proteases shown, which are highly regulated and more specialized, these two protease domains are connected to one or more domains <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5292/">homologous</a> to domains found in epidermal <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5255/">growth factor</a> (EGF; <i>green),</i> to a calcium-binding protein <i>(yellow),</i> or to a “kringle” <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a> <i>(blue)</i> that contains three internal disulfide bridges. Chymotrypsin is illustrated in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A411/?report=objectonly" rid-figpopup="figA411" rid-ob="figobA411" target="object">Figure 3-14</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721042558-919929188.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-19 - <span class="title">The three-dimensional structures of some protein modules</span></span></center>
<p>In these ribbon diagrams, β-sheet strands are shown as <i>arrows,</i> and the N- and C-termini are indicated by <i>red spheres.</i> (Adapted from M. Baron, D.<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a>. Norman, and I.D. Campbell, <i>Trends Biochem. Sci.</i> 16:13–17, 1991, and D.J. Leahy et al., <i>Science</i> 258:987–991, 1992.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721042772-502151297.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-20 - <span class="title">An extended structure formed from a series of in-line protein modules</span></span></center>
<p>Four <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5178/">fibronectin</a> type 3 modules (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A420/?report=objectonly" rid-figpopup="figA420" rid-ob="figobA420" target="object">Figure 3-19</a>) from the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5164/">extracellular matrix</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> fibronectin are illustrated in (A) ribbon and (B) space-filling models. (Adapted from D.J. Leahy, I. Aukhil, and H.P. Erickson, <i>Cell</i> 84:155–164, 1996.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721042939-10691200.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-21 - <span class="title">Two identical protein subunits binding together to form a symmetric protein dimer</span></span></center>
<p>The Cro <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5736/">repressor</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> from bacteriophage lambda binds to <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> to turn off viral genes. Its two identical subunits bind head-to-head, held together by a combination of hydrophobic forces <i>(blue)</i> and a set of hydrogen bonds <i>(yellow</i> region). (Adapted from D.H. Ohlendorf, D.E. Tronrud, and B.W. Matthews, <i>J</i>. <i>Mol</i>. <i>Biol</i>. 280:129–136, 1998.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721043104-223983268.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-22 - <span class="title">A protein molecule containing multiple copies of a single protein subunit</span></span></center>
<p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> neuraminidase exists as a ring of four identical <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chains. The small diagram shows how the repeated use of the same binding interaction forms the structure.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721043318-850807159.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-23 - <span class="title">A protein formed as a symmetric assembly of two different subunits</span></span></center>
<p>Hemoglobin is an abundant <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> in red blood cells that contains two copies of α globin and two copies of β globin. Each of these four <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chains contains a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5270/">heme</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> <i>(red),</i> which is the site where oxygen (O<sub>2</sub>) is bound. Thus, each molecule of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5272/">hemoglobin</a> in the blood carries four molecules of oxygen.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721043488-708198536.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-24 - <span class="title">A collection of protein molecules, shown at the same scale</span></span></center>
<p>For comparison, a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> bound to a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> is also illustrated. These space-filling models represent a range of sizes and shapes. Hemoglobin, catalase, porin, <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4793/">alcohol</a> dehydrogenase, and aspartate transcarbamoylase are formed from multiple copies of subunits. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5792/">SH2 domain</a> <i>(top left)</i> is presented in detail in <a href="https://www.ncbi.nlm.nih.gov/books/NBK26830/box/A400/?report=objectonly" rid-ob="figobA400" target="object">Panel 3-2</a> (pp. 138–139). (After David S. Goodsell, Our Molecular Nature. New York: Springer-Verlag, 1996.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721043793-162911609.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-25 - <span class="title">Protein assemblies</span></span></center>
<p>(A) A <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> with just one <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4882/">binding site</a> can form a dimer with another identical protein. (B) Identical proteins with two different binding sites often form a long helical filament. (C) If the two binding sites are disposed appropriately in relation to each other, the protein subunits may form a closed ring instead of a helix. (For an example of A, see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A426/?report=objectonly" rid-figpopup="figA426" rid-ob="figobA426" target="object">Figure 3-21</a>; for an example of C, see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A427/?report=objectonly" rid-figpopup="figA427" rid-ob="figobA427" target="object">Figure 3-22</a>.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721044030-41886365.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-26 - <span class="title">Actin filaments</span></span></center>
<p>(A) Transmission <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5118/">electron</a> micrographs of negatively stained <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4766/">actin</a> filaments. (B) The helical arrangement of actin molecules in an actin filament. (A, courtesy of Roger Craig.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721044220-897885602.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-27 - <span class="title">Some properties of a helix</span></span></center>
<p>(A–D) A helix forms when a series of subunits bind to each other in a regular way. At the bottom, the interaction between two subunits is shown; behind them are the helices that result. These helices have two (A), three (B), and six (C and D) subunits per helical turn. At the top, the arrangement of subunits has been photographed from directly above the helix. Note that the helix in (D) has a wider path than that in (C), but the same number of subunits per turn. (E) A helix can be either right-handed or left-handed. As a reference, it is useful to remember that standard metal screws, which insert when turned clockwise, are right-handed. Note that a helix retains the same handedness when it is turned upside down.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721044397-383743312.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-28 - <span class="title">Collagen and elastin</span></span></center>
<p>(A) Collagen is a triple helix formed by three extended <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> chains that wrap around one another <i>(bottom).</i> Many rodlike <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5004/">collagen</a> molecules are cross-linked together in the extracellular space to form unextendable collagen fibrils <i>(top)</i> that have the tensile strength of steel. The striping on the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5003/">collagen fibril</a> is caused by the regular repeating arrangement of the collagen molecules within the fibril. (B) Elastin <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chains are cross-linked together to form rubberlike, elastic fibers. Each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5115/">elastin</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> uncoils into a more extended <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> when the fiber is stretched and recoils spontaneously as soon as the stretching force is relaxed.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721044617-204078784.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-29 - <span class="title">Disulfide bonds</span></span></center>
<p>This diagram illustrates how covalent disulfide bonds form between adjacent cysteine side chains. As indicated, these cross-linkages can join either two parts of the same <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chain or two different polypeptide chains. Since the energy required to break one <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5034/">covalent bond</a> is much larger than the energy required to break even a whole set of noncovalent bonds (see <a href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A165/table/A191/?report=objectonly" target="object">Table 2-2</a>, p. 57), a disulfide bond can have a major stabilizing effect on a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721044784-603690173.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-30 - <span class="title">An example of single protein subunit assembly requiring multiple protein–protein contacts</span></span></center>
<p>Hexagonally packed <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5233/">globular protein</a> subunits can form either a flat sheet or a tube.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721044944-926437515.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-31 - <span class="title">The capsids of some viruses, all shown at the same scale</span></span></center>
<p>(A) Tomato bushy stunt <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5926/">virus</a>; (B) poliovirus; (C) simian virus 40 (SV40); (D) satellite tobacco necrosis virus. The structures of all of these capsids have been <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by x-ray crystallography and are known in atomic detail. (Courtesy of Robert Grant, Stephan Crainic, and James M. Hogle.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721045151-288671630.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-32 - <span class="title">The structure of a spherical virus</span></span></center>
<p>In many viruses, identical <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> subunits pack together to create a spherical shell (a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4907/">capsid</a>) that encloses the viral <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/">genome</a>, composed of either <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> or <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> (see also <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A443/?report=objectonly" rid-figpopup="figA443" rid-ob="figobA443" target="object">Figure 3-31</a>). For geometric reasons, no more than 60 identical subunits can pack together in a precisely symmetric way. If slight irregularities are allowed, however, more subunits can be used to produce a larger capsid. The tomato bushy stunt <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5926/">virus</a> (TBSV) shown here, for example, is a spherical virus about 33 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5540/">nm</a> in diameter formed from 180 identical copies of a 386 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> capsid protein plus an RNA genome of 4500 nucleotides. To construct such a large capsid, the protein must be able to fit into three somewhat different environments, each of which is differently colored in the virus particle shown here. The postulated pathway of assembly is shown; the precise three-dimensional structure has been <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by x-ray diffraction. (Courtesy of Steve Harrison.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721045381-221386164.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-33 - <span class="title">The structure of tobacco mosaic virus (TMV)</span></span></center>
<p>(A) An <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5118/">electron</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5460/">micrograph</a> of the viral particle, which consists of a single long <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> enclosed in a cylindrical <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> coat composed of identical protein subunits. (B) A model showing part of the structure of TMV. A single-stranded RNA molecule of 6000 nucleotides is packaged in a helical coat constructed from 2130 copies of a coat protein 158 amino acids long. Fully infective viral particles can self-assemble in a test tube from purified RNA and protein molecules. (A, courtesy of Robley Williams; B, courtesy of Richard J. Feldmann.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721045552-395299786.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-34 - <span class="title">Three mechanisms of length determination for large protein assemblies</span></span></center>
<p>(A) Coassembly along an elongated core <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> or other <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5419/">macromolecule</a> that acts as a measuring device. (B) Termination of assembly because of strain that accumulates in the polymeric structure as additional subunits are added, so that beyond a certain length the energy required to fit another <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a> onto the chain becomes excessively large. (C) A vernier type of assembly, in which two sets of rodlike molecules differing in length form a staggered <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a> that grows until their ends exactly match. The name derives from a measuring device based on the same principle, used in mechanical instruments.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721045717-801353555.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-35 - <span class="title">An electron micrograph of bacteriophage lambda</span></span></center>
<p>The tip of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5926/">virus</a> tail attaches to a specific <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> on the surface of a bacterial cell, after which the tightly packaged <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> in the head is injected through the tail into the cell. The tail has a precise length, <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by the mechanism shown in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26830/figure/A447/?report=objectonly" rid-figpopup="figA447" rid-ob="figobA447" target="object">Figure 3-34A</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721045882-906127867.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-36 - <span class="title">Proteolytic cleavage in insulin assembly</span></span></center>
<p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5295/">hormone</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5341/">insulin</a> cannot spontaneously re-form efficiently if its disulfide bonds are disrupted. It is synthesized as a larger <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> <i>(proinsulin)</i> that is cleaved by a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5699/">proteolytic enzyme</a> after the protein chain has folded into a specific shape. Excision of part of the proinsulin polypeptide chain removes some of the information needed for the protein to fold spontaneously into its normal <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> once it has been denatured and its two polypeptide chains separated.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722475612-99470778.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-37 - <span class="title">The selective binding of a protein to another molecule</span></span></center>
<p>Many weak bonds are needed to enable a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> to bind tightly to a second <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>, which is called a <i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5393/">ligand</a></i> for the protein<i>.</i> A ligand must therefore fit precisely into a protein’s <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4882/">binding site</a>, like a hand into a glove, so that a large number of noncovalent bonds can be formed between the protein and the ligand.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722475802-283180078.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-38 - <span class="title">The binding site of a protein</span></span></center>
<p>(A) The folding of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chain typically creates a crevice or cavity on the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> surface. This crevice contains a set of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> side chains disposed in such a way that they can make noncovalent bonds only with certain ligands. (B) A close-up of an actual <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4882/">binding site</a> showing the hydrogen bonds and ionic interactions formed between a protein and its <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5393/">ligand</a> (in this example, cyclic AMP is the bound ligand).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722476014-724692846.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-39 - <span class="title">An unusually reactive amino acid at the active site of an enzyme</span></span></center>
<p>This example is the “catalytic triad” found in chymotrypsin, elastase, and other serine proteases (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A388/figure/A411/?report=objectonly" rid-figpopup="figA411" rid-ob="figobA411" target="object">Figure 3-14</a>). The aspartic <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4761/">acid</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5793/">side chain</a> (Asp 102) induces the histidine (His 57) to remove the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5701/">proton</a> from serine 195. This activates the serine to form a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5034/">covalent bond</a> with the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a>, hydrolyzing a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5605/">peptide bond</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722476212-487094411.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-40 - <span class="title">The evolutionary trace method applied to the SH2 domain</span></span></center>
<p>(A) Front and back views of a space-filling model of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5792/">SH2 domain</a>, with evolutionarily conserved amino acids on the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> surface colored <i>yellow</i>, and those more toward the protein interior colored <i>red.</i> (B) The structure of the SH2 domain with its bound <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a>. Here, those amino acids located within 0.4 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5540/">nm</a> of the bound <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5393/">ligand</a> are colored <i>blue</i>. The two key amino acids of the ligand are <i>yellow,</i> and the others are <i>purple</i>. (Adapted from O. Lichtarge, H.R. Bourne, and F.E. Cohen, <i>J. Mol. Biol.</i> 257:342–358, 1996.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722476397-574734860.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-41 - <span class="title">Three ways in which two proteins can bind to each other</span></span></center>
<p>Only the interacting parts of the two proteins are shown. (A) A rigid surface on one <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> can bind to an extended loop of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chain (a “string”) on a second protein. (B) Two α helices can bind together to form a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5002/">coiled-coil</a>. (C) Two <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/">complementary</a> rigid surfaces often link two proteins together.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722476562-834955302.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-42 - <span class="title">An antibody molecule</span></span></center>
<p>(A) A typical antibody <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> is Y-shaped and has two identical binding sites for its <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4830/">antigen</a>, one on each arm of the Y. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> is composed of four <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chains (two identical heavy chains and two identical and smaller light chains) held together by disulfide bonds. Each chain is made up of several different immunoglobulin domains, here shaded either <i>blue</i> or <i>gray</i>. The antigen-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4882/">binding site</a> is formed where a heavy-chain variable <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a> (V<sub>H</sub>) and a light-chain variable domain (V<sub>L</sub>) come close together. These are the domains that differ most in their sequence and structure in different antibodies. (B) This ribbon model of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5395/">light chain</a> shows the parts of the V<sub>L</sub> domain most closely involved in binding to the antigen in <i>red.</i> They contribute half of the fingerlike loops that fold around each of the antigen molecules in (A).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722476764-463843216.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-43 - <span class="title">How noncovalent bonds mediate interactions between macromolecules</span></span></center><center><img src="https://39363.org/IMAGE_BUCKET/1631722476991-292507463.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-44 - <span class="title">Relating binding energies to the equilibrium constant</span></span></center>
<p>(A) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5146/">equilibrium</a> between molecules A and B and the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a> AB is maintained by a balance between the two opposing reactions shown in panels 1 and 2. Molecules A and B must collide if they are to react, and the association rate is therefore proportional to the product of their individual concentrations [A] × [B]. (Square brackets indicate concentration.) As shown in panel 3, the ratio of the rate constants for the association and the dissociation reactions is equal to the equilibrium constant (<i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5365/">K</a></i>) for the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a>. (B) The equilibrium constant in panel 3 is that for the association reaction A + B ↔ AB, and the larger its value, the stronger the binding between A and B. Note that, for every 1.4 kcal/<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5483/">mole</a> of free-energy drop, the equilibrium constant increases by a factor of 10. (C) An example of the dramatic effect that the presence or absence of a few weak bonds can have in a biological context.</p><p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5146/">equilibrium</a> constant here has units of liters/<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5483/">mole</a>: for simple binding interactions it is also called the <i>affinity constant</i> or <i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4846/">association constant</a>,</i> denoted <i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5365/">K</a></i><sub>a</sub>. The reciprocal of <i>K</i><sub>a</sub> is called the <i>dissociation constant, K</i><sub>d</sub> (in units of moles/liter).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722477160-640807815.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-45 - <span class="title">Enzyme kinetics</span></span></center>
<p>The rate of an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> <i>(V)</i> increases as the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a> concentration increases until a maximum value <i>(V</i><sub>max</sub>) is reached. At this point all substrate-binding sites on the enzyme molecules are fully occupied, and the rate of reaction is limited by the rate of the catalytic process on the enzyme surface. For most enzymes, the concentration of substrate at which the reaction rate is half-maximal <i>(<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5365/">K</a></i><sub>m</sub>) is a measure of how tightly the substrate is bound, with a large value of <i>K</i><sub>m</sub> corresponding to weak binding.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722477326-923379991.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-46 - <span class="title">Enzymatic acceleration of chemical reactions by decreasing the activation energy</span></span></center>
<p>Often both the uncatalyzed <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> (A) and the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a>-catalyzed reaction (B) can go through several transition states. It is the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5889/">transition state</a> with the highest energy (S<sup>T</sup> and ES<sup>T</sup>) that determines the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4771/">activation energy</a> and limits the rate of the reaction. (S = <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a>; P = product of the reaction.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722477491-911233822.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-47 - <span class="title">Catalytic antibodies</span></span></center>
<p>The stabilization of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5889/">transition state</a> by an antibody creates an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a>. (A) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> path for the hydrolysis of an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4805/">amide</a> bond goes through a tetrahedral intermediate, the high-energy transition state for the reaction. (B) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> on the left was covalently linked to a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> and used as an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4830/">antigen</a> to generate an antibody that binds tightly to the region of the molecule shown in <i>yellow</i>. Because this antibody also bound tightly to the transition state in (A), it was found to function as an enzyme that efficiently catalyzed the hydrolysis of the amide bond in the molecule on the right.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722477663-578835425.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-48 - <span class="title">The rate accelerations caused by five different enzymes</span></span></center>
<p>(Modified from A. Radzicka and R. Wolfenden, <i>Science</i> 267:90–93, 1995.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722477825-644814879.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-49 - <span class="title">Acid catalysis and base catalysis</span></span></center>
<p>(A) The start of the uncatalyzed <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> shown in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A473/?report=objectonly" rid-figpopup="figA473" rid-ob="figobA473" target="object">Figure 3-47A</a> is diagrammed, with <i>blue</i> indicating <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5118/">electron</a> distribution in the water and carbonyl bonds. (B) An <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4761/">acid</a> likes to donate a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5701/">proton</a> (<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5263/">H<sup>+</sup></a>) to other atoms. By pairing with the carbonyl oxygen, an acid causes electrons to move away from the carbonyl carbon, making this atom much more attractive to the electronegative oxygen of an attacking water <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>. (C) A <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a> likes to take up H<sup>+</sup>. By pairing with a hydrogen of the attacking water molecule, a base causes electrons to move toward the water oxygen, making it a better attacking group for the carbonyl carbon. (D) By having appropriately positioned atoms on its surface, an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> can perform both acid catalysis and base catalysis at the same time.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722478002-520314045.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-50 - <span class="title">The reaction catalyzed by lysozyme</span></span></center>
<p>(A) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5416/">lysozyme</a> (denoted as E) catalyzes the cutting of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5662/">polysaccharide</a> chain, which is its <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a> (S). The enzyme first binds to the chain to form an enzyme–substrate <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a> (ES) and then catalyzes the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/">cleavage</a> of a specific <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5034/">covalent bond</a> in the backbone of the polysaccharide, forming an enzyme–product complex (EP) that rapidly dissociates. Release of the severed chain (the products P) leaves the enzyme free to act on another substrate <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>. (B) A space-filling model of the lysozyme molecule bound to a short length of polysaccharide chain before cleavage. (B, courtesy of Richard J. Feldmann.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722478176-394794564.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-51 - <span class="title">Events at the active site of lysozyme</span></span></center>
<p>The top left and top right drawings depict the free <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a> and the free products, respectively, whereas the other three drawings depict the sequential events at the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/">active site</a>. Note the change in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5842/">sugar</a> D in the enzyme–substrate <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a>; this shape change stabilizes the oxocarbenium <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5355/">ion</a>-like transition states required for formation and hydrolysis of the covalent intermediate shown in the middle panel. (Based on D.J. Vocadlo et al., <i>Nature</i> 412:835–838, 2001.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722478418-298969692.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-52 - <span class="title">Some general strategies of enzyme catalysis</span></span></center>
<p>(A) Holding substrates together in a precise alignment. (B) Charge stabilization of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> intermediates. (C) Altering bond angles in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a> to increase the rate of a particular reaction.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722478586-498055131.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-53 - <span class="title">Retinal and heme</span></span></center>
<p>(A) The structure of retinal, the light-sensitive <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> attached to <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5750/">rhodopsin</a> in the eye. (B) The structure of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5270/">heme</a> group. The carbon-containing heme ring is <i>red</i> and the iron atom at its center is <i>orange</i>. A heme group is tightly bound to each of the four <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chains in <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5272/">hemoglobin</a>, the oxygen-carrying <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> whose structure is shown in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A388/figure/A428/?report=objectonly" rid-figpopup="figA428" rid-ob="figobA428" target="object">Figure 3-23</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722478753-235933162.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-54 - <span class="title">The structure of pyruvate dehydrogenase</span></span></center>
<p>This <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a> catalyzes the conversion of pyruvate to <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4758/">acetyl CoA</a>, as part of the pathway that oxidizes sugars to CO<sub>2</sub> and H<sub>2</sub>O. It is an example of a large multienzyme complex in which <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> intermediates are passed directly from one enzyme to another.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722478920-132822733.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-55 - <span class="title">Feedback inhibition of a single biosynthetic pathway</span></span></center>
<p>The end-product Z inhibits the first <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> that is unique to its synthesis and thereby controls its own level in the cell. This is an example of negative regulation.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722479084-911966737.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-56 - <span class="title">Multiple feedback inhibition</span></span></center>
<p>In this example, which shows the biosynthetic pathways for four different amino acids in bacteria, the <i>red arrows</i> indicate positions at which products feed back to inhibit enzymes. Each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> controls the first <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> specific to its own synthesis, thereby controlling its own levels and avoiding a wasteful buildup of intermediates. The products can also separately inhibit the initial set of reactions common to all the syntheses; in this case, three different enzymes catalyze the initial <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a>, each inhibited by a different product.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722479304-89322563.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-57 - <span class="title">Positive regulation caused by conformational coupling between two distant binding sites</span></span></center>
<p>In this example, both <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5234/">glucose</a> and <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> X bind best to the <i>closed</i> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> with two domains. Because both glucose and molecule X drive the protein toward its closed conformation, each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5393/">ligand</a> helps the other to bind. Glucose and molecule X are therefore said to bind <i>cooperatively</i> to the protein.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722479509-652158100.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-58 - <span class="title">Negative regulation caused by conformational coupling between two distant binding sites</span></span></center>
<p>The scheme here resembles that in the previous figure, but here <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> X prefers the <i>open</i> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a>, while <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5234/">glucose</a> prefers the <i>closed</i> conformation. Because glucose and molecule X drive the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> toward opposite conformations (closed and open, respectively), the presence of either <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5393/">ligand</a> interferes with the binding of the other.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722479713-21776823.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-59 - <span class="title">Enzyme activity versus the concentration of inhibitory ligand for single-subunit and multi-subunit allosteric enzymes</span></span></center>
<p>For an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> with a single <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a> <i>(red line),</i> a drop from 90% enzyme activity to 10% activity (indicated by the two dots on the curve) requires a 100-fold increase in the concentration of inhibitor. The enzyme activity is calculated from the simple <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5146/">equilibrium</a> relationship <i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5365/">K</a></i> = [I][P]/[IP], where P is active <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>, I is inhibitor, and IP is the inactive protein bound to inhibitor. An identical curve applies to any simple binding interaction between two molecules, A and B. In contrast, a multisubunit allosteric enzyme can respond in a switchlike manner to a change in <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5393/">ligand</a> concentration: the steep response is caused by a cooperative binding of the ligand molecules, as explained in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A498/?report=objectonly" rid-figpopup="figA498" rid-ob="figobA498" target="object">Figure 3-60</a>. Here, the <i>green line</i> represents the idealized result expected for the cooperative binding of two inhibitory ligand molecules to an allosteric enzyme with two subunits, and the <i>blue line</i> shows the idealized response of an enzyme with four subunits. As indicated by the two dots on each of these curves, the more <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a> enzymes drop from 90% to 10% activity over a much narrower range of inhibitor concentration than does the enzyme composed of a single subunit.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722479883-58714662.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-60 - <span class="title">A cooperative allosteric transition in an enzyme composed of two identical subunits</span></span></center>
<p>This diagram illustrates how the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> of one <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a> can influence that of its neighbor. The binding of a single <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> of an inhibitory <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5393/">ligand</a> <i>(yellow)</i> to one subunit of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> occurs with difficulty because it changes the conformation of this subunit and thereby destroys the symmetry of the enzyme. Once this conformational change has occurred, however, the energy gained by restoring the symmetric pairing interaction between the two subunits makes it especially easy for the second subunit to bind the inhibitory ligand and undergo the same conformational change. Because the binding of the first molecule of ligand increases the affinity with which the other subunit binds the same ligand, the response of the enzyme to changes in the concentration of the ligand is much steeper than the response of an enzyme with only one subunit (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A497/?report=objectonly" rid-figpopup="figA497" rid-ob="figobA497" target="object">Figure 3-59</a>).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722480061-526484670.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-61 - <span class="title">The transition between R and T states in the enzyme aspartate transcarbamoylase</span></span></center>
<p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> consists of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a> of six catalytic subunits and six regulatory subunits, and the structures of its inactive (T state) and active (R state) forms have been <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by x-ray crystallography. The enzyme is turned off by <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5173/">feedback inhibition</a> when CTP concentrations rise. Each regulatory <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a> can bind one <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> of CTP, which is one of the final products in the pathway. By means of this negative feedback regulation, the pathway is prevented from producing more CTP than the cell needs. (Based on <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5365/">K</a>.L. Krause, K.W. Volz, and W.N. Lipscomb, <i>Proc</i>. <i>Natl</i>. <i>Acad</i>. <i>Sci</i>. <i>USA</i> 82:1643–1647, 1985.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722480279-992711122.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-62 - <span class="title">Part of the on–off switch in the catalytic subunits of aspartate transcarbamoylase</span></span></center>
<p>Changes in the indicated hydrogen-bonding interactions are partly responsible for switching this <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a>’s <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/">active site</a> between active <i>(yellow)</i> and inactive conformations. Hydrogen bonds are indicated by <i>thin red lines.</i> The amino acids involved in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a>–subunit interaction are shown in <i>red</i>, while those that form the active site of the enzyme are shown in <i>blue.</i> The large drawings show the catalytic site in the interior of the enzyme; the boxed sketches show the same subunits viewed from the enzyme’s external surface. (Adapted from E.R. Kantrowitz and W.N. Lipscomb, <i>Trends Biochem</i>. <i>Sci</i>. 15:53–59, 1990.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722480492-875052285.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-63 - <span class="title">Protein phosphorylation</span></span></center>
<p>Many thousands of proteins in a typical eucaryotic cell are modified by the covalent addition of a phosphate group. (A) The general <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a>, shown here, entails the transfer of a phosphate group from ATP to an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4807/">amino acid</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5793/">side chain</a> of the target <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> by a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5691/">protein kinase</a>. Removal of the phosphate group is catalyzed by a second <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a>, a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5694/">protein phosphatase</a>. In this example, the phosphate is added to a serine side chain; in other cases, the phosphate is instead linked to the –<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5308/">OH</a> group of a threonine or a tyrosine in the protein. (B) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5629/">phosphorylation</a> of a protein by a protein kinase can either increase or decrease the protein’s activity, depending on the site of phosphorylation and the structure of the protein.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722480660-772333819.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-64 - <span class="title">The three-dimensional structure of a protein kinase</span></span></center>
<p>Superimposed on this structure are <i>red arrowheads</i> to indicate sites where insertions of 5–100 amino acids are found in some members of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5691/">protein kinase</a> family. These insertions are located in loops on the surface of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> where other ligands interact with the protein. Thus, they distinguish different kinases and confer on them distinctive interactions with other proteins. The ATP (which donates a phosphate group) and the peptide to be phosphorylated are held in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/">active site</a>, which extends between the phosphate-binding loop <i>(yellow)</i> and the catalytic loop <i>(orange).</i> See also <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A388/figure/A407/?report=objectonly" rid-figpopup="figA407" rid-ob="figobA407" target="object">Figure 3-12</a>. (Adapted from D.R. Knighton et al., <i>Science</i> 253:407–414, 1991.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722480825-550069185.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-65 - <span class="title">An evolutionary tree of selected protein kinases</span></span></center>
<p>Although a higher eucaryotic cell contains hundreds of such enzymes, and the human <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/">genome</a> codes for more than 500, only some of those discussed in this book are shown.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722480993-21375743.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-66 - <span class="title">How a Cdk protein acts as an integrating device</span></span></center>
<p>The function of these central regulators of the cell cycle is discussed in Chapter 17.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722481132-773585619.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-67 - <span class="title">The domain structure of the Src family of protein kinases, mapped along the amino acid sequence</span></span></center><center><img src="https://39363.org/IMAGE_BUCKET/1631722481295-29980885.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-68 - <span class="title">The activation of a Src-type protein kinase by two sequential events</span></span></center>
<p>(Adapted from I. Moareti, et al., <i>Nature</i> 385:650–653, 1997.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722481499-403877491.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-69 - <span class="title">How a Src-type protein kinase acts as an integrating device</span></span></center>
<p>The disruption of the SH3 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a> interaction <i>(green)</i> can involve replacing its binding to the indicated <i>red</i> linker region with a tighter interaction with the Nef <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>, as illustrated in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A510/?report=objectonly" rid-figpopup="figA510" rid-ob="figobA510" target="object">Figure 3-68</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722481664-549163570.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-70 - <span class="title">GTP-binding proteins as molecular switches</span></span></center>
<p>The activity of a GTP-binding <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> (also called a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5259/">GTPase</a>) generally requires the presence of a tightly bound GTP <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> (switch “on”). Hydrolysis of this GTP molecule produces GDP and inorganic phosphate (P<sub>i</sub>), and it causes the protein to convert to a different, usually inactive, <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> (switch “off”). As shown here, resetting the switch requires the tightly bound GDP to dissociate, a slow step that is greatly accelerated by specific signals; once the GDP has dissociated, a molecule of GTP is quickly rebound.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722481829-361636056.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-71 - <span class="title">The structure of the Ras protein in its GTP-bound form</span></span></center>
<p>This monomeric <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5259/">GTPase</a> illustrates the structure of a GTP-binding <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a>, which is present in a large family of GTP-binding proteins. The <i>red</i> regions change their <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> when the GTP <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> is hydrolyzed to GDP and inorganic phosphate by the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>; the GDP remains bound to the protein, while the inorganic phosphate is released. The special role of the “switch helix” in proteins related to Ras is explained next (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A519/?report=objectonly" rid-figpopup="figA519" rid-ob="figobA519" target="object">Figure 3-74</a>).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722482008-10633521.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-72 - <span class="title">A comparison of the two major intracellular signaling mechanisms in eucaryotic cells</span></span></center>
<p>In both cases, a signaling <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> is activated by the addition of a phosphate group and inactivated by the removal of this phosphate. To emphasize the similarities in the two pathways, ATP and GTP are drawn as APPP and GPPP, and ADP and GDP as APP and GPP, respectively. As shown in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A504/?report=objectonly" rid-figpopup="figA504" rid-ob="figobA504" target="object">Figure 3-63</a>, the addition of a phosphate to a protein can also be inhibitory.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722482213-898493962.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-73 - <span class="title">An aminoacyl tRNA molecule bound to EF-Tu</span></span></center>
<p>The three domains of the EF-Tu <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> are colored differently, to match <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A519/?report=objectonly" rid-figpopup="figA519" rid-ob="figobA519" target="object">Figure 3-74</a>. This is a bacterial protein; however, a very similar protein exists in eukaryotes, where it is called EF-1. (Coordinates <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by P. Nissen et al., <i>Science</i> 270:1464–1472, 1995.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722482390-522210800.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-74 - <span class="title">The large conformational change in EF-Tu caused by GTP hydrolysis</span></span></center>
<p>(A) The three-dimensional structure of EF-Tu with GTP bound. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5101/">domain</a> at the top is <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5292/">homologous</a> to the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5716/">Ras protein</a>, and its <i>red</i> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4755/">α helix</a> is the switch helix, which moves after GTP hydrolysis, as shown in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A514/?report=objectonly" rid-figpopup="figA514" rid-ob="figobA514" target="object">Figure 3-71</a>. (B) The change in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> of the switch helix in domain 1 causes domains 2 and 3 to rotate as a single unit by about 90° toward the viewer, which releases the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5902/">tRNA</a> that was shown bound to this structure in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26911/figure/A518/?report=objectonly" rid-figpopup="figA518" rid-ob="figobA518" target="object">Figure 3-73</a>. (A, adapted from H. Berchtold et al., <i>Nature</i> 365:126–132, 1993; B, courtesy of Mathias Sprinzl and Rolf Hilgenfeld.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722482589-972822267.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-75 - <span class="title">An allosteric “walking” protein</span></span></center>
<p>Although its three different conformations allow it to wander randomly back and forth while bound to a thread or a filament, the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> cannot move uniformly in a single direction.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722482758-915639660.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-76 - <span class="title">An allosteric motor protein</span></span></center>
<p>The transition between three different conformations includes a step driven by the hydrolysis of a bound ATP <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>, and this makes the entire cycle essentially irreversible. By repeated cycles, the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> therefore moves continuously to the right along the thread.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722482938-633324690.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-77 - <span class="title">The transport of calcium ions by the Ca<sup><b>2+</b></sup> pump</span></span></center>
<p>(A) The structure of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5708/">pump</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>, formed from a single <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a> of 994 amino acids (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A1999/figure/A2017/?report=objectonly" rid-figpopup="figA2017" rid-ob="figobA2017" target="object">Figure 11-15</a> for details). (B) A model for <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5355/">ion</a> pumping. For simplicity, only two of the four transmembrane α helices that bind Ca<sup>2+</sup> are shown.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631722483162-961950825.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 3-78 - <span class="title">A map of the protein–protein interactions observed between different functional groups of proteins in the yeast <i>S. cerevisiae.</i></span></span></center>
<p>To produce this map, more than 1500 proteins were assigned to the indicated 31 functional groups. About 70 percent of the known <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>–protein interactions were observed to occur between proteins assigned to the same functional group. However, as indicated, a surprisingly large number of interactions crossed between these groups. Each line in this diagram designates that more than 15 protein–protein interactions were observed between proteins in the two functional groups that are connected by that line. The three functional groups highlighted in <i>yellow</i> display at least 15 interactions with 10 or more of the 31 functional groups defined in the study. (From C.L. Tucker, J.F. Gera, and P. Uetz, <i>Trends Cell Biol.</i> 11:102–106, 2001.)</p></div></div>
</body>
</html>