<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>

<style type='text/css'>html {overflow-x: initial !important;}:root { --bg-color: #ffffff; --text-color: #333333; --select-text-bg-color: #B5D6FC; --select-text-font-color: auto; --monospace: "Lucida Console",Consolas,"Courier",monospace; --title-bar-height: 20px; }
.mac-os-11 { --title-bar-height: 28px; }
html { font-size: 14px; background-color: var(--bg-color); color: var(--text-color); font-family: "Helvetica Neue", Helvetica, Arial, sans-serif; -webkit-font-smoothing: antialiased; }
body { margin: 0px; padding: 0px; height: auto; inset: 0px; font-size: 1rem; line-height: 1.42857143; overflow-x: hidden; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; tab-size: 4; background-position: inherit; background-repeat: inherit; }
iframe { margin: auto; }
a.url { word-break: break-all; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { text-shadow: none; background: var(--select-text-bg-color); color: var(--select-text-font-color); }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; white-space: normal; overflow-x: visible; padding-top: 36px; }
#write.first-line-indent p { text-indent: 2em; }
#write.first-line-indent li p, #write.first-line-indent p * { text-indent: 0px; }
#write.first-line-indent li { margin-left: 2em; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
.typora-export .footnote-line, .typora-export li, .typora-export p { white-space: pre-wrap; }
.typora-export .task-list-item input { pointer-events: none; }
@media screen and (max-width: 500px) { 
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  #write { padding-left: 20px; padding-right: 20px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
#write li > figure:last-child { margin-bottom: 0.5rem; }
#write ol, #write ul { position: relative; }
img { max-width: 100%; vertical-align: middle; image-orientation: from-image; }
button, input, select, textarea { color: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-caps: inherit; font-weight: inherit; font-stretch: inherit; line-height: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
*, ::after, ::before { box-sizing: border-box; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p, #write pre { width: inherit; }
#write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write p { position: relative; }
p { line-height: inherit; }
h1, h2, h3, h4, h5, h6 { break-after: avoid-page; break-inside: avoid; orphans: 4; }
p { orphans: 4; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
.md-math-block, .md-rawblock, h1, h2, h3, h4, h5, h6, p { margin-top: 1rem; margin-bottom: 1rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: 700; font-style: italic; }
a { cursor: pointer; }
sup.md-footnote { padding: 2px 4px; background-color: rgba(238, 238, 238, 0.7); color: rgb(85, 85, 85); border-top-left-radius: 4px; border-top-right-radius: 4px; border-bottom-right-radius: 4px; border-bottom-left-radius: 4px; cursor: pointer; }
sup.md-footnote a, sup.md-footnote a:hover { color: inherit; text-transform: inherit; text-decoration: inherit; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; }
figure { overflow-x: auto; margin: 1.2em 0px; max-width: calc(100% + 16px); padding: 0px; }
figure > table { margin: 0px; }
tr { break-inside: avoid; break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; break-inside: auto; text-align: left; }
table.md-table td { min-width: 32px; }
.CodeMirror-gutters { border-right-width: 0px; background-color: inherit; }
.CodeMirror-linenumber { }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
#write pre { white-space: pre-wrap; }
#write.fences-no-line-wrapping pre { white-space: pre; }
#write pre.ty-contain-cm { white-space: normal; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; position: relative !important; background-position: inherit; background-repeat: inherit; }
.md-fences-adv-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
#write .md-fences.mock-cm { white-space: pre-wrap; }
.md-fences.md-fences-with-lineno { padding-left: 0px; }
#write.fences-no-line-wrapping .md-fences.mock-cm { white-space: pre; overflow-x: auto; }
.md-fences.mock-cm.md-fences-with-lineno { padding-left: 8px; }
.CodeMirror-line, twitterwidget { break-inside: avoid; }
.footnotes { opacity: 0.8; font-size: 0.9rem; margin-top: 1em; margin-bottom: 1em; }
.footnotes + .footnotes { margin-top: 0px; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; line-height: normal; font-weight: 400; text-align: left; box-sizing: content-box; direction: ltr; background-position: 0px 0px; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li .mathjax-block, li p { margin: 0.5rem 0px; }
li blockquote { margin: 1rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child, li > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; white-space: normal; }
#write .footnote-line { white-space: pre-wrap; }
@media print { 
  body, html { border: 1px solid transparent; height: 99%; break-after: avoid; break-before: avoid; font-variant-ligatures: no-common-ligatures; }
  #write { margin-top: 0px; padding-top: 0px; border-color: transparent !important; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  .typora-export #write { break-after: avoid; }
  .typora-export #write::after { height: 0px; }
  .is-mac table { break-inside: avoid; }
  .typora-export-show-outline .typora-export-sidebar { display: none; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 0.8rem; white-space: pre-wrap; background-color: rgb(204, 204, 204); display: block; overflow-x: hidden; }
p > .md-image:only-child:not(.md-img-error) img, p > img:only-child { display: block; margin: auto; }
#write.first-line-indent p > .md-image:only-child:not(.md-img-error) img { left: -2em; position: relative; }
p > .md-image:only-child { display: inline-block; width: 100%; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.md-math-block { width: 100%; }
.md-math-block:not(:empty)::after { display: none; }
.MathJax_ref { fill: currentcolor; }
[contenteditable="true"]:active, [contenteditable="true"]:focus, [contenteditable="false"]:active, [contenteditable="false"]:focus { outline: 0px; box-shadow: none; }
.md-task-list-item { position: relative; list-style-type: none; }
.task-list-item.md-task-list-item { padding-left: 0px; }
.md-task-list-item > input { position: absolute; top: 0px; left: 0px; margin-left: -1.2em; margin-top: calc(1em - 10px); border: none; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-top-left-radius: 10px; border-top-right-radius: 10px; border-bottom-right-radius: 10px; border-bottom-left-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc-content::after, .md-toc::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); }
.md-toc-item a { text-decoration: none; }
.md-toc-inner:hover { text-decoration: underline; }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: 700; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) { 
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: "."; }
code, pre, samp, tt { font-family: var(--monospace); }
kbd { margin: 0px 0.1em; padding: 0.1em 0.6em; font-size: 0.8em; color: rgb(36, 39, 41); background-color: rgb(255, 255, 255); border: 1px solid rgb(173, 179, 185); border-top-left-radius: 3px; border-top-right-radius: 3px; border-bottom-right-radius: 3px; border-bottom-left-radius: 3px; box-shadow: rgba(12, 13, 14, 0.2) 0px 1px 0px, rgb(255, 255, 255) 0px 0px 0px 2px inset; white-space: nowrap; vertical-align: middle; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: var(--monospace); }
code { text-align: left; }
a.md-print-anchor { white-space: pre !important; border: none !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: 0px !important; text-shadow: initial !important; background-position: 0px 0px !important; }
.os-windows.monocolor-emoji .md-emoji { font-family: "Segoe UI Symbol", sans-serif; }
.md-diagram-panel > svg { max-width: 100%; }
[lang="flow"] svg, [lang="mermaid"] svg { max-width: 100%; height: auto; }
[lang="mermaid"] .node text { font-size: 1rem; }
table tr th { border-bottom-width: 0px; }
video { max-width: 100%; display: block; margin: 0px auto; }
iframe { max-width: 100%; width: 100%; border: none; }
.highlight td, .highlight tr { border: 0px; }
mark { background-color: rgb(255, 255, 0); color: rgb(0, 0, 0); }
.md-html-inline .md-plain, .md-html-inline strong, mark .md-inline-math, mark strong { color: inherit; }
.md-expand mark .md-meta { opacity: 0.3 !important; }
mark .md-meta { color: rgb(0, 0, 0); }
@media print { 
  .typora-export h1, .typora-export h2, .typora-export h3, .typora-export h4, .typora-export h5, .typora-export h6 { break-inside: avoid; }
}
.md-diagram-panel .messageText { stroke: none !important; }
.md-diagram-panel .start-state { fill: var(--node-fill); }
.md-diagram-panel .edgeLabel rect { opacity: 1 !important; }
.md-require-zoom-fix { height: auto; margin-top: 16px; margin-bottom: 16px; }
.md-require-zoom-fix foreignObject { font-size: var(--mermaid-font-zoom); }
.md-fences.md-fences-math { font-size: 1em; }
.md-fences-advanced:not(.md-focus) { padding: 0px; white-space: nowrap; border: 0px; }
.md-fences-advanced:not(.md-focus) { background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; background-position: inherit; background-repeat: inherit; }
.typora-export-show-outline .typora-export-content { max-width: 1440px; margin: auto; display: flex; flex-direction: row; }
.typora-export-sidebar { width: 300px; font-size: 0.8rem; margin-top: 80px; margin-right: 18px; }
.typora-export-show-outline #write { --webkit-flex: 2; flex: 2 1 0%; }
.typora-export-sidebar .outline-content { position: fixed; top: 0px; max-height: 100%; overflow: hidden auto; padding-bottom: 30px; padding-top: 60px; width: 300px; }
@media screen and (max-width: 1024px) { 
  .typora-export-sidebar, .typora-export-sidebar .outline-content { width: 240px; }
}
@media screen and (max-width: 800px) { 
  .typora-export-sidebar { display: none; }
}
.outline-content li, .outline-content ul { margin-left: 0px; margin-right: 0px; padding-left: 0px; padding-right: 0px; list-style: none; }
.outline-content ul { margin-top: 0px; margin-bottom: 0px; }
.outline-content strong { font-weight: 400; }
.outline-expander { width: 1rem; height: 1.428571429rem; position: relative; display: table-cell; vertical-align: middle; cursor: pointer; padding-left: 4px; }
.outline-expander::before { content: ''; position: relative; font-family: Ionicons; display: inline-block; font-size: 8px; vertical-align: middle; }
.outline-item { padding-top: 3px; padding-bottom: 3px; cursor: pointer; }
.outline-expander:hover::before { content: ''; }
.outline-h1 > .outline-item { padding-left: 0px; }
.outline-h2 > .outline-item { padding-left: 1em; }
.outline-h3 > .outline-item { padding-left: 2em; }
.outline-h4 > .outline-item { padding-left: 3em; }
.outline-h5 > .outline-item { padding-left: 4em; }
.outline-h6 > .outline-item { padding-left: 5em; }
.outline-label { cursor: pointer; display: table-cell; vertical-align: middle; text-decoration: none; color: inherit; }
.outline-label:hover { text-decoration: underline; }
.outline-item:hover { border-color: rgb(245, 245, 245); background-color: var(--item-hover-bg-color); }
.outline-item:hover { margin-left: -28px; margin-right: -28px; border-left-width: 28px; border-left-style: solid; border-left-color: transparent; border-right-width: 28px; border-right-style: solid; border-right-color: transparent; }
.outline-item-single .outline-expander::before, .outline-item-single .outline-expander:hover::before { display: none; }
.outline-item-open > .outline-item > .outline-expander::before { content: ''; }
.outline-children { display: none; }
.info-panel-tab-wrapper { display: none; }
.outline-item-open > .outline-children { display: block; }
.typora-export .outline-item { padding-top: 1px; padding-bottom: 1px; }
.typora-export .outline-item:hover { margin-right: -8px; border-right-width: 8px; border-right-style: solid; border-right-color: transparent; }
.typora-export .outline-expander::before { content: "+"; font-family: inherit; top: -1px; }
.typora-export .outline-expander:hover::before, .typora-export .outline-item-open > .outline-item > .outline-expander::before { content: '−'; }
.typora-export-collapse-outline .outline-children { display: none; }
.typora-export-collapse-outline .outline-item-open > .outline-children, .typora-export-no-collapse-outline .outline-children { display: block; }
.typora-export-no-collapse-outline .outline-expander::before { content: "" !important; }
.typora-export-show-outline .outline-item-active > .outline-item .outline-label { font-weight: 700; }
.md-inline-math-container mjx-container { zoom: 0.95; }


:root {
  --mermaid-theme: night;
}

[lang='mermaid'] .label {
  color: #333;
}

/* CSS Document */

/** code highlight */

.cm-s-inner .cm-variable,
.cm-s-inner .cm-operator,
.cm-s-inner .cm-property {
    color: #b8bfc6;
}

.cm-s-inner .cm-keyword {
    color: #C88FD0;
}

.cm-s-inner .cm-tag {
    color: #7DF46A;
}

.cm-s-inner .cm-attribute {
    color: #7575E4;
}

.CodeMirror div.CodeMirror-cursor {
    border-left: 1px solid #b8bfc6;
    z-index: 3;
}

.cm-s-inner .cm-string {
    color: #D26B6B;
}

.cm-s-inner .cm-comment,
.cm-s-inner.cm-comment {
    color: #DA924A;
}

.cm-s-inner .cm-header,
.cm-s-inner .cm-def,
.cm-s-inner.cm-header,
.cm-s-inner.cm-def {
    color: #8d8df0;
}

.cm-s-inner .cm-quote,
.cm-s-inner.cm-quote {
    color: #57ac57;
}

.cm-s-inner .cm-hr {
    color: #d8d5d5;
}

.cm-s-inner .cm-link {
    color: #d3d3ef;
}

.cm-s-inner .cm-negative {
    color: #d95050;
}

.cm-s-inner .cm-positive {
    color: #50e650;
}

.cm-s-inner .cm-string-2 {
    color: #f50;
}

.cm-s-inner .cm-meta,
.cm-s-inner .cm-qualifier {
    color: #b7b3b3;
}

.cm-s-inner .cm-builtin {
    color: #f3b3f8;
}

.cm-s-inner .cm-bracket {
    color: #997;
}

.cm-s-inner .cm-atom,
.cm-s-inner.cm-atom {
    color: #84B6CB;
}

.cm-s-inner .cm-number {
    color: #64AB8F;
}

.cm-s-inner .cm-variable {
    color: #b8bfc6;
}

.cm-s-inner .cm-variable-2 {
    color: #9FBAD5;
}

.cm-s-inner .cm-variable-3 {
    color: #1cc685;
}

.CodeMirror-selectedtext,
.CodeMirror-selected {
    background: #4a89dc;
    color: #fff !important;
    text-shadow: none;
}

.CodeMirror-gutters {
    border-right: none;
}

/* CSS Document */

/** markdown source **/
.cm-s-typora-default .cm-header, 
.cm-s-typora-default .cm-property
{
    color: #cebcca;
}

.CodeMirror.cm-s-typora-default div.CodeMirror-cursor{
    border-left: 3px solid #b8bfc6;
}

.cm-s-typora-default .cm-comment {
    color: #9FB1FF;
}

.cm-s-typora-default .cm-string {
    color: #A7A7D9
}

.cm-s-typora-default .cm-atom, .cm-s-typora-default .cm-number {
    color: #848695;
    font-style: italic;
}

.cm-s-typora-default .cm-link {
    color: #95B94B;
}

.cm-s-typora-default .CodeMirror-activeline-background {
    background: rgba(51, 51, 51, 0.72);
}

.cm-s-typora-default .cm-comment, .cm-s-typora-default .cm-code {
	color: #8aa1e1;
}@import "";
@import "";
@import "";

:root {
    --bg-color:  #363B40;
    --side-bar-bg-color: #2E3033;
    --text-color: #b8bfc6;

    --select-text-bg-color:#4a89dc;

    --item-hover-bg-color: #0a0d16;
    --control-text-color: #b7b7b7;
    --control-text-hover-color: #eee;
    --window-border: 1px solid #555;

    --active-file-bg-color: rgb(34, 34, 34);
    --active-file-border-color: #8d8df0;

    --primary-color: #a3d5fe;

    --active-file-text-color: white;
    --item-hover-bg-color: #70717d;
    --item-hover-text-color: white;
    --primary-color: #6dc1e7;

    --rawblock-edit-panel-bd: #333;

    --search-select-bg-color: #428bca;
}

html {
    font-size: 16px;
    -webkit-font-smoothing: antialiased;
}

html,
body {
    -webkit-text-size-adjust: 100%;
    -ms-text-size-adjust: 100%;
    background: #363B40;
    background: var(--bg-color);
    fill: currentColor;
    line-height: 1.625rem;
}

#write {
    max-width: 914px;
}


@media only screen and (min-width: 1400px) {
    #write {
        max-width: 1024px;
    }
}

@media only screen and (min-width: 1800px) {
    #write {
        max-width: 1200px;
    }
}

html,
body,
button,
input,
select,
textarea,
div.code-tooltip-content {
    color: #b8bfc6;
    border-color: transparent;
}

div.code-tooltip,
.md-hover-tip .md-arrow:after {
    background: #333;
}

.native-window #md-notification {
    border: 1px solid #70717d;
}

.popover.bottom > .arrow:after {
    border-bottom-color: #333;
}

html,
body,
button,
input,
select,
textarea {
    font-family: "Helvetica Neue", Helvetica, Arial, 'Segoe UI Emoji', sans-serif;
}

hr {
    height: 2px;
    border: 0;
    margin: 24px 0 !important;
}

h1,
h2,
h3,
h4,
h5,
h6 {
    font-family: "Lucida Grande", "Corbel", sans-serif;
    font-weight: normal;
    clear: both;
    -ms-word-wrap: break-word;
    word-wrap: break-word;
    margin: 0;
    padding: 0;
    color: #DEDEDE
}

h1 {
    font-size: 2.5rem;
    /* 36px */
    line-height: 2.75rem;
    /* 40px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1.5px;
}

h2 {
    font-size: 1.63rem;
    /* 24px */
    line-height: 1.875rem;
    /* 30px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h3 {
    font-size: 1.17rem;
    /* 18px */
    line-height: 1.5rem;
    /* 24px */
    margin-bottom: 1.5rem;
    /* 24px */
    letter-spacing: -1px;
    font-weight: bold;
}

h4 {
    font-size: 1.12rem;
    /* 16px */
    line-height: 1.375rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    color: white;
}

h5 {
    font-size: 0.97rem;
    /* 16px */
    line-height: 1.25rem;
    /* 22px */
    margin-bottom: 1.5rem;
    /* 24px */
    font-weight: bold;
}

h6 {
    font-size: 0.93rem;
    /* 16px */
    line-height: 1rem;
    /* 16px */
    margin-bottom: 0.75rem;
    color: white;
}

@media (min-width: 980px) {
    h3.md-focus:before,
    h4.md-focus:before,
    h5.md-focus:before,
    h6.md-focus:before {
        color: #ddd;
        border: 1px solid #ddd;
        border-radius: 3px;
        position: absolute;
        left: -1.642857143rem;
        top: .357142857rem;
        float: left;
        font-size: 9px;
        padding-left: 2px;
        padding-right: 2px;
        vertical-align: bottom;
        font-weight: normal;
        line-height: normal;
    }

    h3.md-focus:before {
        content: 'h3';
    }

    h4.md-focus:before {
        content: 'h4';
    }

    h5.md-focus:before {
        content: 'h5';
        top: 0px;
    }

    h6.md-focus:before {
        content: 'h6';
        top: 0px;
    }
}

a {
    text-decoration: none;
    outline: 0;
}

a:hover {
    outline: 0;
}

a:focus {
    outline: thin dotted;
}

sup.md-footnote {
    background-color: #555;
    color: #ddd;
}

p {
    -ms-word-wrap: break-word;
    word-wrap: break-word;
}

p,
ul,
dd,
ol,
hr,
address,
pre,
table,
iframe,
.wp-caption,
.wp-audio-shortcode,
.wp-video-shortcode {
    margin-top: 0;
    margin-bottom: 1.5rem;
    /* 24px */
}

audio:not([controls]) {
    display: none;
}

[hidden] {
    display: none;
}

::-moz-selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

*.in-text-selection,
::selection {
    background: #4a89dc;
    color: #fff;
    text-shadow: none;
}

ul,
ol {
    padding: 0 0 0 1.875rem;
    /* 30px */
}

ul {
    list-style: square;
}

ol {
    list-style: decimal;
}

ul ul,
ol ol,
ul ol,
ol ul {
    margin: 0;
}

b,
th,
dt,
strong {
    font-weight: bold;
}

i,
em,
dfn,
cite {
    font-style: italic;
}

/*blockquote {
    padding-left: 1.875rem;
    margin: 0 0 1.875rem 1.875rem;
    border-left: solid 2px #474d54;
    padding-left: 30px;
    margin-top: 35px;
}*/

blockquote {
    padding-left: 1.875rem;
    margin: 0 0 1.875rem 1.875rem;
    padding-left: 30px;
    margin-top: 35px;
    border-left: 2px solid #D5C5C8;
    background: #474d54;
    background-color: #474d54;
    color: #B197FC;
}

/*#772D8B , #2660A4 , #D90368 , #048BA8*/
/*https://coolors.co/d6ffb7-61cc9e-048ba8-ffbfa0-d5c5c8*/
/*blockquote blockquote { border-left: 3px solid #61CC9E }*/
blockquote blockquote { border-left: 2px solid #D6FFB7; border-bottom:none; }
blockquote blockquote blockquote { border-left: 2px solid #FFBFA0; border-bottom:none; }
blockquote blockquote blockquote blockquote { border-left: 2px solid #048BA8; border-bottom:none; }
blockquote blockquote blockquote blockquote blockquote { border-left: 2px solid #D90368; border-bottom:none; }

blockquote .md-math-block {
    color: #FFEBFF;
}

blockquote h2 {
    color: #EBD6A6;
}

blockquote h6 {
    border-radius:2em;
    opacity: 0.9;
    background-color:#B1C6FC;
    color:black;
    hite-space:normal;
    display:inline-block;
    padding:0.2em;
}


pre,
code,
kbd,
tt,
var {
    font-size: 0.875em;
    font-family: Monaco, Consolas, "Andale Mono", "DejaVu Sans Mono", monospace;
}

code,
tt,
var {
    background: rgba(0, 0, 0, 0.05);
}

kbd {
    padding: 2px 4px;
    font-size: 90%;
    color: #fff;
    background-color: #333;
    border-radius: 3px;
    box-shadow: inset 0 -1px 0 rgba(0,0,0,.25);
}

pre.md-fences {
    padding: 10px 10px 10px 30px;
    margin-bottom: 20px;
    background: #333;
}

.CodeMirror-gutters {
    background: #333;
    border-right: 1px solid transparent;
}

.enable-diagrams pre.md-fences[lang="sequence"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="flow"] .code-tooltip,
.enable-diagrams pre.md-fences[lang="mermaid"] .code-tooltip {
    bottom: -2.2em;
    right: 4px;
}

code,
kbd,
tt,
var {
    padding: 2px 5px;
}

table {
    max-width: 100%;
    width: 100%;
    border-collapse: collapse;
    border-spacing: 0;
}

th,
td {
    padding: 5px 10px;
    vertical-align: top;
}

a {
    -webkit-transition: all .2s ease-in-out;
    transition: all .2s ease-in-out;
}

hr {
    background: #474d54;
    /* variable */
}

h1 {
    margin-top: 2em;
}

a {
    color: #e0e0e0;
    text-decoration: underline;
}

a:hover {
    color: #fff;
}

.md-inline-math script {
    color: #81b1db;
}

b,
th,
dt,
strong {
    color: #DEDEDE;
    /* variable */
}

mark {
    background: #D3D40E;
}

/*blockquote {
    color: #9DA2A6;
}
*/
table a {
    color: #DEDEDE;
    /* variable */
}

th,
td {
    border: solid 1px #474d54;
    /* variable */
}

.task-list {
    padding-left: 0;
}

.md-task-list-item {
    padding-left: 1.25rem;
}

.md-task-list-item > input {
    top: auto;
}

.md-task-list-item > input:before {
    content: "";
    display: inline-block;
    width: 0.875rem;
    height: 0.875rem;
    vertical-align: middle;
    text-align: center;
    border: 1px solid #b8bfc6;
    background-color: #363B40;
    margin-top: -0.4rem;
}

.md-task-list-item > input:checked:before,
.md-task-list-item > input[checked]:before {
    content: '\221A';
    /*◘*/
    font-size: 0.625rem;
    line-height: 0.625rem;
    color: #DEDEDE;
}

/** quick open **/
.auto-suggest-container {
    border: 0px;
    background-color: #525C65;
}

#typora-quick-open {
    background-color: #525C65;
}

#typora-quick-open input{
    background-color: #525C65;
    border: 0;
    border-bottom: 1px solid grey;
}

.typora-quick-open-item {
    background-color: inherit;
    color: inherit;
}

.typora-quick-open-item.active,
.typora-quick-open-item:hover {
    background-color: #4D8BDB;
    color: white;
}

.typora-quick-open-item:hover {
    background-color: rgba(77, 139, 219, 0.8);
}

.typora-search-spinner > div {
  background-color: #fff;
}

#write pre.md-meta-block {
    border-bottom: 1px dashed #ccc;
    background: transparent;
    padding-bottom: 0.6em;
    line-height: 1.6em;
}

.btn,
.btn .btn-default {
    background: transparent;
    color: #b8bfc6;
}

.ty-table-edit {
    border-top: 1px solid gray;
    background-color: #363B40;
}

.popover-title {
    background: transparent;
}

.md-image>.md-meta {
    color: #BBBBBB;
    background: transparent;
}

.md-expand.md-image>.md-meta {
    color: #DDD;
}

#write>h3:before,
#write>h4:before,
#write>h5:before,
#write>h6:before {
    border: none;
    border-radius: 0px;
    color: #888;
    text-decoration: underline;
    left: -1.4rem;
    top: 0.2rem;
}

#write>h3.md-focus:before {
    top: 2px;
}

#write>h4.md-focus:before {
    top: 2px;
}

.md-toc-item {
    color: #A8C2DC;
}

#write div.md-toc-tooltip {
    background-color: #363B40;
}

.dropdown-menu .btn:hover,
.dropdown-menu .btn:focus,
.md-toc .btn:hover,
.md-toc .btn:focus {
    color: white;
    background: black;
}

#toc-dropmenu {
    background: rgba(50, 54, 59, 0.93);
    border: 1px solid rgba(253, 253, 253, 0.15);
}

#toc-dropmenu .divider {
    background-color: #9b9b9b;
}

.outline-expander:before {
    top: 2px;
}

#typora-sidebar {
    box-shadow: none;
    border-right: 1px dashed;
    border-right: none;
}

.sidebar-tabs {
    border-bottom:0;
}

#typora-sidebar:hover .outline-title-wrapper {
    border-left: 1px dashed;
}

.outline-title-wrapper .btn {
    color: inherit;
}

.outline-item:hover {
    border-color: #363B40;
    background-color: #363B40;
    color: white;
}

h1.md-focus .md-attr,
h2.md-focus .md-attr,
h3.md-focus .md-attr,
h4.md-focus .md-attr,
h5.md-focus .md-attr,
h6.md-focus .md-attr,
.md-header-span .md-attr {
    color: #8C8E92;
    display: inline;
}

.md-comment {
    color: #5a95e3;
    opacity: 1;
}

.md-inline-math svg {
    color: #b8bfc6;
}

#math-inline-preview .md-arrow:after {
    background: black;
}

.modal-content {
    background: var(--bg-color);
    border: 0;
}

.modal-title {
    font-size: 1.5em;
}

.modal-content input {
    background-color: rgba(26, 21, 21, 0.51);
    color: white;
}

.modal-content .input-group-addon {
    color: white;
}

.modal-backdrop {
    background-color: rgba(174, 174, 174, 0.7);
}

.modal-content .btn-primary {
    border-color: var(--primary-color);
}

.md-table-resize-popover {
    background-color: #333;
}

.form-inline .input-group .input-group-addon {
    color: white;
}

#md-searchpanel {
    border-bottom: 1px dashed grey;
}

/** UI for electron */

.context-menu,
#spell-check-panel,
#footer-word-count-info {
    background-color: #42464A;
}

.context-menu.dropdown-menu .divider,
.dropdown-menu .divider {
    background-color: #777777;
    opacity: 1;
}

footer {
    color: inherit;
}

@media (max-width: 1000px) {
    footer {
        border-top: none;
    }
    footer:hover {
        color: inherit;
    }
}

#file-info-file-path .file-info-field-value:hover {
    background-color: #555;
    color: #dedede;
}

.megamenu-content,
.megamenu-opened header {
    background: var(--bg-color);
}

.megamenu-menu-panel h2,
.megamenu-menu-panel h1,
.long-btn {
    color: inherit;
}

.megamenu-menu-panel input[type='text'] {
    background: inherit;
    border: 0;
    border-bottom: 1px solid;
}

#recent-file-panel-action-btn {
    background: inherit;
    border: 1px grey solid;
}

.megamenu-menu-panel .dropdown-menu > li > a {
    color: inherit;
    background-color: #2F353A;
    text-decoration: none;
}

.megamenu-menu-panel table td:nth-child(1) {
    color: inherit;
    font-weight: bold;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(1) {
    color: white;
}

.modal-footer .btn-default,
.modal-footer .btn-primary,
.modal-footer .btn-default:not(:hover) {
    border: 1px solid;
    border-color: transparent;
}

.btn-primary {
    color: white;
}

.btn-default:hover, .btn-default:focus, .btn-default.focus, .btn-default:active, .btn-default.active, .open > .dropdown-toggle.btn-default {
    color: white;
    border: 1px solid #ddd;
    background-color: inherit;
}

.modal-header {
    border-bottom: 0;
}

.modal-footer {
    border-top: 0;
}

#recent-file-panel tbody tr:nth-child(2n-1) {
    background-color: transparent !important;
}

.megamenu-menu-panel tbody tr:hover td:nth-child(2) {
    color: inherit;
}

.megamenu-menu-panel .btn {
    border: 1px solid #eee;
    background: transparent;
}

.mouse-hover .toolbar-icon.btn:hover,
#w-full.mouse-hover,
#w-pin.mouse-hover {
    background-color: inherit;
}

.typora-node::-webkit-scrollbar {
    width: 5px;
}

.typora-node::-webkit-scrollbar-thumb:vertical {
    background: rgba(250, 250, 250, 0.3);
}

.typora-node::-webkit-scrollbar-thumb:vertical:active {
    background: rgba(250, 250, 250, 0.5);
}

#w-unpin {
    background-color: #4182c4;
}

#top-titlebar, #top-titlebar * {
    color: var(--item-hover-text-color);
}

.typora-sourceview-on #toggle-sourceview-btn,
#footer-word-count:hover,
.ty-show-word-count #footer-word-count {
    background: #333333;
}

#toggle-sourceview-btn:hover {
    color: #eee;
    background: #333333;
}

/** focus mode */
.on-focus-mode .md-end-block:not(.md-focus):not(.md-focus-container) * {
    color: #686868 !important;
}

.on-focus-mode .md-end-block:not(.md-focus) img,
.on-focus-mode .md-task-list-item:not(.md-focus-container)>input {
    opacity: #686868 !important;
}

.on-focus-mode li[cid]:not(.md-focus-container){
    color: #686868;
}

.on-focus-mode .md-fences.md-focus .CodeMirror-code>*:not(.CodeMirror-activeline) *,
.on-focus-mode .CodeMirror.cm-s-inner:not(.CodeMirror-focused) * {
    color: #686868 !important;
}

.on-focus-mode .md-focus,
.on-focus-mode .md-focus-container {
    color: #fff;
}

.on-focus-mode #typora-source .CodeMirror-code>*:not(.CodeMirror-activeline) * {
    color: #686868 !important;
}


/*diagrams*/
#write .md-focus .md-diagram-panel {
    border: 1px solid #ddd;
    margin-left: -1px;
    width: calc(100% + 2px);
}

/*diagrams*/
#write .md-focus.md-fences-with-lineno .md-diagram-panel {
    margin-left: auto;
}

.md-diagram-panel-error {
    color: #f1908e;
}

.active-tab-files #info-panel-tab-file,
.active-tab-files #info-panel-tab-file:hover,
.active-tab-outline #info-panel-tab-outline,
.active-tab-outline #info-panel-tab-outline:hover {
    color: #eee;
}

.sidebar-footer-item:hover,
.footer-item:hover {
    background: inherit;
    color: white;
}

.ty-side-sort-btn.active,
.ty-side-sort-btn:hover,
.selected-folder-menu-item a:after {
    color: white;
}

#sidebar-files-menu {
    border:solid 1px;
    box-shadow: 4px 4px 20px rgba(0, 0, 0, 0.79);
    background-color: var(--bg-color);
}

.file-list-item {
    border-bottom:none;
}

.file-list-item-summary {
    opacity: 1;
}

.file-list-item.active:first-child {
    border-top: none;
}

.file-node-background {
    height: 32px;
}

.file-library-node.active>.file-node-content,
.file-list-item.active {
    color: white;
    color: var(--active-file-text-color);
}

.file-library-node.active>.file-node-background{
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}
.file-list-item.active {
    background-color: rgb(34, 34, 34);
    background-color: var(--active-file-bg-color);
}

#ty-tooltip {
    background-color: black;
    color: #eee;
}

.md-task-list-item>input {
    margin-left: -1.3em;
    margin-top: 0.3rem;
    -webkit-appearance: none;
}

.md-mathjax-midline {
    background-color: #57616b;
    border-bottom: none;
}

footer.ty-footer {
    border-color: #656565;
}

.ty-preferences .btn-default {
    background: transparent;
}
.ty-preferences .btn-default:hover {
    background: #57616b;
}

.ty-preferences select {
    border: 1px solid #989698;
    height: 21px;
}

.ty-preferences .nav-group-item.active,
.export-item.active,
.export-items-list-control,
.export-detail {
    background: var(--item-hover-bg-color);
}

.ty-preferences input[type="search"] {
    border-color: #333;
    background: #333;
    line-height: 22px;
    border-radius: 6px;
    color: white;
}

.ty-preferences input[type="search"]:focus {
    box-shadow: none;
}

[data-is-directory="true"] .file-node-content {
    margin-bottom: 0;
}

.file-node-title {
    line-height: 22px;
}

.html-for-mac .file-node-open-state, .html-for-mac .file-node-icon {
    line-height: 26px;
}

::-webkit-scrollbar-thumb {
    background: rgba(230, 230, 230, 0.30);
}

::-webkit-scrollbar-thumb:active {
    background: rgba(230, 230, 230, 0.50);
}

#typora-sidebar:hover div.sidebar-content-content::-webkit-scrollbar-thumb:horizontal {
    background: rgba(230, 230, 230, 0.30);
}

.nav-group-item:active {
    background-color: #474d54 !important;
}

.md-search-hit {
    background: rgba(199, 140, 60, 0.81);
    color: #eee;
}

.md-search-hit * {
    color: #eee;
}

#md-searchpanel input {
    color: white;
}

.modal-backdrop.in {
    opacity: 1;
    backdrop-filter: blur(1px);
}

.clear-btn-icon {
    top: 8px;
}

 :root {--mermaid-font-zoom:1em ;}  @media print { @page {margin: 0 0 0 0;} body.typora-export {padding-left: 0; padding-right: 0;} #write {padding:0;}}
</style><title>Chapter 6 - Figures</title>
</head>
<body class='typora-export'><div class='typora-export-content'>
<div id='write'  class=''><center><h1>Chapter 6 - Figures</h1></center><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721064527-554711738.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-3 - <span class="title">Genes can be expressed with different efficiencies</span></span></center>
<p>Gene A is transcribed and translated much more efficiently than <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/">gene</a> B. This allows the amount of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> A in the cell to be much greater than that of protein B.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721064707-92166867.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-4 - <span class="title">The chemical structure of RNA</span></span></center>
<p>(A) <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> contains the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5842/">sugar</a> ribose, which differs from deoxyribose, the sugar used in <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a>, by the presence of an additional -<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5308/">OH</a> group. (B) RNA contains the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a> uracil, which differs from thymine, the equivalent base in DNA, by the absence of a -CH<sub>3</sub> group. (C) A short length of RNA. The phosphodiester chemical <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5397/">linkage</a> between nucleotides in RNA is the same as that in DNA.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721064909-330566579.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-5 - <span class="title">Uracil forms base pairs with adenine</span></span></center>
<p>The absence of a methyl group in U has no effect on <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a>-pairing; thus, U-A base pairs closely resemble T-A base pairs (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A596/figure/A599/?report=objectonly" rid-figpopup="figA599" rid-ob="figobA599" target="object">Figure 4-4</a>).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721065088-69614014.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-6 - <span class="title">RNA can fold into specific structures</span></span></center>
<p><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> is largely single-stranded, but it often contains short stretches of nucleotides that can form conventional <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a>-pairs with <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/">complementary</a> sequences found elsewhere on the same <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>. These interactions, along with additional “nonconventional” base-pair interactions, allow an RNA molecule to fold into a three-dimensional structure that is <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by its sequence of nucleotides. (A) Diagram of a folded RNA structure showing only conventional base-pair interactions; (B) structure with both conventional <i>(red)</i> and nonconventional <i>(green)</i> base-pair interactions; (C) structure of an actual RNA, a portion of a group 1 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1032/?report=objectonly" rid-figpopup="figA1032" rid-ob="figobA1032" target="object">Figure 6-36</a>). Each conventional base-pair interaction is indicated by a “rung” in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/">double helix</a>. Bases in other configurations are indicated by broken rungs.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721065293-508522865.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-7 - <span class="title">DNA transcription produces a single-stranded RNA molecule that is complementary to one strand of DNA</span></span></center><center><img src="https://39363.org/IMAGE_BUCKET/1631721065457-212397978.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-8 - <span class="title">DNA is transcribed by the enzyme RNA polymerase</span></span></center>
<p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> <i>(pale blue)</i> moves stepwise along the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a>, unwinding the DNA helix at its <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/">active site</a>. As it progresses, the polymerase adds nucleotides (here, <i>small “T” shapes)</i> one by one to the RNA chain at the polymerization site using an exposed DNA strand as a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/">template</a>. The RNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a> is thus a single-stranded <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/">complementary</a> copy of one of the two DNA strands. The polymerase has a rudder (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A989/?report=objectonly" rid-figpopup="figA989" rid-ob="figobA989" target="object">Figure 6-11</a>) that displaces the newly formed RNA, allowing the two strands of DNA behind the polymerase to rewind. A short region of DNA/RNA helix (approximately nine nucleotides in length) is therefore formed only transiently, and a “window” of DNA/RNA helix therefore moves along the DNA with the polymerase. The incoming nucleotides are in the form of ribonucleoside triphosphates (ATP, UTP, CTP, and GTP), and the energy stored in their phosphate-phosphate bonds provides the driving force for the polymerization <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A754/figure/A758/?report=objectonly" rid-figpopup="figA758" rid-ob="figobA758" target="object">Figure 5-4</a>). (Adapted from a figure kindly supplied by Robert Landick.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721065643-995550205.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-9 - <span class="title">Transcription of two genes as observed under the electron microscope</span></span></center>
<p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5460/">micrograph</a> shows many molecules of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> simultaneously transcribing each of two adjacent genes. Molecules of RNA polymerase are visible as a series of dots along the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> with the newly synthesized transcripts (fine threads) attached to them. The RNA molecules (ribosomal RNAs) shown in this example are not translated into <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> but are instead used directly as components of ribosomes, the machines on which translation takes place. The particles at the 5′ end (the free end) of each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/">rRNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a> are believed to reflect the beginnings of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5754/">ribosome</a> assembly. From the lengths of the newly synthesized transcripts, it can be deduced that the RNA polymerase molecules are transcribing from left to right. (Courtesy of Ulrich Scheer.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721065817-289160108.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-10 - <span class="title">The transcription cycle of bacterial RNA polymerase</span></span></center>
<p>In step 1, the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> holoenzyme (core polymerase plus σ factor) forms and then locates a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/">promoter</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" rid-figpopup="figA991" rid-ob="figobA991" target="object">Figure 6-12</a>). The polymerase unwinds the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> at the position at which transcription is to begin (step 2) and begins transcribing (step 3). This initial RNA synthesis (sometimes called “abortive initiation”) is relatively inefficient. However, once RNA polymerase has managed to synthesize about 10 nucleotides of RNA, σ relaxes its grip, and the polymerase undergoes a series of conformational changes (which probably includes a tightening of its jaws and the placement of RNA in the exit channel [see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A989/?report=objectonly" rid-figpopup="figA989" rid-ob="figobA989" target="object">Figure 6-11</a>]). The polymerase now shifts to the elongation mode of RNA synthesis (step 4), moving rightwards along the DNA in this diagram. During the elongation mode (step 5) transcription is highly processive, with the polymerase leaving the DNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/">template</a> and releasing the newly transcribed RNA only when it encounters a termination signal (step 6). Termination signals are encoded in DNA and many function by forming an RNA structure that destabilizes the polymerase's hold on the RNA, as shown here. In bacteria, all RNA molecules are synthesized by a single type of RNA polymerase and the cycle depicted in the figure therefore applies to the production of mRNAs as well as structural and catalytic RNAs. (Adapted from a figure kindly supplied by Robert Landick.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721066020-428939549.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-11 - <span class="title">The structure of a bacterial RNA polymerase</span></span></center>
<p>Two depictions of the three-dimensional structure of a bacterial <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a>, with the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> and RNA modeled in. This RNA polymerase is formed from four different subunits, indicated by different colors <i>(right).</i> The DNA strand used as a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/">template</a> is <i>red</i>, and the non-template strand is <i>yellow</i>. The rudder wedges apart the DNA-RNA hybrid as the polymerase moves. For simplicity only the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5659/">polypeptide backbone</a> of the rudder is shown in the right-hand figure, and the DNA exiting from the polymerase has been omitted. Because the RNA polymerase is depicted in the elongation mode, the σ factor is absent. (Courtesy of Seth Darst.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721066218-959763135.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-12 - <span class="title">Consensus sequence for the major class of <i>E. coli</i>
<a href="/books/n/mboc4/A4754/#A5684" target="mainwindow">promoters</a>.</span></span></center>
<p>(A) The promoters are characterized by two hexameric <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> sequences, the -35 sequence and the -10 sequence named for their approximate location relative to the start point of transcription (designated +1). For convenience, the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequence of a single strand of DNA is shown; in reality the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> recognizes the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/">promoter</a> as double-stranded DNA. On the basis of a comparison of 300 promoters, the frequencies of the four nucleotides at each position in the -35 and -10 hexamers are given. The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5023/">consensus sequence</a>, shown <i>below</i> the graph, reflects the most common nucleotide found at each position in the collection of promoters. The sequence of nucleotides between the -35 and -10 hexamers shows no significant similarities among promoters. (B) The distribution of spacing between the -35 and -10 hexamers found in <i>E. coli</i> promoters. The information displayed in these two graphs applies to <i>E. coli</i> promoters that are recognized by RNA polymerase and the major σ factor (designated σ<sup>70</sup>). As we shall see in the next chapter, bacteria also contain minor σ factors, each of which recognizes a different promoter sequence. Some particularly strong promoters recognized by RNA polymerase and σ<sup>70</sup> have an additional sequence, located upstream (to the <i>left,</i> in the figure) of the -35 hexamer, which is recognized by another <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a> of RNA polymerase.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721066423-870552831.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-13 - <span class="title">The importance of RNA polymerase orientation</span></span></center>
<p>The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> strand serving as <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/">template</a> must be traversed in a 3′ to 5′ direction, as illustrated in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A984/?report=objectonly" rid-figpopup="figA984" rid-ob="figobA984" target="object">Figure 6-9</a>. Thus, the direction of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> movement determines which of the two DNA strands is to serve as a template for the synthesis of RNA, as shown in (A) and (B). Polymerase direction is, in turn, <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by the orientation of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/">promoter</a> sequence, the site at which the RNA polymerase begins transcription.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721066595-20871281.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-14 - <span class="title">Directions of transcription along a short portion of a bacterial chromosome</span></span></center>
<p>Some genes are transcribed using one <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> strand as a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/">template</a>, while others are transcribed using the other DNA strand. The direction of transcription is <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5070/">determined</a> by the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/">promoter</a> at the beginning of each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/">gene</a> <i>(green arrowheads).</i> Approximately 0.2% (9000 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a> pairs) of the <i>E. coli</i> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4977/">chromosome</a> is depicted here. The genes transcribed from <i>left</i> to <i>right</i> use the bottom DNA strand as the template; those transcribed from <i>right</i> to <i>left</i> use the top strand as the template.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721066763-804901595.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-15 - <span class="title">Structural similarity between a bacterial RNA polymerase and a eucaryotic RNA polymerase II</span></span></center>
<p>Regions of the two <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> polymerases that have similar structures are indicated in <i>green</i>. The eucaryotic polymerase is larger than the bacterial <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5137/">enzyme</a> (12 subunits instead of 5), and some of the additional regions are shown in <i>gray.</i> The <i>blue</i> spheres represent Zn atoms that serve as structural components of the polymerases, and the <i>red</i> sphere represents the Mg atom present at the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/">active site</a>, where polymerization takes place. The RNA polymerases in all modern-day cells (bacteria, archaea, and eucaryotes) are closely related, indicating that the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4877/">basic</a> features of the enzyme were in place before the divergence of the three major branches of life. (Courtesy of P. Cramer and R. Kornberg.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721066944-63433241.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-16 - <span class="title">Initiation of transcription of a eucaryotic gene by RNA polymerase II</span></span></center>
<p>To begin transcription, <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> requires a number of general transcription factors (called TFIIA, TFIIB, and so on). (A) The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/">promoter</a> contains a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> sequence called the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5856/">TATA box</a>, which is located 25 nucleotides away from the site at which transcription is initiated. (B) The TATA box is recognized and bound by <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5882/">transcription factor</a> TFIID, which then enables the adjacent binding of TFIIB (C). For simplicity the DNA distortion produced by the binding of TFIID (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1002/?report=objectonly" rid-figpopup="figA1002" rid-ob="figobA1002" target="object">Figure 6-18</a>) is not shown. (D) The rest of the general transcription factors, as well as the RNA polymerase itself, assemble at the promoter. (E) TFIIH then uses ATP to pry apart the DNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/">double helix</a> at the transcription start point, allowing transcription to begin. TFIIH also phosphorylates RNA polymerase II, changing its <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5019/">conformation</a> so that the polymerase is released from the general factors and can begin the elongation phase of transcription. As shown, the site of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5629/">phosphorylation</a> is a long C-terminal <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> tail that extends from the polymerase <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>. The assembly scheme shown in the figure was deduced from experiments performed <i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5327/">in vitro</a></i>, and the exact order in which the general transcription factors assemble on promoters in cells is not known with certainty. In some cases, the general factors are thought to first assemble with the polymerase, with the whole assembly subsequently binding to the DNA in a single step. The general transcription factors have been highly conserved in evolution; some of those from human cells can be replaced in biochemical experiments by the corresponding factors from simple yeasts.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721067137-651624791.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-17 - <span class="title">Consensus sequences found in the vicinity of eucaryotic RNA polymerase II start points</span></span></center>
<p>The name given to each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5023/">consensus sequence</a> <i>(first column)</i> and the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5217/">general transcription factor</a> that recognizes it <i>(last column)</i> are indicated. N indicates any <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a>, and two nucleotides separated by a slash indicate an equal probability of either nucleotide at the indicated position. In reality, each consensus sequence is a shorthand representation of a histogram similar to that of <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" rid-figpopup="figA991" rid-ob="figobA991" target="object">Figure 6-12</a>. For most <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> II transcription start points, only two or three of the four sequences are present. For example, most polymerase II promoters have a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5856/">TATA box</a> sequence, and those that do not typically have a “strong” INR sequence. Although most of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> sequences that influence transcription initiation are located “upstream” of the transcription start point, a few, such as the DPE shown in the figure, are located in the transcribed region.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721067368-241434463.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-18 - <span class="title">Three-dimensional structure of TBP (TATA-binding protein) bound to DNA</span></span></center>
<p>The TBP is the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5840/">subunit</a> of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5217/">general transcription factor</a> TFIID that is responsible for recognizing and binding to the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5856/">TATA box</a> sequence in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> <i>(red).</i> The unique DNA bending caused by TBP—two kinks in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/">double helix</a> separated by partly unwound DNA—may serve as a landmark that helps to attract the other general transcription factors. TBP is a single <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5658/">polypeptide</a> chain that is folded into two very similar domains <i>(blue</i> and <i>green).</i> (Adapted from J.L. Kim et al., <i>Nature</i> 365:520–527, 1993.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721067553-58721257.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-19 - <span class="title">Transcription initiation by RNA polymerase II in a eucaryotic cell</span></span></center>
<p>Transcription initiation <i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5328/">in vivo</a></i> requires the presence of transcriptional activator proteins. As described in Chapter 7, these proteins bind to specific short sequences in <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a>. Although only one is shown here, a typical eucaryotic <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/">gene</a> has many activator proteins, which together determine its rate and pattern of transcription. Sometimes acting from a distance of several thousand <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> pairs (indicated by the dashed DNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>), these gene regulatory proteins help <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a>, the general factors, and the mediator all to assemble at the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/">promoter</a>. In addition, activators attract ATP-dependent <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4974/">chromatin</a>-remodeling complexes and <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5283/">histone</a> acetylases.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721067720-474462486.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-20 - <span class="title">Superhelical tension in DNA causes DNA supercoiling</span></span></center>
<p>(A) For a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> with one free end (or a nick in one strand that serves as a swivel), the DNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5106/">double helix</a> rotates by one turn for every 10 <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> pairs opened. (B) If rotation is prevented, superhelical tension is introduced into the DNA by helix opening. One way of accommodating this tension would be to increase the helical twist from 10 to 11 nucleotide pairs per turn in the double helix that remains in this example; the DNA helix, however, resists such a deformation in a springlike fashion, preferring to relieve the superhelical tension by bending into supercoiled loops. As a result, one DNA supercoil forms in the DNA double helix for every 10 nucleotide pairs opened. The supercoil formed in this case is a positive supercoil. (C) Supercoiling of DNA is induced by a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> tracking through the DNA double helix. The two ends of the DNA shown here are unable to rotate freely relative to each other, and the protein molecule is assumed also to be prevented from rotating freely as it moves. Under these conditions, the movement of the protein causes an excess of helical turns to accumulate in the DNA helix ahead of the protein and a deficit of helical turns to arise in the DNA behind the protein, as shown.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721067919-198158141.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-21 - <span class="title">Summary of the steps leading from gene to protein in eucaryotes and bacteria</span></span></center>
<p>The final level of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> in the cell depends on the efficiency of each step and on the rates of degradation of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> and protein molecules. (A) In eucaryotic cells the RNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> produced by transcription alone (sometimes referred to as the primary <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a>) would contain both coding (<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/">exon</a>) and noncoding (<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a>) sequences. Before it can be translated into protein, the two ends of the RNA are modified, the introns are removed by an enzymatically catalyzed <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/">RNA splicing</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a>, and the resulting <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> is transported from the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a> to the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5053/">cytoplasm</a>. Although these steps are depicted as occurring one at a time, in a sequence, in reality they are coupled and different steps can occur simultaneously. For example, the RNA cap is added and splicing typically begins before transcription has been completed. Because of this coupling, complete primary RNA transcripts do not typically exist in the cell. (B) In procaryotes the production of mRNA molecules is much simpler. The 5′ end of an mRNA molecule is produced by the initiation of transcription by <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a>, and the 3′ end is produced by the termination of transcription. Since procaryotic cells lack a nucleus, transcription and translation take place in a common <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5009/">compartment</a>. In fact, translation of a bacterial mRNA often begins before its synthesis has been completed.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721068118-99397848.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-22 - <span class="title">A comparison of the structures of procaryotic and eucaryotic mRNA molecules</span></span></center>
<p>(A) The 5′ and 3′ ends of a bacterial <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> are the unmodified ends of the chain synthesized by the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a>, which initiates and terminates transcription at those points, respectively. The corresponding ends of a eucaryotic mRNA are formed by adding a 5′ cap and by <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/">cleavage</a> of the pre-mRNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a> and the addition of a poly-A tail, respectively. The figure also illustrates another difference between the procaryotic and eucaryotic mRNAs: bacterial mRNAs can contain the instructions for several different proteins, whereas eucaryotic mRNAs nearly always contain the information for only a single <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>. (B) The structure of the cap at the 5′ end of eucaryotic mRNA molecules. Note the unusual 5′-to-5′ <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5397/">linkage</a> of the 7-methyl <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a> to the remainder of the RNA. Many eucaryotic mRNAs carry an additional modification: the 2′-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5308/">hydroxyl</a> group on the second ribose <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5842/">sugar</a> in the mRNA is methylated (not shown).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721068298-742025054.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-23 - <span class="title">The “RNA factory” concept for eucaryotic RNA polymerase II</span></span></center>
<p>Not only does the polymerase transcribe <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> into <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a>, but it also carries pre-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a>-processing proteins on its tail, which are then transferred to the nascent RNA at the appropriate time. There are many RNA-processing enzymes, and not all travel with the polymerase. For <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/">RNA splicing</a>, for example, only a few critical components are carried on the tail; once transferred to an RNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>, they serve as a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5560/">nucleation</a> site for the remaining components. The RNA-processing proteins first bind to the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> tail when it is phosphorylated late in the process of transcription initiation (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1000/?report=objectonly" rid-figpopup="figA1000" rid-ob="figobA1000" target="object">Figure 6-16</a>). Once RNA polymerase II finishes transcribing, it is released from DNA, the phosphates on its tail are removed by soluble phosphatases, and it can reinitiate transcription. Only this dephosphorylated form of RNA polymerase II is competent to start RNA synthesis at a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5684/">promoter</a>.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721068470-538994635.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-24 - <span class="title">The reactions that cap the 5′ end of each RNA molecule synthesized by RNA polymerase II</span></span></center>
<p> The final cap contains a novel 5′-to-5′ <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5397/">linkage</a> between the positively charged 7-methyl <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5737/">residue</a> and the 5′ end of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1009/?report=objectonly" rid-figpopup="figA1009" rid-ob="figobA1009" target="object">Figure 6-22B</a>). The letter N represents any one of the four ribonucleotides, although the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> that starts an RNA chain is usually a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5709/">purine</a> (an A or a G). (After A.J. Shatkin, <i>BioEssays</i> 7:275–277, 1987. © ICSU Press.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721068631-605144221.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-25 - <span class="title">Structure of two human genes showing the arrangement of exons and introns</span></span></center>
<p>(A) The relatively small β-globin <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/">gene</a>, which encodes one of the subunits of the oxygen-carrying <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5272/">hemoglobin</a>, contains 3 exons (see also <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A596/figure/A603/?report=objectonly" rid-figpopup="figA603" rid-ob="figobA603" target="object">Figure 4-7</a>). (B) The much larger Factor VIII gene contains 26 exons; it codes for a protein (Factor VIII) that functions in the blood-clotting pathway. Mutations in this gene are responsible for the most prevalent form of hemophilia.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721068787-837545481.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-26 - <span class="title">The RNA splicing reaction</span></span></center>
<p>(A) In the first step, a specific adenine <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a> sequence (indicated in <i>red)</i> attacks the 5′ splice site and cuts the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5842/">sugar</a>-phosphate backbone of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> at this point. The cut 5′ end of the intron becomes covalently linked to the adenine nucleotide, as shown in detail in (B), thereby creating a loop in the RNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>. The released free 3′-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5308/">OH</a> end of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/">exon</a> sequence then reacts with the start of the next exon sequence, joining the two exons together and releasing the intron sequence in the shape of a <i>lariat.</i> The two exon sequences thereby become joined into a continuous coding sequence; the released intron sequence is degraded in due course.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721068988-299863648.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-27 - <span class="title">Alternative splicing of the α-tropomyosin gene from rat</span></span></center>
<p> α-tropomyosin is a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5002/">coiled-coil</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A388/figure/A405/?report=objectonly" rid-figpopup="figA405" rid-ob="figobA405" target="object">Figure 3-11</a>) that regulates contraction in muscle cells. The primary <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a> can be spliced in different ways, as indicated in the figure, to produce distinct mRNAs, which then give rise to variant proteins. Some of the splicing patterns are specific for certain types of cells. For example, the α-tropomyosin made in <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5835/">striated muscle</a> is different from that made from the same <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/">gene</a> in smooth muscle. The arrowheads in the top part of the figure demark the sites where <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/">cleavage</a> and poly-A addition can occur.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721069167-432128970.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-28 - <span class="title">The consensus nucleotide sequences in an RNA molecule that signal the beginning and the end of most introns in humans</span></span></center>
<p>Only the three blocks of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequences shown are required to remove an <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a> sequence; the rest of the intron can be occupied by any nucleotide. Here A, <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a>, U, and C are the standard <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> nucleotides; R stands for either A or G; Y stands for either C or U. The A highlighted in <i>red</i> forms the branch point of the lariat produced by splicing. Only the GU at the start of the intron and the AG at its end are invariant nucleotides in the splicing consensus sequences. The remaining positions (even the branch point A) can be occupied by a variety of nucleotides, although the indicated nucleotides are preferred. The distances along the RNA between the three splicing consensus sequences are highly variable; however, the distance between the branch point and 3′ splice junction is typically much shorter than that between the 5′ splice junction and the branch point.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721069338-843413083.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-29 - <span class="title">The RNA splicing mechanism</span></span></center>
<p><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/">RNA splicing</a> is catalyzed by an assembly of snRNPs (shown as <i>colored circles)</i> plus other proteins (most of which are not shown), which together constitute the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/">spliceosome</a>. The spliceosome recognizes the splicing signals on a pre-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>, brings the two ends of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a> together, and provides the enzymatic activity for the two <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> steps (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1015/?report=objectonly" rid-figpopup="figA1015" rid-ob="figobA1015" target="object">Figure 6-26</a>). The branch-point site is first recognized by the BBP (branch-point binding <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>) and U2AF, a helper protein. In the next steps, the U2 snRNP displaces BBP and U2AF and forms <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a> pairs with the branch-point site <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5023/">consensus sequence</a>, and the U1 snRNP forms base-pairs with the 5′ splice junction (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" rid-figpopup="figA1021" rid-ob="figobA1021" target="object">Figure 6-30</a>). At this point, the U4/U6•U5 “triple” snRNP enters the spliceosome. In this triple snRNP, the U4 and U6 snRNAs are held firmly together by base-pair interactions and the U5 snRNP is more loosely associated. Several RNA-RNA rearrangements then occur that break apart the U4/U6 base pairs (as shown, the U4 snRNP is ejected from the splicesome before splicing is complete) and allow the U6 snRNP to displace U1 at the 5′ splice junction (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1021/?report=objectonly" rid-figpopup="figA1021" rid-ob="figobA1021" target="object">Figure 6-30</a>). Subsequent rearrangements create the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4772/">active site</a> of the spliceosome and position the appropriate portions of the pre-mRNA <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5838/">substrate</a> for the splicing reaction to occur. Although not shown in the figure, each splicing event requires additional proteins, some of which hydrolyze ATP and promote the RNA-RNA rearrangements.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721069560-121122424.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-30 - <span class="title">Several of the rearrangements that take place in the spliceosome during pre-mRNA splicing</span></span></center>
<p>Shown here are the details for the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5937/">yeast</a> <i><a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5770/">Saccharomyces</a> cerevisiae</i>, in which the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequences involved are slightly different from those in human cells. (A) The exchange of U1 snRNP for U6 snRNP occurs before the first phosphoryl-transfer <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" rid-figpopup="figA1020" rid-ob="figobA1020" target="object">Figure 6-29</a>). This exchange allows the 5′ splice site to be read by two different snRNPs, thereby increasing the accuracy of 5′ splice site selection by the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/">spliceosome</a>. (B) The branch-point site is first recognized by BBP and subsequently by U2 snRNP; as in (A), this “check and recheck” strategy provides increased accuracy of site selection. The binding of U2 to the branch-point forces the appropriate adenine (in <i>red)</i> to be unpaired and thereby activates it for the attack on the 5′ splice site (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" rid-figpopup="figA1020" rid-ob="figobA1020" target="object">Figure 6-29</a>). This, in combination with recognition by BBP, is the way in which the spliceosome accurately chooses the adenine that is ultimately to form the branch point. (C) After the first phosphoryl-transfer reaction <i>(left)</i> has occurred, U5 snRNP undergoes a rearrangement that brings the two exons into close proximity for the second phosphoryl-transfer reaction <i>(right).</i> The snRNAs both position the reactants and provide (either all or in part) the catalytic site for the two reactions. The U5 snRNP is present in the spliceosome before this rearrangement occurs; for clarity it has been omitted from the left panel. As discussed in the text, all of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a>-RNA rearrangements shown in this figure (as well as others that occur in the spliceosome but are not shown) require the participation of additional proteins and ATP hydrolysis.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721069743-476446361.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-31 - <span class="title">Two types of splicing errors</span></span></center>
<p>Both types might be expected to occur frequently if splice-site selection were performed by the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/">spliceosome</a> on a preformed, <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a>-free <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a>. “Cryptic” splicing signals are <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequences of RNA that closely resemble true splicing signals.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721069911-246453253.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-32 - <span class="title">Variation in intron and exon lengths in the human, worm, and fly genomes</span></span></center>
<p>(A) Size distribution of exons. (B) Size distribution of introns. Note that <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/">exon</a> length is much more uniform than <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a> length. (Adapted from International Human Genome Sequencing Consortium, <i>Nature</i> 409:860–921, 2001.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721070099-856510900.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-33 - <span class="title">The exon definition hypothesis</span></span></center>
<p>According to one proposal, SR proteins bind to each <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/">exon</a> sequence in the pre-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> and thereby help to guide the snRNPs to the proper <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a>/exon boundaries. This demarcation of exons by the SR proteins occurs co-transcriptionally, beginning at the CBC (cap-binding <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5014/">complex</a>) at the 5′ end. As indicated, the intron sequences in the pre-mRNA, which can be extremely long, are packaged into hnRNP (heterogeneous nuclear ribonucleoprotein) complexes that compact them into more manageable structures and perhaps mask cryptic splice sites. Each hnRNP complex forms a particle approximately twice the diameter of a <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5566/">nucleosome</a>, and the core is composed of a set of at least eight different proteins. It has been proposed that hnRNP proteins preferentially associate with intron sequences and that this preference also helps the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/">spliceosome</a> distinguish introns from exons. However, as shown, at least some hnRNP proteins may bind to exon sequences but their role, if any, in exon definition has yet to be established. (Adapted from R. Reed, <i>Curr. Opin. Cell Biol.</i> 12:340–345, 2000.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721070276-594334182.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-34 - <span class="title">Outline of the mechanisms used for three types of RNA splicing</span></span></center>
<p>(A) Three types of spliceosomes. The major <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/">spliceosome</a> <i>(left),</i> the AT-AC spliceosome (middle), and the trans-spliceosome <i>(right)</i> are each shown at two stages of assembly. The U5 snRNP is the only component that is common to all three spliceosomes. Introns removed by the AT-AC spliceosome have a different set of consensus <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequences from those removed by the major spliceosome. In humans, it is estimated that 0.1% of introns are removed by the AT-AC spliceosome. In <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5897/">trans-splicing</a>, the SL snRNP is consumed in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a> because a portion of the SL <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5812/">snRNA</a> becomes the first <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/">exon</a> of the mature <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a>. (B) The major U6 snRNP and the U6 AT-AC snRNP both recognize the 5′ splice junction, but they do so through a different set of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a>-pair interactions. The sequences shown are from humans. (Adapted from Y.-T. Yu et al., The <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> World, pp. 487–524. Cold Spring Harbor, New York: Cold Spring Harbor Laboratory Press, 1999.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721070483-367364263.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-35 - <span class="title">Abnormal processing of the β-globin primary RNA transcript in humans with the disease β thalassemia</span></span></center>
<p> In the examples shown, the disease is caused by splice-site mutations, denoted by <i>black arrowheads</i>. The <i>dark blue boxes</i> represent the three normal <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/">exon</a> sequences; the <i>red lines</i> are used to indicate the 5′ and 3′ splice sites that are used in splicing the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a>. The <i>light blue boxes</i> depict new <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequences included in the final <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> as a result of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5502/">mutation</a>. Note that when a mutation leaves a normal splice site without a partner, an exon is skipped or one or more abnormal “cryptic” splice sites nearby is used as the partner site, as in (C) and (D). (Adapted in part from S.H. Orkin, in The Molecular Basis of Blood Diseases [<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a>. Stamatoyannopoulos et al., eds.], pp. 106–126. Philadelphia: Saunders, 1987.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721070688-409870109.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-36 - <span class="title">The two known classes of self-splicing intron sequences</span></span></center>
<p>The group I <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5352/">intron</a> sequences bind a free <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> to a specific site on the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> to initiate splicing, while the group II intron sequences use an especially reactive A nucleotide in the intron sequence itself for the same purpose. The two mechanisms have been drawn to emphasize their similarities. Both are normally aided in the cell by proteins that speed up the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5717/">reaction</a>, but the catalysis is nevertheless mediated by the RNA in the intron sequence. Both types of self-splicing reactions require the intron to be folded into a highly specific three-dimensional structure that provides the catalytic activity for the reaction (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A980/?report=objectonly" rid-figpopup="figA980" rid-ob="figobA980" target="object">Figure 6-6</a>). The mechanism used by group II intron sequences releases the intron as a lariat structure and closely resembles the pathway of pre-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> splicing catalyzed by the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5823/">spliceosome</a> (compare with <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1020/?report=objectonly" rid-figpopup="figA1020" rid-ob="figobA1020" target="object">Figure 6-29</a>). The great majority of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5763/">RNA splicing</a> in eucaryotic cells is performed by the spliceosome, and self-splicing RNAs represent unusual cases. (Adapted from T.R. Cech, <i>Cell</i> 44:207–210, 1986.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721070903-923257350.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-37 - <span class="title">Consensus nucleotide sequences that direct cleavage and polyadenylation to form the 3′ end of a eucaryotic mRNA</span></span></center>
<p> These sequences are encoded in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5222/">genome</a> and are recognized by specific proteins after they are transcribed into <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a>. The hexamer AAUAAA is bound by CPSF, the GU-rich element beyond the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4992/">cleavage</a> site by CstF (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1035/?report=objectonly" rid-figpopup="figA1035" rid-ob="figobA1035" target="object">Figure 6-38</a>), and the CA sequence by a third factor required for the cleavage step. Like other consensus <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequences discussed in this chapter (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A991/?report=objectonly" rid-figpopup="figA991" rid-ob="figobA991" target="object">Figure 6-12</a>), the sequences shown in the figure represent a variety of individual cleavage and polyadenylation signals.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721071075-916990625.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-38 - <span class="title">Some of the major steps in generating the 3′ end of a eucaryotic mRNA</span></span></center>
<p> This process is much more complicated than the analogous process in bacteria, where the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> simply stops at a termination signal and releases both the 3′ end of its <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5879/">transcript</a> and the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5084/">DNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5862/">template</a> (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A988/?report=objectonly" rid-figpopup="figA988" rid-ob="figobA988" target="object">Figure 6-10</a>).</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721071278-402750127.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-39 - <span class="title">Transport of a large mRNA molecule through the nuclear pore complex</span></span></center>
<p>(A) The maturation of a Balbiani Ring <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5486/">molecule</a> as it is synthesized by <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5760/">RNA polymerase</a> and packaged by a variety of nuclear proteins. This drawing of unusually abundant RNA produced by an insect cell is based on EM micrographs such as that shown in (B). Balbiani Rings are described in Chapter 4. (A, adapted from B. Daneholt, <i>Cell</i> 88:585–588, 1997; B, from B.J. Stevens and H. Swift, <i>J</i>. <i>Cell Biol</i>. 31:55–77, 1966. © The Rockefeller University Press.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721071444-698023556.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-40 - <span class="title">Schematic illustration of an “export-ready” mRNA molecule and its transport through the nuclear pore</span></span></center>
<p>As indicated, some proteins travel with the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> as it moves through the pore, whereas others remain in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a>. Once in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5053/">cytoplasm</a>, the mRNA continues to shed previously bound proteins and acquire new ones; these substitutions affect the subsequent translation of the message. Because some are transported with the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a>, the proteins that become bound to an mRNA in the nucleus can influence its subsequent stability and translation in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5055/">cytosol</a>. RNA export factors, shown in the nucleus, play an active role in transporting the mRNA to the cytosol (see <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A2154/figure/A2167/?report=objectonly" rid-figpopup="figA2167" rid-ob="figobA2167" target="object">Figure 12-16</a>). Some are deposited at <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5161/">exon</a>-exon boundaries as splicing is completed, thus signifying those regions of the RNA that have been properly spliced.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721071645-172644297.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-41 - <span class="title">Transcription from tandemly arranged rRNA genes, as seen in the electron microscope</span></span></center>
<p>The pattern of alternating transcribed <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5215/">gene</a> and nontranscribed spacer is readily seen. A higher-magnification view was shown in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A984/?report=objectonly" rid-figpopup="figA984" rid-ob="figobA984" target="object">Figure 6-9</a>. (From V.E. Foe, <i>Cold Spring Harbor Symp. Quant</i>. <i>Biol</i>. 42:723–740, 1978.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721071820-690320537.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-42 - <span class="title">The chemical modification and nucleolytic processing of a eucaryotic 45S precursor rRNA molecule into three separate ribosomal RNAs</span></span></center>
<p>As indicated, two types of chemical modifications (shown in <a class="figpopup" href="https://www.ncbi.nlm.nih.gov/books/NBK26887/figure/A1042/?report=objectonly" rid-figpopup="figA1042" rid-ob="figobA1042" target="object">Figure 6-43</a>) are made to the precursor <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/">rRNA</a> before it is cleaved. Nearly half of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> sequences in this precursor rRNA are discarded and degraded in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a>. The rRNAs are named according to their “S” values, which refer to their rate of sedimentation in an ultra-centrifuge. The larger the S value, the larger the rRNA.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721071986-571385312.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-43 - <span class="title">Modifications of the precursor rRNA by guide RNAs</span></span></center>
<p>(A) Two prominent covalent modifications occur after <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/">rRNA</a> synthesis; the differences from the initially incorporated <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5567/">nucleotide</a> are indicated by <i>red</i> atoms. (B) As indicated, snoRNAs locate the sites of modification by <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4875/">base</a>-pairing to <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5012/">complementary</a> sequences on the precursor rRNA. The snoRNAs are bound to proteins, and the complexes are called snoRNPs. snoRNPs contain the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5756/">RNA</a> modification activities, presumably contributed by the proteins but possibly by the snoRNAs themselves.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721072167-591472184.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-44 - <span class="title">Electron micrograph of a thin section of a nucleolus in a human fibroblast, showing its three distinct zones</span></span></center>
<p>(A) View of entire <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a>. (B) High-power view of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/">nucleolus</a>. It is believed that transcription of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/">rRNA</a> genes takes place between the fibrillar center and the dense fibrillar component and that processing of the rRNAs and their assembly into ribosomes proceeds outward from the dense fibrillar component to the surrounding granular components. (Courtesy of E.<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a>. Jordan and J. McGovern.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721072376-81469627.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-45 - <span class="title">Changes in the appearance of the nucleolus in a human cell during the cell cycle</span></span></center>
<p>Only the cell <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a> is represented in this diagram. In most eucaryotic cells the nuclear <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5438/">membrane</a> breaks down during <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5477/">mitosis</a>, as indicated by the dashed circles.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721072588-674548209.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-46 - <span class="title">Nucleolar fusion</span></span></center>
<p>These light micrographs of human fibroblasts grown in culture show various stages of nucleolar fusion. After <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5477/">mitosis</a>, each of the ten human chromosomes that carry a cluster of <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/">rRNA</a> genes begins to form a tiny <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/">nucleolus</a>, but these rapidly coalesce as they grow to form the single large nucleolus typical of many <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5350/">interphase</a> cells. (Courtesy of E.<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5198/">G</a>. Jordan and J. McGovern.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721072757-23383058.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-47 - <span class="title">The function of the nucleolus in ribosome and other ribonucleoprotein synthesis</span></span></center>
<p>The 45S precursor <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5767/">rRNA</a> is packaged in a large ribonucleoprotein particle containing many ribosomal proteins imported from the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5053/">cytoplasm</a>. While this particle remains in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/">nucleolus</a>, selected pieces are added and others discarded as it is processed into immature large and small ribosomal subunits. The two ribosomal subunits are thought to attain their final functional form only as each is individually transported through the nuclear pores into the cytoplasm. Other ribonucleoprotein complexes, including <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5858/">telomerase</a> shown here, are also assembled in the nucleolus.</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721072979-904945282.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-48 - <span class="title">Visualization of chromatin and nuclear bodies</span></span></center>
<p>(A)-(D) show micrographs of the same human cell <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a>, each processed differently to show a particular set of nuclear structures. (E) shows an enlarged superposition of all four individual images. (A) shows the location of the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5688/">protein</a> fibrillarin (a component of several snoRNPs), which is present at both nucleoli and Cajal bodies, the latter indicated by arrows. (B) shows interchromatin granule clusters or “speckles” detected by using antibodies against a protein involved in pre-<a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5496/">mRNA</a> splicing. (C) is stained to show bulk <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A4974/">chromatin</a>. (D) shows the location of the protein coilin, which is present at Cajal bodies (indicated by arrows). (From J.R. Swedlow and A.I. Lamond, <i>Gen. Biol.</i> 2:1–7, 2001; micrographs courtesy of Judith Sleeman.)</p><p>&nbsp;</p><center><img src="https://39363.org/IMAGE_BUCKET/1631721073170-25294014.jpg" style="zoom:100%;"></center>
<center><span style="color:#97CA8A">Figure 6-49 - <span class="title">Schematic view of subnuclear structures</span></span></center>
<p>A typical vertebrate <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a> has several Cajal bodies, which are proposed to be the sites where snRNPs and snoRNPs undergo their final modifications. Interchromatin granule clusters are proposed to be storage sites for fully mature snRNPs. A typical vertebrate nucleus has 20–50 interchromatin granule clusters.</p><p>After their initial synthesis, snRNAs are exported from the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5568/">nucleus</a>, after which they undergo 5′ and 3′ end-processing and assemble with the seven common snRNP proteins (called Sm proteins). These complexes are reimported into the nucleus and the snRNPs undergo their final modification in Cajal bodies. In addition, the U6 snRNP requires chemical modification by snoRNAs in the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5563/">nucleolus</a>. The sites of active transcription and splicing (approximately 2000–3000 sites per vertebrate nucleus) correspond to the “perichromatin fibers” seen under the <a class="def" href="https://www.ncbi.nlm.nih.gov/books/n/mboc4/A4754/def-item/A5122/">electron microscope</a>. (Adapted from J.D. Lewis and D. Tollervey, <i>Science</i>288:1385–1389, 2000.)</p><p>&nbsp;</p><p>&nbsp;</p></div></div>
</body>
</html>